<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
<title>The Elements of Bioinformatics</title>
<meta name="description" content="An interactive periodic table of common bioinformatics tools and their alternatives.">
<link rel="stylesheet" type="text/css" href="elements.css">
<script type="text/javascript" src="http://impactstory.org/embed/v1/impactstory.js"></script><script type="text/javascript" src="getElementsByClassName.js"></script><script type="text/javascript" src="html5slider.js"></script><script type="text/javascript">
var highlightIds=[];
function highlight(catId) {
 while (highlightId = highlightIds.pop()) {
   lowlight(highlightId);
 }
 var els = getElementsByClassName('cat_'+catId);
 while (el = els.pop()) {
   el.className += ' highlight';  
 }
 highlightIds.push(catId);
} 
function lowlight(catId) {
 var els = getElementsByClassName('cat_'+catId);
 while (el = els.pop()) {
   el.className = el.className.replace('highlight','');
 }
} 
var highlightSearchEls=[];
function highlightSearch(searchTerm) {
 while (el = highlightSearchEls.pop()) {
   el.className = el.className.replace('glowName','');
 }
 if (searchTerm.length==0) return;
 var els = getElementsBySearchTerm(searchTerm.toUpperCase());
 while (el = els.pop()) {
   el.className += ' glowName';  
   highlightSearchEls.push(el);
 }
} 
function getElementsBySearchTerm(searchTerm) {
 var matches=[];
 var els = getElementsByClassName('element');
 while (el = els.pop()) {
   var potentialMatches = getElementsByClassName('name', 'P', el);
   while (potential = potentialMatches.pop()) {
     var potentialText = potential.innerHTML.toUpperCase();
     if (potentialText.indexOf(searchTerm)!=-1) {
       matches.push(el);
     } 
   }
 }
 return matches;
}
var fadeYearEls=[];
function fadeYear(year) {
 if (year.length<1) { year = '0'; }
 if (year.length<2) { year = '0'+year; }
 if (year.length==2) { 
   year = ((year>=50)?'19':'20') + year;
 }  
 while (el = fadeYearEls.pop()) {
   el.className = el.className.replace('fadeYear','');
 }
 var els = getElementsAfterYear(year);
 while (el = els.pop()) {
   el.className += ' fadeYear';
   fadeYearEls.push(el);
 }
 document.getElementById('yearNumber').innerHTML=year;
}
function getElementsAfterYear(year) {
 var matches=[];
 var els = getElementsByClassName('element');
 while (el = els.pop()) {
   var potentialMatches = getElementsByClassName('year', 'P', el);
   while (potential = potentialMatches.pop()) {
     var potentialText = potential.innerHTML.toUpperCase();
     if (potentialText!='XX') {
       var potentialYear = ((potentialText>=50)?'19':'20') + potentialText;
       if (potentialYear>year) {
         matches.push(el);
       }
     } else if (year<2014) {
	 matches.push(el);
     }
   }
 }
 return matches;
}
var detailShowing='';
var openIds=[];
function show(divId) {
 while (openId = openIds.pop()) {
   hide(openId);
 }
 var item = document.getElementById(divId);
 item.style.display='';
 openIds.push(divId);
} 
function hide(divId) {
 var item = document.getElementById(divId);
 item.style.display='none';
} 
function showDetail(divId) {
 var item=document.getElementById('detail_'+divId); 
 item.style.display='';
 detailShowing = divId;
}
function hideDetail(divId) {
 if (divId=='') return;
 var item=document.getElementById('detail_'+divId); 
 item.style.display='none';
 detailShowing = '';
}
        </script>
</head>
<body>
<script type="text/javascript">
  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-6011705-1']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();
</script><script>(function(d, s, id) {
  var js, fjs = d.getElementsByTagName(s)[0];
  if (d.getElementById(id)) return;
  js = d.createElement(s); js.id = id;
  js.src = "//connect.facebook.net/en_GB/all.js#xfbml=1";
  fjs.parentNode.insertBefore(js, fjs);
}(document, 'script', 'facebook-jssdk'));</script><script>(function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0];if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src="//platform.twitter.com/widgets.js";fjs.parentNode.insertBefore(js,fjs);}}(document,"script","twitter-wjs"));</script><script type="text/javascript">
  window.___gcfg = {lang: 'en-GB'};
  (function() {
    var po = document.createElement('script'); po.type = 'text/javascript'; po.async = true;
    po.src = 'https://apis.google.com/js/plusone.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(po, s);
  })();
</script><script type="text/javascript">
document.onkeyup = function(e) {
  if (e.keyCode == 27) { hideDetail(detailShowing); }   // esc
};
</script><div id="header">
<h1>
<a href="http://www.eaglegenomics.com/"><img src="logo.png" height="110" width="100" alt="Eagle Genomics logo."></a>The Elements of Bioinformatics</h1>
<div id="search"><p>Search by name:<br><input type="text" size="20" onkeyup="javascript:highlightSearch(this.value);"><br>Filter by year:<br>1970<nobr></nobr><input type="range" name="points" min="1970" max="2014" value="2014" step="1" onchange="javascript:fadeYear(this.value);"><nobr></nobr>2014<br><span id="yearNumber">2014</span></p></div>
<div id="social">
<div id="fb-root"></div>
<div class="fb-like" data-send="false" data-layout="button_count" data-width="450" data-show-faces="false"></div>
<div class="tw-like"><a href="https://twitter.com/share" class="twitter-share-button" data-related="eaglegen" data-hashtags="egelements">Tweet</a></div>
<div class="g-plusone" data-size="small"></div>
<div class="whatsnew"><a href="elements-about.html">What's new?</a></div>
</div>
</div>
<div id="content"><div id="ptable">
<div id="key">
<table id="explain"><tr><td class="element cat_Xx " onmouseover="javascript:show('idp81728');">
<div class="magnifiable">
<p class="year">Year</p>
<br><p class="label unlinked">Xx</p>
<br><p class="name">NAME</p>
<div class="elementMagnify cat_Xx new" id="idp81728" style="display: none;" onmouseout="javascript:hide('idp81728');">
<p class="year">20Year</p>
<br><a href="javascript:showDetail('idp81728');"><p class="label unlinked">Xx</p></a><br><p class="name">NAME</p>
</div>
</div>
<div class="elementDetail" id="detail_idp81728" style="display: none;">
<div class="bg"></div>
<div class="fg cat_Xx">
<p class="detailClose"><a href="javascript:hideDetail('idp81728');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">NAME</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> </td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">20Year</td>
<td rowspan="7">No impact data available.</td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="index.html"></a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight"></td>
</tr>
<tr>
<td class="detailLeft">Key publication:</td>
<td class="detailRight"><a target="_blank" href="index.html"></a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight"></td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight"></td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight"></td>
</tr>
</table>
</div>
</div>
</td></tr></table>
<table id="keycolours">
<tr><th colspan="2">KEY TO TOOL TYPE (stripes indicate new update)</th></tr>
<tr>
<td class="cat_ALP" onmouseover="javascript:highlight('ALP');" onmouseout="javascript:lowlight('ALP');">Aligners (pairwise)</td>
<td class="cat_GPM" onmouseover="javascript:highlight('GPM');" onmouseout="javascript:lowlight('GPM');">Gene Prediction (mRNA)</td>
</tr>
<tr>
<td class="cat_ALS" onmouseover="javascript:highlight('ALS');" onmouseout="javascript:lowlight('ALS');">Aligners (short read)</td>
<td class="cat_GPN" onmouseover="javascript:highlight('GPN');" onmouseout="javascript:lowlight('GPN');">Gene Prediction (ncRNA)</td>
</tr>
<tr>
<td class="cat_OA" onmouseover="javascript:highlight('OA');" onmouseout="javascript:lowlight('OA');">Other Aligners</td>
<td class="cat_ST" onmouseover="javascript:highlight('ST');" onmouseout="javascript:lowlight('ST');">Sequence Tools</td>
</tr>
<tr>
<td class="cat_MSA" onmouseover="javascript:highlight('MSA');" onmouseout="javascript:lowlight('MSA');">Multiple Sequence Aligners</td>
<td class="cat_DB" onmouseover="javascript:highlight('DB');" onmouseout="javascript:lowlight('DB');">Database/Warehouse</td>
</tr>
<tr>
<td class="cat_AGL" onmouseover="javascript:highlight('AGL');" onmouseout="javascript:lowlight('AGL');">Assemblers Genomic (long read)</td>
<td class="cat_WF" onmouseover="javascript:highlight('WF');" onmouseout="javascript:lowlight('WF');">Workflows</td>
</tr>
<tr>
<td class="cat_AGS" onmouseover="javascript:highlight('AGS');" onmouseout="javascript:lowlight('AGS');">Assemblers Genomic (short read)</td>
<td class="cat_GB" onmouseover="javascript:highlight('GB');" onmouseout="javascript:lowlight('GB');">Genome Browsers</td>
</tr>
<tr>
<td class="cat_ASM" onmouseover="javascript:highlight('ASM');" onmouseout="javascript:lowlight('ASM');">Assemblers (mRNA)</td>
<td class="cat_TK" onmouseover="javascript:highlight('TK');" onmouseout="javascript:lowlight('TK');">ToolKits and APIs</td>
</tr>
<tr>
<td class="cat_SM" onmouseover="javascript:highlight('SM');" onmouseout="javascript:lowlight('SM');">Structure Modelling</td>
<td class="cat_SV" onmouseover="javascript:highlight('SV');" onmouseout="javascript:lowlight('SV');">Structure Visualisation</td>
</tr>
</table>
</div>
 <p class="tablelabel opensource">Open Source Tools</p>
<table id="opensource">
  <tr>
  <td class="element cat_ALP " onmouseover="javascript:show('idp53280');">
<div class="magnifiable">
<p class="year">90</p>
<br><p class="label linked">Bn</p>
<br><p class="name">BLAST NCBI</p>
<div class="elementMagnify cat_ALP new" id="idp53280" style="display: none;" onmouseout="javascript:hide('idp53280');">
<p class="year">1990</p>
<br><a href="javascript:showDetail('idp53280');"><p class="label linked">Bn</p></a><br><p class="name">BLAST NCBI</p>
</div>
</div>
<div class="elementDetail" id="detail_idp53280" style="display: none;">
<div class="bg"></div>
<div class="fg cat_ALP">
<p class="detailClose"><a href="javascript:hideDetail('idp53280');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">BLAST NCBI</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> The Basic Local Alignment Search Tool aka (BLAST) is the most widely used sequence similarity tool. There are versions of BLAST that compare protein queries to protein databases, nucleotide queries to nucleotide databases, as well as versions that translate nucleotide queries or databases in all six frames and compare to protein databases or queries.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">1990</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1016/S0022-2836(05)80360-2" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">http://blast.ncbi.nlm.nih.gov/Blast.cgi</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Stephen Altschul, Warren Gish, Webb Miller, Eugene Myers, and David J. Lipman</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1016/S0022-2836(05)80360-2">10.1016/S0022-2836(05)80360-2</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Windows, Unix, Mac OS X</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Public Domain</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Aligners (pairwise)</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_ALS " onmouseover="javascript:show('idp76656');">
<div class="magnifiable">
<p class="year">08</p>
<br><p class="label linked">So</p>
<br><p class="name">SOAP</p>
<div class="elementMagnify cat_ALS new" id="idp76656" style="display: none;" onmouseout="javascript:hide('idp76656');">
<p class="year">2008</p>
<br><a href="javascript:showDetail('idp76656');"><p class="label linked">So</p></a><br><p class="name">SOAP</p>
</div>
</div>
<div class="elementDetail" id="detail_idp76656" style="display: none;">
<div class="bg"></div>
<div class="fg cat_ALS">
<p class="detailClose"><a href="javascript:hideDetail('idp76656');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">SOAP</td>
<td class="detailLogo" colspan="2"><img src="http://www.eaglegenomics.com/wp-content/uploads/2012/08/Soap.jpg"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> SOAPaligner/soap2 is a member of the SOAP (Short Oligonucleotide Analysis Package). It is an updated version of SOAP software for short oligonucleotide alignment. The new program features in super fast and accurate alignment for huge amounts of short reads generated by Illumina/Solexa Genome Analyzer.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2008</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/bioinformatics/btn025" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://soap.genomics.org.cn/">http://soap.genomics.org.cn/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Prof. T.W. Lam, Alan Tam, Simon Wong, Edward Wu and S.M. Yiu</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/bioinformatics/btn025">10.1093/bioinformatics/btn025</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, MacOSX</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GNU GPL v.3</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Aligners (short read)</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="empty" colspan="11"></td>
  <td class="element cat_TK " onmouseover="javascript:show('idp28272');">
<div class="magnifiable">
<p class="year">81</p>
<br><p class="label linked">Py</p>
<br><p class="name">PHYLIP</p>
<div class="elementMagnify cat_TK new" id="idp28272" style="display: none;" onmouseout="javascript:hide('idp28272');">
<p class="year">1981</p>
<br><a href="javascript:showDetail('idp28272');"><p class="label linked">Py</p></a><br><p class="name">PHYLIP</p>
</div>
</div>
<div class="elementDetail" id="detail_idp28272" style="display: none;">
<div class="bg"></div>
<div class="fg cat_TK">
<p class="detailClose"><a href="javascript:hideDetail('idp28272');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">PHYLIP</td>
<td class="detailLogo" colspan="2"><img src="http://www.eaglegenomics.com/wp-content/uploads/2012/09/phylip.gif"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> PHYLIP (the PHYLogeny Inference Package) is a package of programs for inferring phylogenies (evolutionary trees).</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">1981</td>
<td rowspan="7">No impact data available.</td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://evolution.genetics.washington.edu/phylip.html">http://evolution.genetics.washington.edu/phylip.html</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Jerry Shurman, Chistopher Meacham, Mary Kuhner, Jan Yamato, Naruya Saitou and Mark Moehring</td>
</tr>
<tr>
<td class="detailLeft">Key publication:</td>
<td class="detailRight"><a target="_blank" href="http://nebc.nerc.ac.uk/bioinformatics/docs/phylip.html">http://nebc.nerc.ac.uk/bioinformatics/docs/phylip.html</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, Mac OS X and Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Open source</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">ToolKits and APIs</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_TK " onmouseover="javascript:show('idp216240');">
<div class="magnifiable">
<p class="year">00</p>
<br><p class="label linked">Em</p>
<br><p class="name">EMBOSS</p>
<div class="elementMagnify cat_TK new" id="idp216240" style="display: none;" onmouseout="javascript:hide('idp216240');">
<p class="year">2000</p>
<br><a href="javascript:showDetail('idp216240');"><p class="label linked">Em</p></a><br><p class="name">EMBOSS</p>
</div>
</div>
<div class="elementDetail" id="detail_idp216240" style="display: none;">
<div class="bg"></div>
<div class="fg cat_TK">
<p class="detailClose"><a href="javascript:hideDetail('idp216240');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">EMBOSS</td>
<td class="detailLogo" colspan="2"><img src="http://www.eaglegenomics.com/wp-content/uploads/2012/09/emboss.jpg"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> EMBOSS is 'The European Molecular Biology Open Software Suite'. EMBOSS is a free Open Source software analysis package specially developed for the needs of the molecular biology user community. The software automatically copes with data in a variety of formats and even allows transparent retrieval of sequence data from the web.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2000</td>
<td rowspan="7">No impact data available.</td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://emboss.sourceforge.net/">http://emboss.sourceforge.net/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Tim Carver and Lisa Mullan</td>
</tr>
<tr>
<td class="detailLeft">Key publication:</td>
<td class="detailRight"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/10827456">http://www.ncbi.nlm.nih.gov/pubmed/10827456</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, Mac OS X, Unix and Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GPL and LGPL</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">ToolKits and APIs</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_TK " onmouseover="javascript:show('idp219568');">
<div class="magnifiable">
<p class="year">00</p>
<br><p class="label linked">Bj</p>
<br><p class="name">BIOJAVA</p>
<div class="elementMagnify cat_TK new" id="idp219568" style="display: none;" onmouseout="javascript:hide('idp219568');">
<p class="year">2000</p>
<br><a href="javascript:showDetail('idp219568');"><p class="label linked">Bj</p></a><br><p class="name">BIOJAVA</p>
</div>
</div>
<div class="elementDetail" id="detail_idp219568" style="display: none;">
<div class="bg"></div>
<div class="fg cat_TK">
<p class="detailClose"><a href="javascript:hideDetail('idp219568');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">BIOJAVA</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> BioJava is an open-source project dedicated to providing a Java framework for processing biological data. It includes objects for manipulating biological sequences, file parsers, DAS client and server support, access to BioSQL and Ensembl databases, tools for making sequence analysis GUIs and powerful analysis and statistical routines including a dynamic programming toolkit.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2000</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/bioinformatics/bts494" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://biojava.org/wiki/Main_Page">http://biojava.org/wiki/Main_Page</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Thimas Down, Michael Heuer, David Huen, Matthew Pocock, Mark Schreiber, Richard Holland, Martin Szugat, Keith James, Sylvain Foisy, Andreas Prlic, Dickson S. Guedes, Francois Pepin and Jianjiong Gao</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/bioinformatics/bts494">10.1093/bioinformatics/bts494</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Mac OS X, Unix, Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">LGPL v 2.1</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">ToolKits and APIs</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_TK " onmouseover="javascript:show('idp222832');">
<div class="magnifiable">
<p class="year">01</p>
<br><p class="label linked">Bc</p>
<br><p class="name">BIOCONDUCTOR</p>
<div class="elementMagnify cat_TK new" id="idp222832" style="display: none;" onmouseout="javascript:hide('idp222832');">
<p class="year">2001</p>
<br><a href="javascript:showDetail('idp222832');"><p class="label linked">Bc</p></a><br><p class="name">BIOCONDUCTOR</p>
</div>
</div>
<div class="elementDetail" id="detail_idp222832" style="display: none;">
<div class="bg"></div>
<div class="fg cat_TK">
<p class="detailClose"><a href="javascript:hideDetail('idp222832');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">BIOCONDUCTOR</td>
<td class="detailLogo" colspan="2"><img src="http://www.eaglegenomics.com/wp-content/uploads/2012/09/logo_bioconductor.gif"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Bioconductor is an open source, open development software project to provide tools for the analysis and comprehension of high-throughput genomic data.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2001</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1186/gb-2004-5-10-r80" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.bioconductor.org/help/publications/books/bioinformatics-and-computational-biology-solutions/microarrays/">http://www.bioconductor.org/help/publications/books/bioinformatics-and-computational-biology-solutions/microarrays/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Vince Carey, Marc Carlson, Sean Davis and others.</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1186/gb-2004-5-10-r80">10.1186/gb-2004-5-10-r80</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, Mac OS X and Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Artistic 2.0, GPL2</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">ToolKits and APIs</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_OA " onmouseover="javascript:show('idp226352');">
<div class="magnifiable">
<p class="year">10</p>
<br><p class="label linked">Bi</p>
<br><p class="name">Bismark</p>
<div class="elementMagnify cat_OA new" id="idp226352" style="display: none;" onmouseout="javascript:hide('idp226352');">
<p class="year">2010</p>
<br><a href="javascript:showDetail('idp226352');"><p class="label linked">Bi</p></a><br><p class="name">Bismark</p>
</div>
</div>
<div class="elementDetail" id="detail_idp226352" style="display: none;">
<div class="bg"></div>
<div class="fg cat_OA">
<p class="detailClose"><a href="javascript:hideDetail('idp226352');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">Bismark</td>
<td class="detailLogo" colspan="2"><img src="http://www.bioinformatics.babraham.ac.uk/projects/bismark/bismark.png"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> A tool to map bisulfite converted sequence reads and determine cytosine methylation states.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2010</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/bioinformatics/btr167" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.bioinformatics.babraham.ac.uk/projects/bismark/">http://www.bioinformatics.babraham.ac.uk/projects/bismark/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Felix Krueger</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/bioinformatics/btr167">10.1093/bioinformatics/btr167</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, Mac OS X and Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GPL v3 or later</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Other Aligners</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_SM " onmouseover="javascript:show('idp229760');">
<div class="magnifiable">
<p class="year">03</p>
<br><p class="label linked">Ms</p>
<br><p class="name">SWISS-MODEL</p>
<div class="elementMagnify cat_SM new" id="idp229760" style="display: none;" onmouseout="javascript:hide('idp229760');">
<p class="year">2003</p>
<br><a href="javascript:showDetail('idp229760');"><p class="label linked">Ms</p></a><br><p class="name">SWISS-MODEL</p>
</div>
</div>
<div class="elementDetail" id="detail_idp229760" style="display: none;">
<div class="bg"></div>
<div class="fg cat_SM">
<p class="detailClose"><a href="javascript:hideDetail('idp229760');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">SWISS-MODEL</td>
<td class="detailLogo" colspan="2"><img src="http://swissmodel.expasy.org/img/sms_medium.png"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> SWISS-MODEL is a fully automated protein structure homology-modeling server, accessible via the ExPASy web server, or from the program DeepView (Swiss Pdb-Viewer). The purpose of this server is to make Protein Modelling accessible to all biochemists and molecular biologists worldwide.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2003</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/bioinformatics/bti770" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://swissmodel.expasy.org/">http://swissmodel.expasy.org/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Manuel Peitsch, et al</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/bioinformatics/bti770">10.1093/bioinformatics/bti770</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Web-based</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Free, commercial users need to sign an agreement.</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Structure Modelling</td>
</tr>
</table>
</div>
</div>
</td>
  </tr>
  <tr>
  <td class="element cat_ALP " onmouseover="javascript:show('idp233584');">
<div class="magnifiable">
<p class="year">98</p>
<br><p class="label linked">Hm</p>
<br><p class="name">HMMER</p>
<div class="elementMagnify cat_ALP new" id="idp233584" style="display: none;" onmouseout="javascript:hide('idp233584');">
<p class="year">1998</p>
<br><a href="javascript:showDetail('idp233584');"><p class="label linked">Hm</p></a><br><p class="name">HMMER</p>
</div>
</div>
<div class="elementDetail" id="detail_idp233584" style="display: none;">
<div class="bg"></div>
<div class="fg cat_ALP">
<p class="detailClose"><a href="javascript:hideDetail('idp233584');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">HMMER</td>
<td class="detailLogo" colspan="2"><img src="http://www.eaglegenomics.com/wp-content/uploads/2012/08/hmmer_titlebar_small.gif"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> HMMER is used for searching sequence databases for homologs of protein sequences, and for making protein sequence alignments. It implements methods using probabilistic models called profile hidden Markov models.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">1998</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/nar/gkr367" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://hmmer.janelia.org/">http://hmmer.janelia.org/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Sean R. Eddy</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/nar/gkr367">10.1093/nar/gkr367</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Windows, MacOSX, Linux</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GPLv3</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Aligners (pairwise)</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_ALS " onmouseover="javascript:show('idp237040');">
<div class="magnifiable">
<p class="year">07</p>
<br><p class="label linked">Mq</p>
<br><p class="name">MAQ</p>
<div class="elementMagnify cat_ALS new" id="idp237040" style="display: none;" onmouseout="javascript:hide('idp237040');">
<p class="year">2007</p>
<br><a href="javascript:showDetail('idp237040');"><p class="label linked">Mq</p></a><br><p class="name">MAQ</p>
</div>
</div>
<div class="elementDetail" id="detail_idp237040" style="display: none;">
<div class="bg"></div>
<div class="fg cat_ALS">
<p class="detailClose"><a href="javascript:hideDetail('idp237040');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">MAQ</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Maq stands for Mapping and Assembly with Quality It builds assembly by mapping short reads to reference sequences. Maq is previously known as mapass2. Maq is a software that builds mapping assemblies from short reads generated by the next-generation sequencing machines. It is particularly designed for Illumina-Solexa 1G Genetic Analyzer, and has preliminary functions to handle ABI SOLiD data.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2007</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1101/gr.078212.108" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://maq.sourceforge.net/">http://maq.sourceforge.net/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Heng Li</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1101/gr.078212.108">10.1101/gr.078212.108</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GNU GPL v3</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Aligners (short read)</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="empty" colspan="10"></td>
  <td class="element cat_DB " onmouseover="javascript:show('idp240848');">
<div class="magnifiable">
<p class="year">01</p>
<br><p class="label linked">Bs</p>
<br><p class="name">bioSQL</p>
<div class="elementMagnify cat_DB new" id="idp240848" style="display: none;" onmouseout="javascript:hide('idp240848');">
<p class="year">2001</p>
<br><a href="javascript:showDetail('idp240848');"><p class="label linked">Bs</p></a><br><p class="name">bioSQL</p>
</div>
</div>
<div class="elementDetail" id="detail_idp240848" style="display: none;">
<div class="bg"></div>
<div class="fg cat_DB">
<p class="detailClose"><a href="javascript:hideDetail('idp240848');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">bioSQL</td>
<td class="detailLogo" colspan="2"><img src="http://www.eaglegenomics.com/wp-content/uploads/2012/09/BioSQL.jpg"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> BioSQL is a generic unifying schema for storing sequences from different sources, for instance Genbank or Swissprot. BioSQL is meant to be a common data storage layer supported by all the different Bio* projects, Bioperl, Biojava, Biopython, and Bioruby. Entries stored through an application written in, say, Bioperl could be retrieved by another written in Biojava.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2001</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1186/2041-1480-1-8" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.biosql.org/wiki/Main_Page">http://www.biosql.org/wiki/Main_Page</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Ewan Birney</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1186/2041-1480-1-8">10.1186/2041-1480-1-8</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Unix, Mac OS X, Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GNU Lesser General Public Licence v 3</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Database/Warehouse</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_WF " onmouseover="javascript:show('idp244528');">
<div class="magnifiable">
<p class="year">00</p>
<br><p class="label linked">Eh</p>
<br><p class="name">eHIVE</p>
<div class="elementMagnify cat_WF new" id="idp244528" style="display: none;" onmouseout="javascript:hide('idp244528');">
<p class="year">2000</p>
<br><a href="javascript:showDetail('idp244528');"><p class="label linked">Eh</p></a><br><p class="name">eHIVE</p>
</div>
</div>
<div class="elementDetail" id="detail_idp244528" style="display: none;">
<div class="bg"></div>
<div class="fg cat_WF">
<p class="detailClose"><a href="javascript:hideDetail('idp244528');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">eHIVE</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> This is a distributed processing system based on 'autonomous agents' and the behavioural structure of honey bees. It implements all functionality of both data-flow graphs and block-branch diagrams which should allow it to codify any program, algorithm, or parallel processing job control system. It is not bound to any processing 'farm' system and can be adapted to any GRID.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2000</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1186/1471-2105-11-240" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.ensembl.org/info/docs/eHive/index.html">http://www.ensembl.org/info/docs/eHive/index.html</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Abel Ureta-Vidal, Jessica Severin, Michael Schuster</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1186/1471-2105-11-240">10.1186/1471-2105-11-240</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, Mac OS X, Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Open Source</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Workflows</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_TK " onmouseover="javascript:show('idp247952');">
<div class="magnifiable">
<p class="year">00</p>
<br><p class="label linked">En</p>
<br><p class="name">ENSEMBL API</p>
<div class="elementMagnify cat_TK new" id="idp247952" style="display: none;" onmouseout="javascript:hide('idp247952');">
<p class="year">2000</p>
<br><a href="javascript:showDetail('idp247952');"><p class="label linked">En</p></a><br><p class="name">ENSEMBL API</p>
</div>
</div>
<div class="elementDetail" id="detail_idp247952" style="display: none;">
<div class="bg"></div>
<div class="fg cat_TK">
<p class="detailClose"><a href="javascript:hideDetail('idp247952');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">ENSEMBL API</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> The Ensembl API (application programming interface) is a framework for applications that need to access or store data in Ensembl's databases.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2000</td>
<td rowspan="7">No impact data available.</td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.ensembl.org/info/docs/api/index.html">http://www.ensembl.org/info/docs/api/index.html</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Glenn Proctor, Ian Longden and Patrick Meidl</td>
</tr>
<tr>
<td class="detailLeft">Key publication:</td>
<td class="detailRight"><a target="_blank" href="index.html"></a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Unix, Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Apache-style licence.</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">ToolKits and APIs</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_TK " onmouseover="javascript:show('idp250656');">
<div class="magnifiable">
<p class="year">00</p>
<br><p class="label linked">Br</p>
<br><p class="name">bioRUBY</p>
<div class="elementMagnify cat_TK new" id="idp250656" style="display: none;" onmouseout="javascript:hide('idp250656');">
<p class="year">2000</p>
<br><a href="javascript:showDetail('idp250656');"><p class="label linked">Br</p></a><br><p class="name">bioRUBY</p>
</div>
</div>
<div class="elementDetail" id="detail_idp250656" style="display: none;">
<div class="bg"></div>
<div class="fg cat_TK">
<p class="detailClose"><a href="javascript:hideDetail('idp250656');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">bioRUBY</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> BioRuby is an open source Ruby library for developing bioinformatics software.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2000</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/bioinformatics/btq475" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://bioruby.open-bio.org/">http://bioruby.open-bio.org/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Toshiaki Katayama</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/bioinformatics/btq475">10.1093/bioinformatics/btq475</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, Mac OS X and Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Ruby licence</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">ToolKits and APIs</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_TK " onmouseover="javascript:show('idp253712');">
<div class="magnifiable">
<p class="year">03</p>
<br><p class="label linked">La</p>
<br><p class="name">LAGAN</p>
<div class="elementMagnify cat_TK new" id="idp253712" style="display: none;" onmouseout="javascript:hide('idp253712');">
<p class="year">2003</p>
<br><a href="javascript:showDetail('idp253712');"><p class="label linked">La</p></a><br><p class="name">LAGAN</p>
</div>
</div>
<div class="elementDetail" id="detail_idp253712" style="display: none;">
<div class="bg"></div>
<div class="fg cat_TK">
<p class="detailClose"><a href="javascript:hideDetail('idp253712');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">LAGAN</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> The LAGAN Tookit consists of four components:CHAOS, LAGAN, Multi-LAGAN,Shuffle-LAGAN.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2003</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1101/gr.926603" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://lagan.stanford.edu/lagan_web/authors.shtml">http://lagan.stanford.edu/lagan_web/authors.shtml</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Michael Brudno</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1101/gr.926603">10.1101/gr.926603</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GNU GPL</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">ToolKits and APIs</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_OA new" onmouseover="javascript:show('idp256832');">
<div class="magnifiable">
<p class="year">10</p>
<br><p class="label linked">Mve</p>
<br><p class="name">Mauve</p>
<div class="elementMagnify cat_OA new" id="idp256832" style="display: none;" onmouseout="javascript:hide('idp256832');">
<p class="year">2010</p>
<br><a href="javascript:showDetail('idp256832');"><p class="label linked">Mve</p></a><br><p class="name">Mauve</p>
</div>
</div>
<div class="elementDetail" id="detail_idp256832" style="display: none;">
<div class="bg"></div>
<div class="fg cat_OA">
<p class="detailClose"><a href="javascript:hideDetail('idp256832');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">Mauve</td>
<td class="detailLogo" colspan="2"><img src="http://asap.ahabs.wisc.edu/images/stories/mauve_logo2.png"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Mauve is a system for efficiently constructing multiple genome alignments in the presence of large-scale evolutionary events such as rearrangement and inversion. Multiple genome alignment provides a basis for research into comparative genomics and the study of evolutionary dynamics.  Aligning whole genomes is a fundamentally different problem than aligning short sequences.  Mauve has been developed with the idea that a multiple genome aligner should require only modest computational resources. It employs algorithmic techniques that scale well in the amount of sequence being aligned. For example, a pair of Y. pestis genomes can be aligned in under a minute, while a group of 9 divergent Enterobacterial genomes can be aligned in a few hours.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2010</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1371/journal.pone.0011147" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://gel.ahabs.wisc.edu/mauve/">http://gel.ahabs.wisc.edu/mauve/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Darling AE, Mau B, Perna NT</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1371/journal.pone.0011147">10.1371/journal.pone.0011147</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Windows, Mac, Linux</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GNU GPL</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Other Aligners</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_SM " onmouseover="javascript:show('idp260816');">
<div class="magnifiable">
<p class="year">05</p>
<br><p class="label linked">Hh</p>
<br><p class="name">HHpred</p>
<div class="elementMagnify cat_SM new" id="idp260816" style="display: none;" onmouseout="javascript:hide('idp260816');">
<p class="year">2005</p>
<br><a href="javascript:showDetail('idp260816');"><p class="label linked">Hh</p></a><br><p class="name">HHpred</p>
</div>
</div>
<div class="elementDetail" id="detail_idp260816" style="display: none;">
<div class="bg"></div>
<div class="fg cat_SM">
<p class="detailClose"><a href="javascript:hideDetail('idp260816');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">HHpred</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> The primary aim in developing HHpred was to provide biologists with a method for sequence database searching and structure prediction that is as easy to use as BLAST or PSI-BLAST and that is at the same time much more sensitive in finding remote homologs. In fact, HHpred's sensitivity is competitive with the most powerful servers for structure prediction currently available.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2005</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/nar/gki408" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://toolkit.tuebingen.mpg.de/hhpred">http://toolkit.tuebingen.mpg.de/hhpred</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Soding, Biegert, Lupas, et al</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/nar/gki408">10.1093/nar/gki408</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, Mac OS X</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GPL v3</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Structure Modelling</td>
</tr>
</table>
</div>
</div>
</td>
  </tr>
  <tr>
  <td class="element cat_ALP " onmouseover="javascript:show('idp206480');">
<div class="magnifiable">
<p class="year">99</p>
<br><p class="label linked">Mu</p>
<br><p class="name">MUMmer</p>
<div class="elementMagnify cat_ALP new" id="idp206480" style="display: none;" onmouseout="javascript:hide('idp206480');">
<p class="year">1999</p>
<br><a href="javascript:showDetail('idp206480');"><p class="label linked">Mu</p></a><br><p class="name">MUMmer</p>
</div>
</div>
<div class="elementDetail" id="detail_idp206480" style="display: none;">
<div class="bg"></div>
<div class="fg cat_ALP">
<p class="detailClose"><a href="javascript:hideDetail('idp206480');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">MUMmer</td>
<td class="detailLogo" colspan="2"><img src="http://www.eaglegenomics.com/wp-content/uploads/2012/08/Mummer-logo.gif"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> MUMmer is a system for rapidly aligning entire genomes, whether in complete or draft form. For example, MUMmer 3.0 can find all 20-basepair or longer exact matches between a pair of 5-megabase genomes in 13.7 seconds, using 78 MB of memory, on a 2.4 GHz Linux desktop computer.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">1999</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1186/gb-2004-5-2-r12" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://mummer.sourceforge.net/">http://mummer.sourceforge.net/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Stefan Kurtz, Adam Phillippy, Art Delcher and Steven Salzberg</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1186/gb-2004-5-2-r12">10.1186/gb-2004-5-2-r12</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Unix</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Artistic License</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Aligners (pairwise)</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_ALS " onmouseover="javascript:show('idp210128');">
<div class="magnifiable">
<p class="year">08</p>
<br><p class="label linked">Bo</p>
<br><p class="name">Bowtie</p>
<div class="elementMagnify cat_ALS new" id="idp210128" style="display: none;" onmouseout="javascript:hide('idp210128');">
<p class="year">2008</p>
<br><a href="javascript:showDetail('idp210128');"><p class="label linked">Bo</p></a><br><p class="name">Bowtie</p>
</div>
</div>
<div class="elementDetail" id="detail_idp210128" style="display: none;">
<div class="bg"></div>
<div class="fg cat_ALS">
<p class="detailClose"><a href="javascript:hideDetail('idp210128');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">Bowtie</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Bowtie is an ultrafast, memory-efficient short read aligner geared toward quickly aligning large sets of short DNA sequences (reads) to large genomes. It aligns 35-base-pair reads to the human genome at a rate of 25 million reads per hour on a typical workstation.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2008</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1038/nmeth.1923" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://bowtie-bio.sourceforge.net/bowtie2/index.shtml">http://bowtie-bio.sourceforge.net/bowtie2/index.shtml</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Ben Langmead and Cole Trapnell</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1038/nmeth.1923">10.1038/nmeth.1923</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Windows, Linux, MacOSX</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GNU GPL v3</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Aligners (short read)</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_MSA " onmouseover="javascript:show('idp213456');">
<div class="magnifiable">
<p class="year">94</p>
<br><p class="label linked">Cl</p>
<br><p class="name">CLUSTALW</p>
<div class="elementMagnify cat_MSA new" id="idp213456" style="display: none;" onmouseout="javascript:hide('idp213456');">
<p class="year">1994</p>
<br><a href="javascript:showDetail('idp213456');"><p class="label linked">Cl</p></a><br><p class="name">CLUSTALW</p>
</div>
</div>
<div class="elementDetail" id="detail_idp213456" style="display: none;">
<div class="bg"></div>
<div class="fg cat_MSA">
<p class="detailClose"><a href="javascript:hideDetail('idp213456');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">CLUSTALW</td>
<td class="detailLogo" colspan="2"><img src="http://www.eaglegenomics.com/wp-content/uploads/2012/08/clustalw_title.png"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Clustal W is a general purpose multiple alignment program for DNA or proteins. Clustal 2 comes in two flavors: the command-line version ClustalW and the graphical version ClustalX.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">1994</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/bioinformatics/btm404" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.clustal.org/clustal2/">http://www.clustal.org/clustal2/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Higgins DG, Sharp PM</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/bioinformatics/btm404">10.1093/bioinformatics/btm404</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, MacOSX, Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GNU Lesser GPL</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Multiple Sequence Aligners</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_MSA " onmouseover="javascript:show('idp283232');">
<div class="magnifiable">
<p class="year">06</p>
<br><p class="label linked">Pe</p>
<br><p class="name">PECAN</p>
<div class="elementMagnify cat_MSA new" id="idp283232" style="display: none;" onmouseout="javascript:hide('idp283232');">
<p class="year">2006</p>
<br><a href="javascript:showDetail('idp283232');"><p class="label linked">Pe</p></a><br><p class="name">PECAN</p>
</div>
</div>
<div class="elementDetail" id="detail_idp283232" style="display: none;">
<div class="bg"></div>
<div class="fg cat_MSA">
<p class="detailClose"><a href="javascript:hideDetail('idp283232');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">PECAN</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Pecan is a consistency based multiple-alignment program developed by Benedict Paten in Ewan Birney's group at the EBI.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2006</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1101/gr.076554.108" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.ebi.ac.uk/~bjp/pecan/">http://www.ebi.ac.uk/~bjp/pecan/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Benedict Paten</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1101/gr.076554.108">10.1101/gr.076554.108</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight"></td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Multiple Sequence Aligners</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_AGL " onmouseover="javascript:show('idp286080');">
<div class="magnifiable">
<p class="year">97</p>
<br><p class="label linked">Mi</p>
<br><p class="name">MIRA 97</p>
<div class="elementMagnify cat_AGL new" id="idp286080" style="display: none;" onmouseout="javascript:hide('idp286080');">
<p class="year">1997</p>
<br><a href="javascript:showDetail('idp286080');"><p class="label linked">Mi</p></a><br><p class="name">MIRA 97</p>
</div>
</div>
<div class="elementDetail" id="detail_idp286080" style="display: none;">
<div class="bg"></div>
<div class="fg cat_AGL">
<p class="detailClose"><a href="javascript:hideDetail('idp286080');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">MIRA 97</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> MIRA is a whole genome shotgun and EST sequence assembler for Sanger, 454, Solexa (Illumina), IonTorrent data and PacBio. It can be seen as a Swiss army knife of sequence assembly developed and used in the past 12 years to get assembly jobs done efficiently - and especially accurately.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">1997</td>
<td rowspan="7">No impact data available.</td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://sourceforge.net/apps/mediawiki/mira-assembler/index.php?title=Main_Page">http://sourceforge.net/apps/mediawiki/mira-assembler/index.php?title=Main_Page</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Bastien Chevreux</td>
</tr>
<tr>
<td class="detailLeft">Key publication:</td>
<td class="detailRight"><a target="_blank" href="http://www.bioinfo.de/isb/gcb99/talks/chevreux/">http://www.bioinfo.de/isb/gcb99/talks/chevreux/</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, Unix</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GNU GPL v2</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Assemblers Genomic (long read)</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_AGL " onmouseover="javascript:show('idp289200');">
<div class="magnifiable">
<p class="year">04</p>
<br><p class="label linked">Se</p>
<br><p class="name">SEQCLEAN</p>
<div class="elementMagnify cat_AGL new" id="idp289200" style="display: none;" onmouseout="javascript:hide('idp289200');">
<p class="year">2004</p>
<br><a href="javascript:showDetail('idp289200');"><p class="label linked">Se</p></a><br><p class="name">SEQCLEAN</p>
</div>
</div>
<div class="elementDetail" id="detail_idp289200" style="display: none;">
<div class="bg"></div>
<div class="fg cat_AGL">
<p class="detailClose"><a href="javascript:hideDetail('idp289200');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">SEQCLEAN</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> A script for automated trimming and validation of ESTs or other DNA sequences by screening for various contaminants, low quality and low-complexity sequences.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2004</td>
<td rowspan="7">No impact data available.</td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://compbio.dfci.harvard.edu/tgi/software/">http://compbio.dfci.harvard.edu/tgi/software/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">The Gene Indices Group  -Geo Pertea-</td>
</tr>
<tr>
<td class="detailLeft">Key publication:</td>
<td class="detailRight"><a target="_blank" href="index.html"></a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Artistic licence</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Assemblers Genomic (long read)</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_SV " onmouseover="javascript:show('idp291856');">
<div class="magnifiable">
<p class="year">01</p>
<br><p class="label linked">Jm</p>
<br><p class="name">Jmol</p>
<div class="elementMagnify cat_SV new" id="idp291856" style="display: none;" onmouseout="javascript:hide('idp291856');">
<p class="year">2001</p>
<br><a href="javascript:showDetail('idp291856');"><p class="label linked">Jm</p></a><br><p class="name">Jmol</p>
</div>
</div>
<div class="elementDetail" id="detail_idp291856" style="display: none;">
<div class="bg"></div>
<div class="fg cat_SV">
<p class="detailClose"><a href="javascript:hideDetail('idp291856');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">Jmol</td>
<td class="detailLogo" colspan="2"><img src="http://jmol.sourceforge.net/images/Jmol_smallLogo.jpg"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Jmol is a free, open source molecule viewer for students, educators, and researchers in chemistry and biochemistry.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2001</td>
<td rowspan="7">No impact data available.</td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://jmol.sourceforge.net/">http://jmol.sourceforge.net/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Egon Willighagen, et al</td>
</tr>
<tr>
<td class="detailLeft">Key publication:</td>
<td class="detailRight"><a target="_blank" href="http://www.scribd.com/doc/14333194/Processing-CML-Conventions-in-Java">http://www.scribd.com/doc/14333194/Processing-CML-Conventions-in-Java</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, Windows, Mac OS X</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">LGPL</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Structure Visualisation</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_AGS " onmouseover="javascript:show('idp295024');">
<div class="magnifiable">
<p class="year">09</p>
<br><p class="label linked">Sd</p>
<br><p class="name">SOAP: DE NOVO 2</p>
<div class="elementMagnify cat_AGS new" id="idp295024" style="display: none;" onmouseout="javascript:hide('idp295024');">
<p class="year">2009</p>
<br><a href="javascript:showDetail('idp295024');"><p class="label linked">Sd</p></a><br><p class="name">SOAP: DE NOVO 2</p>
</div>
</div>
<div class="elementDetail" id="detail_idp295024" style="display: none;">
<div class="bg"></div>
<div class="fg cat_AGS">
<p class="detailClose"><a href="javascript:hideDetail('idp295024');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">SOAP: DE NOVO 2</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> SOAPdenovo is a novel short-read assembly method that can build a de novo draft assembly for the human-sized genomes. The program is specially designed to assemble Illumina GA short reads.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2009</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1186/2047-217X-1-18" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://soap.genomics.org.cn/soapdenovo.html">http://soap.genomics.org.cn/soapdenovo.html</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">BGI</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1186/2047-217X-1-18">10.1186/2047-217X-1-18</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, Mac OS X </td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GPLv3</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Assemblers Genomic (short read)</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_ASM " onmouseover="javascript:show('idp298128');">
<div class="magnifiable">
<p class="year">03</p>
<br><p class="label linked">Un</p>
<br><p class="name">UNIGENE</p>
<div class="elementMagnify cat_ASM new" id="idp298128" style="display: none;" onmouseout="javascript:hide('idp298128');">
<p class="year">2003</p>
<br><a href="javascript:showDetail('idp298128');"><p class="label linked">Un</p></a><br><p class="name">UNIGENE</p>
</div>
</div>
<div class="elementDetail" id="detail_idp298128" style="display: none;">
<div class="bg"></div>
<div class="fg cat_ASM">
<p class="detailClose"><a href="javascript:hideDetail('idp298128');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">UNIGENE</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> UniGene computationally identifies transcripts from the same locus; analyzes expression by tissue, age, and health status; and reports related proteins (protEST) and clone resources.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2003</td>
<td rowspan="7">No impact data available.</td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/unigene/">http://www.ncbi.nlm.nih.gov/unigene/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Boguski MS, Schuler GD</td>
</tr>
<tr>
<td class="detailLeft">Key publication:</td>
<td class="detailRight"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/books/NBK21083/">http://www.ncbi.nlm.nih.gov/books/NBK21083/</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, Mac OS X, Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Open licence</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Assemblers (mRNA)</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_GPM " onmouseover="javascript:show('idp301280');">
<div class="magnifiable">
<p class="year">98</p>
<br><p class="label linked">Gl</p>
<br><p class="name">GLIMMER 3</p>
<div class="elementMagnify cat_GPM new" id="idp301280" style="display: none;" onmouseout="javascript:hide('idp301280');">
<p class="year">1998</p>
<br><a href="javascript:showDetail('idp301280');"><p class="label linked">Gl</p></a><br><p class="name">GLIMMER 3</p>
</div>
</div>
<div class="elementDetail" id="detail_idp301280" style="display: none;">
<div class="bg"></div>
<div class="fg cat_GPM">
<p class="detailClose"><a href="javascript:hideDetail('idp301280');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">GLIMMER 3</td>
<td class="detailLogo" colspan="2"><img src="http://www.eaglegenomics.com/wp-content/uploads/2012/09/Glimmer.jpg"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Glimmer is a system for finding genes in microbial DNA, especially the genomes of bacteria, archaea, and viruses. Glimmer (Gene Locator and Interpolated Markov ModelER) uses interpolated Markov models (IMMs) to identify the coding regions and distinguish them from noncoding DNA.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">1998</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/bioinformatics/btm009" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.cbcb.umd.edu/software/glimmer/">http://www.cbcb.umd.edu/software/glimmer/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Arthur Delcher</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/bioinformatics/btm009">10.1093/bioinformatics/btm009</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">OSI Certified Open source</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Gene Prediction (mRNA)</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_ST " onmouseover="javascript:show('idp305168');">
<div class="magnifiable">
<p class="year">06</p>
<br><p class="label linked">Cd</p>
<br><p class="name">CD-HIT</p>
<div class="elementMagnify cat_ST new" id="idp305168" style="display: none;" onmouseout="javascript:hide('idp305168');">
<p class="year">2006</p>
<br><a href="javascript:showDetail('idp305168');"><p class="label linked">Cd</p></a><br><p class="name">CD-HIT</p>
</div>
</div>
<div class="elementDetail" id="detail_idp305168" style="display: none;">
<div class="bg"></div>
<div class="fg cat_ST">
<p class="detailClose"><a href="javascript:hideDetail('idp305168');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">CD-HIT</td>
<td class="detailLogo" colspan="2"><img src="http://weizhong-lab.ucsd.edu/cd-hit/image/cd-hit-logo.gif"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> CD-HIT is a very widely used program for clustering and comparing protein or nucleotide sequences.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2006</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/bioinformatics/btl158" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://weizhong-lab.ucsd.edu/cd-hit/">http://weizhong-lab.ucsd.edu/cd-hit/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Wizhong Li, Adam Godzik, Lukasz Jaroszewski</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/bioinformatics/btl158">10.1093/bioinformatics/btl158</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GPL v2</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Sequence Tools</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_GPN " onmouseover="javascript:show('idp308928');">
<div class="magnifiable">
<p class="year">97</p>
<br><p class="label linked">Tr</p>
<br><p class="name">tRNAscan</p>
<div class="elementMagnify cat_GPN new" id="idp308928" style="display: none;" onmouseout="javascript:hide('idp308928');">
<p class="year">1997</p>
<br><a href="javascript:showDetail('idp308928');"><p class="label linked">Tr</p></a><br><p class="name">tRNAscan</p>
</div>
</div>
<div class="elementDetail" id="detail_idp308928" style="display: none;">
<div class="bg"></div>
<div class="fg cat_GPN">
<p class="detailClose"><a href="javascript:hideDetail('idp308928');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">tRNAscan</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> tRNAscan-SE was designed to make rapid, sensitive searches of genomic sequence feasible using the selectivity of the Cove analysis package. Search for tRNA genes in genomic sequence.tRNAscan-SE identifies transfer RNA genes in genomic DNA or RNA sequences. </td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">1997</td>
<td rowspan="7">No impact data available.</td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://lowelab.ucsc.edu/tRNAscan-SE/">http://lowelab.ucsc.edu/tRNAscan-SE/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Oeter Schattner, Angela N. Brooks and Todd M. Lowe</td>
</tr>
<tr>
<td class="detailLeft">Key publication:</td>
<td class="detailRight"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC146525/">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC146525/</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Unix</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GNU GPL</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Gene Prediction (ncRNA)</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_DB " onmouseover="javascript:show('idp312128');">
<div class="magnifiable">
<p class="year">03</p>
<br><p class="label linked">Bm</p>
<br><p class="name">bioMART</p>
<div class="elementMagnify cat_DB new" id="idp312128" style="display: none;" onmouseout="javascript:hide('idp312128');">
<p class="year">2003</p>
<br><a href="javascript:showDetail('idp312128');"><p class="label linked">Bm</p></a><br><p class="name">bioMART</p>
</div>
</div>
<div class="elementDetail" id="detail_idp312128" style="display: none;">
<div class="bg"></div>
<div class="fg cat_DB">
<p class="detailClose"><a href="javascript:hideDetail('idp312128');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">bioMART</td>
<td class="detailLogo" colspan="2"><img src="http://www.eaglegenomics.com/wp-content/uploads/2012/09/Biomart.jpg"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> The purpose of BioMart is to convert one or more data source (flat files or relational) into data marts which can be accessed via its  tandardised web browser interface and also via its Perl, Java and webservice APIs.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2003</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/database/bar038" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.biomart.org/">http://www.biomart.org/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Arek Kasprzyk</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/database/bar038">10.1093/database/bar038</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Open Source</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Database/Warehouse</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_WF " onmouseover="javascript:show('idp316128');">
<div class="magnifiable">
<p class="year">03</p>
<br><p class="label linked">Ta</p>
<br><p class="name">TAVERNA</p>
<div class="elementMagnify cat_WF new" id="idp316128" style="display: none;" onmouseout="javascript:hide('idp316128');">
<p class="year">2003</p>
<br><a href="javascript:showDetail('idp316128');"><p class="label linked">Ta</p></a><br><p class="name">TAVERNA</p>
</div>
</div>
<div class="elementDetail" id="detail_idp316128" style="display: none;">
<div class="bg"></div>
<div class="fg cat_WF">
<p class="detailClose"><a href="javascript:hideDetail('idp316128');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">TAVERNA</td>
<td class="detailLogo" colspan="2"><img src="http://www.eaglegenomics.com/wp-content/uploads/2012/09/taverna-wheel-logo.png"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Taverna is an open source and domain-independent Workflow Management System &#8211; a suite of tools used to design and execute scientific workflows and aid in silico experimentation.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2003</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/nar/gkt328" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.taverna.org.uk/">http://www.taverna.org.uk/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">myGrid (University of Manchester) and EBI)</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/nar/gkt328">10.1093/nar/gkt328</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, Mac OS X, Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">LPGL</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Workflows</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_GB " onmouseover="javascript:show('idp319488');">
<div class="magnifiable">
<p class="year">00</p>
<br><p class="label linked">Eb</p>
<br><p class="name">ENSEMBL BROWSER</p>
<div class="elementMagnify cat_GB new" id="idp319488" style="display: none;" onmouseout="javascript:hide('idp319488');">
<p class="year">2000</p>
<br><a href="javascript:showDetail('idp319488');"><p class="label linked">Eb</p></a><br><p class="name">ENSEMBL BROWSER</p>
</div>
</div>
<div class="elementDetail" id="detail_idp319488" style="display: none;">
<div class="bg"></div>
<div class="fg cat_GB">
<p class="detailClose"><a href="javascript:hideDetail('idp319488');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">ENSEMBL BROWSER</td>
<td class="detailLogo" colspan="2"><img src="http://projects.ensembl.org/eprojects/gfx/ebang-60.png"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Ensembl is a joint project between EMBL - EBI and the Wellcome Trust Sanger Institute to develop a software system which produces and maintains automatic annotation on selected eukaryotic genomes.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2000</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1101/gr.1863004" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.ensembl.org/index.html">http://www.ensembl.org/index.html</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">EBI and Wellcome Trust Sanger Institute</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1101/gr.1863004">10.1101/gr.1863004</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Apache-style licence.</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Genome Browsers</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_TK " onmouseover="javascript:show('idp323104');">
<div class="magnifiable">
<p class="year">00</p>
<br><p class="label linked">Bp</p>
<br><p class="name">bioPERL</p>
<div class="elementMagnify cat_TK new" id="idp323104" style="display: none;" onmouseout="javascript:hide('idp323104');">
<p class="year">2000</p>
<br><a href="javascript:showDetail('idp323104');"><p class="label linked">Bp</p></a><br><p class="name">bioPERL</p>
</div>
</div>
<div class="elementDetail" id="detail_idp323104" style="display: none;">
<div class="bg"></div>
<div class="fg cat_TK">
<p class="detailClose"><a href="javascript:hideDetail('idp323104');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">bioPERL</td>
<td class="detailLogo" colspan="2"><img src="http://www.eaglegenomics.com/wp-content/uploads/2012/09/bioperl-logo.jpg"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> BioPerl is a toolkit of perl modules useful in building bioinformatics solutions in Perl. BioPerl project is an international association of developers of open source Perl tools for bioinformatics, genomics and life science research.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2000</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1101/gr.361602" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.bioperl.org/wiki/Main_Page">http://www.bioperl.org/wiki/Main_Page</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Chris Dagdigian, Richard Resnick, Lew Gramer,Alessandro Guffanti and others.</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1101/gr.361602">10.1101/gr.361602</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, Mac OS X, Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Perl Artistic Licence</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">ToolKits and APIs</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_TK " onmouseover="javascript:show('idp326992');">
<div class="magnifiable">
<p class="year">07</p>
<br><p class="label linked">Pl</p>
<br><p class="name">PLINK</p>
<div class="elementMagnify cat_TK new" id="idp326992" style="display: none;" onmouseout="javascript:hide('idp326992');">
<p class="year">2007</p>
<br><a href="javascript:showDetail('idp326992');"><p class="label linked">Pl</p></a><br><p class="name">PLINK</p>
</div>
</div>
<div class="elementDetail" id="detail_idp326992" style="display: none;">
<div class="bg"></div>
<div class="fg cat_TK">
<p class="detailClose"><a href="javascript:hideDetail('idp326992');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">PLINK</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> PLINK is a free, open-source whole genome association analysis toolset, designed to perform a range of basic, large-scale analyses in a computationally efficient manner.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2007</td>
<td rowspan="7">No impact data available.</td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://pngu.mgh.harvard.edu/~purcell/plink/">http://pngu.mgh.harvard.edu/~purcell/plink/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Shaun Purcell</td>
</tr>
<tr>
<td class="detailLeft">Key publication:</td>
<td class="detailRight"><a target="_blank" href="http://pngu.mgh.harvard.edu/purcell/plink/">http://pngu.mgh.harvard.edu/purcell/plink/</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, Mac OS X</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GNU v.2</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">ToolKits and APIs</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_TK " onmouseover="javascript:show('idp330096');">
<div class="magnifiable">
<p class="year">09</p>
<br><p class="label linked">Sa</p>
<br><p class="name">SAMtools</p>
<div class="elementMagnify cat_TK new" id="idp330096" style="display: none;" onmouseout="javascript:hide('idp330096');">
<p class="year">2009</p>
<br><a href="javascript:showDetail('idp330096');"><p class="label linked">Sa</p></a><br><p class="name">SAMtools</p>
</div>
</div>
<div class="elementDetail" id="detail_idp330096" style="display: none;">
<div class="bg"></div>
<div class="fg cat_TK">
<p class="detailClose"><a href="javascript:hideDetail('idp330096');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">SAMtools</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> SAMtools is a set of utilities for efficiently processing nucleotide alignments in the Sequence Alignment/Map format (SAM) and its binary representation (BAM format), and accurately calling the SNPs and short INDELs from the alignment.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2009</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/bioinformatics/btp352" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://samtools.sourceforge.net/">http://samtools.sourceforge.net/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Heng Li,Bob Handsaker, Jue Ruan, John Marshall and Petr Danecek</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/bioinformatics/btp352">10.1093/bioinformatics/btp352</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">BSD licence, MIT Licence</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">ToolKits and APIs</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_SM " onmouseover="javascript:show('idp333600');">
<div class="magnifiable">
<p class="year">99</p>
<br><p class="label linked">Ps</p>
<br><p class="name">PSIPRED</p>
<div class="elementMagnify cat_SM new" id="idp333600" style="display: none;" onmouseout="javascript:hide('idp333600');">
<p class="year">1999</p>
<br><a href="javascript:showDetail('idp333600');"><p class="label linked">Ps</p></a><br><p class="name">PSIPRED</p>
</div>
</div>
<div class="elementDetail" id="detail_idp333600" style="display: none;">
<div class="bg"></div>
<div class="fg cat_SM">
<p class="detailClose"><a href="javascript:hideDetail('idp333600');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">PSIPRED</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> The PSIPRED Protein Sequence Analysis Workbench aggregates several UCL structure prediction methods into one location. Users can submit a protein sequence, perform the predictions of their choice and receive the results of the prediction via e-mail or the web.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">1999</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/nar/gkq427" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://bioinf.cs.ucl.ac.uk/psipred/">http://bioinf.cs.ucl.ac.uk/psipred/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Jones, Buchan, Nugent, Minneci, Bryson et al</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/nar/gkq427">10.1093/nar/gkq427</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Web-based, Linux</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Custom free licence</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Structure Modelling</td>
</tr>
</table>
</div>
</div>
</td>
  </tr>
  <tr>
  <td class="element cat_ALP " onmouseover="javascript:show('idp337952');">
<div class="magnifiable">
<p class="year">03</p>
<br><p class="label linked">Ss</p>
<br><p class="name">SSAHA</p>
<div class="elementMagnify cat_ALP new" id="idp337952" style="display: none;" onmouseout="javascript:hide('idp337952');">
<p class="year">2003</p>
<br><a href="javascript:showDetail('idp337952');"><p class="label linked">Ss</p></a><br><p class="name">SSAHA</p>
</div>
</div>
<div class="elementDetail" id="detail_idp337952" style="display: none;">
<div class="bg"></div>
<div class="fg cat_ALP">
<p class="detailClose"><a href="javascript:hideDetail('idp337952');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">SSAHA</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b>  ssaha is a tool for rapidly finding near exact matches in DNA or protein  databases. The name is an acronym standing for Sequence Search and Alignment by Hashing Algorithm. It works by converting a sequence database into a hash table. This is then rapidly quizzed for hits, which are concatenated into matches.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2003</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1101/gr.194201" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.sanger.ac.uk/resources/software/ssaha/">http://www.sanger.ac.uk/resources/software/ssaha/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Prof. T.W. Lam, Alan Tam, Simon Wong, Edward Wu and S.M. Yiu</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1101/gr.194201">10.1101/gr.194201</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GNU GPL v2</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Aligners (pairwise)</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_ALS " onmouseover="javascript:show('idp337632');">
<div class="magnifiable">
<p class="year">09</p>
<br><p class="label linked">Bw</p>
<br><p class="name">BWA</p>
<div class="elementMagnify cat_ALS new" id="idp337632" style="display: none;" onmouseout="javascript:hide('idp337632');">
<p class="year">2009</p>
<br><a href="javascript:showDetail('idp337632');"><p class="label linked">Bw</p></a><br><p class="name">BWA</p>
</div>
</div>
<div class="elementDetail" id="detail_idp337632" style="display: none;">
<div class="bg"></div>
<div class="fg cat_ALS">
<p class="detailClose"><a href="javascript:hideDetail('idp337632');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">BWA</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Burrows-Wheeler Aligner (BWA) is an efficient program that aligns relatively short nucleotide sequences against a long reference sequence such as the human genome. It implements two algorithms, bwa-short and BWA-SW.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2009</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/bioinformatics/btp324" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://bio-bwa.sourceforge.net/">http://bio-bwa.sourceforge.net/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Heng Li, Chi-Kwong, Nong Ge and Yuta Mori.</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/bioinformatics/btp324">10.1093/bioinformatics/btp324</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight"></td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GPLv3</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Aligners (short read)</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_MSA " onmouseover="javascript:show('idp344368');">
<div class="magnifiable">
<p class="year">00</p>
<br><p class="label linked">Cf</p>
<br><p class="name">T-COFFEE</p>
<div class="elementMagnify cat_MSA new" id="idp344368" style="display: none;" onmouseout="javascript:hide('idp344368');">
<p class="year">2000</p>
<br><a href="javascript:showDetail('idp344368');"><p class="label linked">Cf</p></a><br><p class="name">T-COFFEE</p>
</div>
</div>
<div class="elementDetail" id="detail_idp344368" style="display: none;">
<div class="bg"></div>
<div class="fg cat_MSA">
<p class="detailClose"><a href="javascript:hideDetail('idp344368');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">T-COFFEE</td>
<td class="detailLogo" colspan="2"><img src="http://www.eaglegenomics.com/wp-content/uploads/2012/08/T-coffee.jpg"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> T-Coffee is a multiple sequence alignment package. You can use T-Coffee to align sequences or to combine the output of your favorite alignment methods (Clustal, Mafft, Probcons, Muscle...) into one unique alignment.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2000</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1006/jmbi.2000.4042" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://tcoffee.org/">http://tcoffee.org/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Cedric Notredame, Higgins DG, Heringa J.</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1006/jmbi.2000.4042">10.1006/jmbi.2000.4042</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Unix, Linux, MacOSX, Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GNU GPLv3</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Multiple Sequence Aligners</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_MSA " onmouseover="javascript:show('idp348224');">
<div class="magnifiable">
<p class="year">10</p>
<br><p class="label linked">Pr</p>
<br><p class="name">PRANK</p>
<div class="elementMagnify cat_MSA new" id="idp348224" style="display: none;" onmouseout="javascript:hide('idp348224');">
<p class="year">2010</p>
<br><a href="javascript:showDetail('idp348224');"><p class="label linked">Pr</p></a><br><p class="name">PRANK</p>
</div>
</div>
<div class="elementDetail" id="detail_idp348224" style="display: none;">
<div class="bg"></div>
<div class="fg cat_MSA">
<p class="detailClose"><a href="javascript:hideDetail('idp348224');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">PRANK</td>
<td class="detailLogo" colspan="2"><img src="http://www.eaglegenomics.com/wp-content/uploads/2012/08/webprank_logo.png"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> PRANK is a probabilistic multiple alignment program for DNA, codon and amino-acid sequences that has been shown to produce exceptionally accurate alignments for evolutionary analyses. It is based on a novel algorithm that treats insertions correctly and avoids over-estimation of the number of deletion events. </td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2010</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1186/1471-2105-11-579" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.ebi.ac.uk/goldman-srv/prank/">http://www.ebi.ac.uk/goldman-srv/prank/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Ari Löytynoja</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1186/1471-2105-11-579">10.1186/1471-2105-11-579</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight"></td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GPL</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Multiple Sequence Aligners</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_AGL " onmouseover="javascript:show('idp351808');">
<div class="magnifiable">
<p class="year">97</p>
<br><p class="label linked">Ce</p>
<br><p class="name">CELERA</p>
<div class="elementMagnify cat_AGL new" id="idp351808" style="display: none;" onmouseout="javascript:hide('idp351808');">
<p class="year">1997</p>
<br><a href="javascript:showDetail('idp351808');"><p class="label linked">Ce</p></a><br><p class="name">CELERA</p>
</div>
</div>
<div class="elementDetail" id="detail_idp351808" style="display: none;">
<div class="bg"></div>
<div class="fg cat_AGL">
<p class="detailClose"><a href="javascript:hideDetail('idp351808');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">CELERA</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Celera Assembler is a de novo whole-genome shotgun (WGS) DNA sequence assembler.Celera Assembler is a de novo whole-genome shotgun (WGS) DNA sequence assembler.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">1997</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1126/science.287.5461.2196" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://sourceforge.net/apps/mediawiki/wgs-assembler/index.php?title=Main_Page">http://sourceforge.net/apps/mediawiki/wgs-assembler/index.php?title=Main_Page</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Celera Gomes</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1126/science.287.5461.2196">10.1126/science.287.5461.2196</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, Mac OS X, Unix</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GNU GPL/ BSD</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Assemblers Genomic (long read)</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_AGL " onmouseover="javascript:show('idp355168');">
<div class="magnifiable">
<p class="year">09</p>
<br><p class="label linked">Fo</p>
<br><p class="name">FORGE</p>
<div class="elementMagnify cat_AGL new" id="idp355168" style="display: none;" onmouseout="javascript:hide('idp355168');">
<p class="year">2009</p>
<br><a href="javascript:showDetail('idp355168');"><p class="label linked">Fo</p></a><br><p class="name">FORGE</p>
</div>
</div>
<div class="elementDetail" id="detail_idp355168" style="display: none;">
<div class="bg"></div>
<div class="fg cat_AGL">
<p class="detailClose"><a href="javascript:hideDetail('idp355168');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">FORGE</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Forge is a classic Overlap layout consensus genome assembler. Implemented in C++ and using the parallel MPI library, it runs on one or more machines in a network and can scale to very large numbers of reads provided there is enough collective memory on the machines used.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2009</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1186/gb-2009-10-9-r94" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://combiol.org/forge/">http://combiol.org/forge/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Darren Platt and Dirk Evers</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1186/gb-2009-10-9-r94">10.1186/gb-2009-10-9-r94</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Unix</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Licenced under Apache 2.0 licence</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Assemblers Genomic (long read)</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_SV new" onmouseover="javascript:show('idp358752');">
<div class="magnifiable">
<p class="year">95</p>
<br><p class="label linked">Rm</p>
<br><p class="name">RasMol</p>
<div class="elementMagnify cat_SV new" id="idp358752" style="display: none;" onmouseout="javascript:hide('idp358752');">
<p class="year">1995</p>
<br><a href="javascript:showDetail('idp358752');"><p class="label linked">Rm</p></a><br><p class="name">RasMol</p>
</div>
</div>
<div class="elementDetail" id="detail_idp358752" style="display: none;">
<div class="bg"></div>
<div class="fg cat_SV">
<p class="detailClose"><a href="javascript:hideDetail('idp358752');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">RasMol</td>
<td class="detailLogo" colspan="2"><img src="http://rasmol.org/html_graphics/rasmollogo.jpg"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> RasMol is an important scientific tool for visualisation of molecules created by Roger Sayle in 1992. RasMol is used by hundreds of thousands of users world-wide to view macromolecules and to prepare publication-quality images.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">1995</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1016/S0968-0004(00)89080-5" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://rasmol.org/">http://rasmol.org/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">R. A. Sayle and E. J. Milner-White</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1016/S0968-0004(00)89080-5">10.1016/S0968-0004(00)89080-5</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Universal</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GPL</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Structure Visualisation</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_AGS " onmouseover="javascript:show('idp362432');">
<div class="magnifiable">
<p class="year">09</p>
<br><p class="label linked">Sg</p>
<br><p class="name">SGA</p>
<div class="elementMagnify cat_AGS new" id="idp362432" style="display: none;" onmouseout="javascript:hide('idp362432');">
<p class="year">2009</p>
<br><a href="javascript:showDetail('idp362432');"><p class="label linked">Sg</p></a><br><p class="name">SGA</p>
</div>
</div>
<div class="elementDetail" id="detail_idp362432" style="display: none;">
<div class="bg"></div>
<div class="fg cat_AGS">
<p class="detailClose"><a href="javascript:hideDetail('idp362432');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">SGA</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> SGA is a de novo genome assembler based on the concept of string graphs. The major goal of SGA is to be very memory efficient, which is achieved by using a compressed representation of DNA sequence reads.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2009</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1101/gr.126953.111" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="https://github.com/jts/sga">https://github.com/jts/sga</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Jared Simpson and Richard Durbin</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1101/gr.126953.111">10.1101/gr.126953.111</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GNU GPL v3</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Assemblers Genomic (short read)</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_ASM " onmouseover="javascript:show('idp365856');">
<div class="magnifiable">
<p class="year">09</p>
<br><p class="label linked">Cu</p>
<br><p class="name">CUFFLINKS</p>
<div class="elementMagnify cat_ASM new" id="idp365856" style="display: none;" onmouseout="javascript:hide('idp365856');">
<p class="year">2009</p>
<br><a href="javascript:showDetail('idp365856');"><p class="label linked">Cu</p></a><br><p class="name">CUFFLINKS</p>
</div>
</div>
<div class="elementDetail" id="detail_idp365856" style="display: none;">
<div class="bg"></div>
<div class="fg cat_ASM">
<p class="detailClose"><a href="javascript:hideDetail('idp365856');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">CUFFLINKS</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Cufflinks assembles transcripts, estimates their abundances, and tests for differential expression and regulation in RNA-Seq samples. It accepts aligned RNA-Seq reads and assembles the alignments into a parsimonious set of transcripts. Cufflinks then estimates the relative abundances of these transcripts based on how many reads support each one, taking into account biases in library preparation protocols.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2009</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1038/nbt.1621" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://cufflinks.cbcb.umd.edu/">http://cufflinks.cbcb.umd.edu/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Cole Trapnell, Adam Roberts, Ali Mortazavi, Steven Salzberg, Barabara Wold, Lior Pachter etc.</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1038/nbt.1621">10.1038/nbt.1621</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, Mac OS X</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">OSI-approved Boost Licence</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Assemblers (mRNA)</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_GPM " onmouseover="javascript:show('idp370064');">
<div class="magnifiable">
<p class="year">99</p>
<br><p class="label linked">Eu</p>
<br><p class="name">EUGENE</p>
<div class="elementMagnify cat_GPM new" id="idp370064" style="display: none;" onmouseout="javascript:hide('idp370064');">
<p class="year">1999</p>
<br><a href="javascript:showDetail('idp370064');"><p class="label linked">Eu</p></a><br><p class="name">EUGENE</p>
</div>
</div>
<div class="elementDetail" id="detail_idp370064" style="display: none;">
<div class="bg"></div>
<div class="fg cat_GPM">
<p class="detailClose"><a href="javascript:hideDetail('idp370064');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">EUGENE</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> An open gene finder for eukaryotic organisms. Compared to most existing gene finders, EuGène is characterized by its ability to simply integrate arbitrary sources of information in its prediction process. </td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">1999</td>
<td rowspan="7">No impact data available.</td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://eugene.toulouse.inra.fr/">http://eugene.toulouse.inra.fr/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">P. Bardou, MJ. Cros, S. Foissac, J Gouzy, A. and Moisan, T. Schiex</td>
</tr>
<tr>
<td class="detailLeft">Key publication:</td>
<td class="detailRight"><a target="_blank" href="http://www.inra.fr/mia/T/schiex/Export/LNCS-EuGene.pdf">http://www.inra.fr/mia/T/schiex/Export/LNCS-EuGene.pdf</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Artistic licence</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Gene Prediction (mRNA)</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_ST " onmouseover="javascript:show('idp369744');">
<div class="magnifiable">
<p class="year">09</p>
<br><p class="label linked">Fx</p>
<br><p class="name">FASTX-Toolkit</p>
<div class="elementMagnify cat_ST new" id="idp369744" style="display: none;" onmouseout="javascript:hide('idp369744');">
<p class="year">2009</p>
<br><a href="javascript:showDetail('idp369744');"><p class="label linked">Fx</p></a><br><p class="name">FASTX-Toolkit</p>
</div>
</div>
<div class="elementDetail" id="detail_idp369744" style="display: none;">
<div class="bg"></div>
<div class="fg cat_ST">
<p class="detailClose"><a href="javascript:hideDetail('idp369744');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">FASTX-Toolkit</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> The FASTX-Toolkit is a collection of command line tools for Short-Reads FASTA/FASTQ files preprocessing.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2009</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1006/geno.1997.4995" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://hannonlab.cshl.edu/fastx_toolkit/">http://hannonlab.cshl.edu/fastx_toolkit/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Assaf Gordon</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1006/geno.1997.4995">10.1006/geno.1997.4995</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, Unix</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GNU Affero</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Sequence Tools</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_GPN " onmouseover="javascript:show('idp376208');">
<div class="magnifiable">
<p class="year">10</p>
<br><p class="label linked">Rn</p>
<br><p class="name">RNAfold</p>
<div class="elementMagnify cat_GPN new" id="idp376208" style="display: none;" onmouseout="javascript:hide('idp376208');">
<p class="year">2010</p>
<br><a href="javascript:showDetail('idp376208');"><p class="label linked">Rn</p></a><br><p class="name">RNAfold</p>
</div>
</div>
<div class="elementDetail" id="detail_idp376208" style="display: none;">
<div class="bg"></div>
<div class="fg cat_GPN">
<p class="detailClose"><a href="javascript:hideDetail('idp376208');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">RNAfold</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> The RNAfold web server will predict secondary structures of single stranded RNA or DNA sequences.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2010</td>
<td rowspan="7">No impact data available.</td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://rna.tbi.univie.ac.at/cgi-bin/RNAfold.cgi">http://rna.tbi.univie.ac.at/cgi-bin/RNAfold.cgi</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">M Zuker and P Stiegler</td>
</tr>
<tr>
<td class="detailLeft">Key publication:</td>
<td class="detailRight"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC326673/">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC326673/</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, Mac OS X and Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Open Source</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Gene Prediction (ncRNA)</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_DB " onmouseover="javascript:show('idp379296');">
<div class="magnifiable">
<p class="year">07</p>
<br><p class="label linked">In</p>
<br><p class="name">INTERMINE</p>
<div class="elementMagnify cat_DB new" id="idp379296" style="display: none;" onmouseout="javascript:hide('idp379296');">
<p class="year">2007</p>
<br><a href="javascript:showDetail('idp379296');"><p class="label linked">In</p></a><br><p class="name">INTERMINE</p>
</div>
</div>
<div class="elementDetail" id="detail_idp379296" style="display: none;">
<div class="bg"></div>
<div class="fg cat_DB">
<p class="detailClose"><a href="javascript:hideDetail('idp379296');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">INTERMINE</td>
<td class="detailLogo" colspan="2"><img src="http://www.eaglegenomics.com/wp-content/uploads/2012/09/InterMine.jpg"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> InterMine is a powerful open source data warehouse system. Using InterMine, you can create databases of biological data accessed by sophisticated web query tools. InterMine includes an attractive, user-friendly web interface that works 'out of the box' and can be easily customised for your specific needs.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2007</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/database/bar062" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://intermine.org/wiki/InterMineOverview?redirectedfrom=InterMine">http://intermine.org/wiki/InterMineOverview?redirectedfrom=InterMine</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Richard Smith, Jakub Kulaviak, Julie Sullivan, Mather Wakeling, Xavier Watkins</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/database/bar062">10.1093/database/bar062</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GNU Lesser General Public Licence</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Database/Warehouse</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_WF " onmouseover="javascript:show('idp383280');">
<div class="magnifiable">
<p class="year">07</p>
<br><p class="label linked">Gx</p>
<br><p class="name">GALAXY</p>
<div class="elementMagnify cat_WF new" id="idp383280" style="display: none;" onmouseout="javascript:hide('idp383280');">
<p class="year">2007</p>
<br><a href="javascript:showDetail('idp383280');"><p class="label linked">Gx</p></a><br><p class="name">GALAXY</p>
</div>
</div>
<div class="elementDetail" id="detail_idp383280" style="display: none;">
<div class="bg"></div>
<div class="fg cat_WF">
<p class="detailClose"><a href="javascript:hideDetail('idp383280');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">GALAXY</td>
<td class="detailLogo" colspan="2"><img src="http://www.eaglegenomics.com/wp-content/uploads/2012/09/galaxyLogoTrimmed.png"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Galaxy is a scientific workflow and data integration platform that aims to make computational biology accessible to research scientists that do not have computer programming experience.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2007</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1186/gb-2010-11-8-r86" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://wiki.g2.bx.psu.edu/FrontPage">http://wiki.g2.bx.psu.edu/FrontPage</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Enis Afgan, Guru Ananda and Dannon Baker</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1186/gb-2010-11-8-r86">10.1186/gb-2010-11-8-r86</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Unix</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GNU</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Workflows</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_GB " onmouseover="javascript:show('idp387072');">
<div class="magnifiable">
<p class="year">02</p>
<br><p class="label linked">Ap</p>
<br><p class="name">APOLLO</p>
<div class="elementMagnify cat_GB new" id="idp387072" style="display: none;" onmouseout="javascript:hide('idp387072');">
<p class="year">2002</p>
<br><a href="javascript:showDetail('idp387072');"><p class="label linked">Ap</p></a><br><p class="name">APOLLO</p>
</div>
</div>
<div class="elementDetail" id="detail_idp387072" style="display: none;">
<div class="bg"></div>
<div class="fg cat_GB">
<p class="detailClose"><a href="javascript:hideDetail('idp387072');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">APOLLO</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Apollo is a genome annotation viewer and editor. It was developed as a collaboration between the Berkeley Drosophila Genome Project (part of the FlyBase consortium) and The Sanger Institute in Cambridge, UK. Apollo allows researchers to explore genomic annotations at many levels of detail, and to perform expert annotation curation, all in a graphical environment.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2002</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1186/gb-2002-3-12-research0082" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://apollo.berkeleybop.org/current/index.html">http://apollo.berkeleybop.org/current/index.html</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Ed Lee, Nomi Harris, Steve Searle, Michele Clamp, Suzanna Lewis, John Richter and Mark Gibson</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1186/gb-2002-3-12-research0082">10.1186/gb-2002-3-12-research0082</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, MacOsX, Unix and Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Artistic Licence</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Genome Browsers</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_GB " onmouseover="javascript:show('idp390800');">
<div class="magnifiable">
<p class="year">04</p>
<br><p class="label linked">Ig</p>
<br><p class="name">IGB</p>
<div class="elementMagnify cat_GB new" id="idp390800" style="display: none;" onmouseout="javascript:hide('idp390800');">
<p class="year">2004</p>
<br><a href="javascript:showDetail('idp390800');"><p class="label linked">Ig</p></a><br><p class="name">IGB</p>
</div>
</div>
<div class="elementDetail" id="detail_idp390800" style="display: none;">
<div class="bg"></div>
<div class="fg cat_GB">
<p class="detailClose"><a href="javascript:hideDetail('idp390800');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">IGB</td>
<td class="detailLogo" colspan="2"><img src="http://www.eaglegenomics.com/wp-content/uploads/2012/09/igbman.gif"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> The Integrated Genome Browser (IGB, pronounced Ig-Bee) is an interactive, zoomable, scrollable software program you can use to visualize and explore genome-scale data sets.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2004</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/bioinformatics/btp472" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://bioviz.org/igb/">http://bioviz.org/igb/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Gregg Helt</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/bioinformatics/btp472">10.1093/bioinformatics/btp472</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, MacOSX, Unix and Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Common Public Licence v 1.0</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Genome Browsers</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_TK " onmouseover="javascript:show('idp394912');">
<div class="magnifiable">
<p class="year">07</p>
<br><p class="label linked">Ut</p>
<br><p class="name">UTGB</p>
<div class="elementMagnify cat_TK new" id="idp394912" style="display: none;" onmouseout="javascript:hide('idp394912');">
<p class="year">2007</p>
<br><a href="javascript:showDetail('idp394912');"><p class="label linked">Ut</p></a><br><p class="name">UTGB</p>
</div>
</div>
<div class="elementDetail" id="detail_idp394912" style="display: none;">
<div class="bg"></div>
<div class="fg cat_TK">
<p class="detailClose"><a href="javascript:hideDetail('idp394912');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">UTGB</td>
<td class="detailLogo" colspan="2"><img src="http://www.eaglegenomics.com/wp-content/uploads/2012/09/utgb-title-small.gif"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> UTGB Toolkit is an open-source software for developing personalized genome browsers.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2007</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/bioinformatics/btp350" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://utgenome.org/">http://utgenome.org/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Shinichi Morishita, Taro Saito, Jun Yoshimura, Koichiro Higasa, Hiroshi Minoshima, Reginaldo Kuroshu and Atsushi Sasaki.</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/bioinformatics/btp350">10.1093/bioinformatics/btp350</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, Mac OS X and Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Apache Licence 2.0</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">ToolKits and APIs</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_TK " onmouseover="javascript:show('idp398496');">
<div class="magnifiable">
<p class="year">10</p>
<br><p class="label linked">Am</p>
<br><p class="name">AMPLICONNOISE</p>
<div class="elementMagnify cat_TK new" id="idp398496" style="display: none;" onmouseout="javascript:hide('idp398496');">
<p class="year">2010</p>
<br><a href="javascript:showDetail('idp398496');"><p class="label linked">Am</p></a><br><p class="name">AMPLICONNOISE</p>
</div>
</div>
<div class="elementDetail" id="detail_idp398496" style="display: none;">
<div class="bg"></div>
<div class="fg cat_TK">
<p class="detailClose"><a href="javascript:hideDetail('idp398496');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">AMPLICONNOISE</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> AmpliconNoise is a collection of programs for the removal of noise from 454 sequenced PCR amplicons. It involves two steps the removal of noise from the sequencing itself and the removal of PCR point errors. This project also includes the Perseus algorithm for chimera removal.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2010</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1186/1471-2105-12-38" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://code.google.com/p/ampliconnoise/">http://code.google.com/p/ampliconnoise/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Chrisopher Quince, Andres Lanzen, Russell J Davenport</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1186/1471-2105-12-38">10.1186/1471-2105-12-38</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux and Mac OS X</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GNU Lesser GPL</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">ToolKits and APIs</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_TK " onmouseover="javascript:show('idp402400');">
<div class="magnifiable">
<p class="year">09</p>
<br><p class="label linked">Qi</p>
<br><p class="name">QIIME</p>
<div class="elementMagnify cat_TK new" id="idp402400" style="display: none;" onmouseout="javascript:hide('idp402400');">
<p class="year">2009</p>
<br><a href="javascript:showDetail('idp402400');"><p class="label linked">Qi</p></a><br><p class="name">QIIME</p>
</div>
</div>
<div class="elementDetail" id="detail_idp402400" style="display: none;">
<div class="bg"></div>
<div class="fg cat_TK">
<p class="detailClose"><a href="javascript:hideDetail('idp402400');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">QIIME</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> QIIME (canonically pronounced &#8216;Chime&#8217;) is a pipeline for performing microbial community analysis that integrates many third party tools which have become standard in the field. QIIME can run on a laptop, a supercomputer, and systems in between such as multicore desktops.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2009</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1038/nmeth.f.303" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.qiime.org/">http://www.qiime.org/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Developed by Knight Lab at the University of Colorado at Boulder</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1038/nmeth.f.303">10.1038/nmeth.f.303</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, MacOSX and Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight"></td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">ToolKits and APIs</td>
</tr>
</table>
</div>
</div>
</td>
  </tr>
  <tr>
  <td class="element cat_ALP " onmouseover="javascript:show('idp405696');">
<div class="magnifiable">
<p class="year">05</p>
<br><p class="label linked">Ex</p>
<br><p class="name">EXONERATE</p>
<div class="elementMagnify cat_ALP new" id="idp405696" style="display: none;" onmouseout="javascript:hide('idp405696');">
<p class="year">2005</p>
<br><a href="javascript:showDetail('idp405696');"><p class="label linked">Ex</p></a><br><p class="name">EXONERATE</p>
</div>
</div>
<div class="elementDetail" id="detail_idp405696" style="display: none;">
<div class="bg"></div>
<div class="fg cat_ALP">
<p class="detailClose"><a href="javascript:hideDetail('idp405696');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">EXONERATE</td>
<td class="detailLogo" colspan="2"><img src="http://www.eaglegenomics.com/wp-content/uploads/2012/08/exonerate_logo.png"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> exonerate is a general tool for sequence comparison. It uses the C4 dynamic programming library. It is designed to be both general and fast. It can produce either gapped or ungapped alignments, according to a variety of different alignment models.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2005</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1186/1471-2105-6-31" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.ebi.ac.uk/~guy/exonerate/">http://www.ebi.ac.uk/~guy/exonerate/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Guy St.C. Slater</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1186/1471-2105-6-31">10.1186/1471-2105-6-31</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">UNIX, Linux</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GNU GPL 2005</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Aligners (pairwise)</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_ALS " onmouseover="javascript:show('idp409392');">
<div class="magnifiable">
<p class="year">09</p>
<br><p class="label linked">Bf</p>
<br><p class="name">BFAST</p>
<div class="elementMagnify cat_ALS new" id="idp409392" style="display: none;" onmouseout="javascript:hide('idp409392');">
<p class="year">2009</p>
<br><a href="javascript:showDetail('idp409392');"><p class="label linked">Bf</p></a><br><p class="name">BFAST</p>
</div>
</div>
<div class="elementDetail" id="detail_idp409392" style="display: none;">
<div class="bg"></div>
<div class="fg cat_ALS">
<p class="detailClose"><a href="javascript:hideDetail('idp409392');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">BFAST</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> BFAST (Blat-like Fast Accurate Search Tool) facilitates the fast and accurate mapping of short reads to reference sequences. BFAST was designed to facilitate whole-genome resequencing, where mapping billions of short reads with variants is of utmost importance. BFAST supports both Illumina and ABI SOLiD data, as well as any other Next-Generation Sequencing Technology (454, Helicos), with particular emphasis on sensitivity towards errors, SNPs and especially indels.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2009</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1371/journal.pone.0007767" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://bfast.sf.net/">http://bfast.sf.net/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Nils Homer</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1371/journal.pone.0007767">10.1371/journal.pone.0007767</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Unix, Linux, MacOSX</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GPL v2</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Aligners (short read)</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_MSA " onmouseover="javascript:show('idp413120');">
<div class="magnifiable">
<p class="year">02</p>
<br><p class="label linked">Ma</p>
<br><p class="name">MAFFT</p>
<div class="elementMagnify cat_MSA new" id="idp413120" style="display: none;" onmouseout="javascript:hide('idp413120');">
<p class="year">2002</p>
<br><a href="javascript:showDetail('idp413120');"><p class="label linked">Ma</p></a><br><p class="name">MAFFT</p>
</div>
</div>
<div class="elementDetail" id="detail_idp413120" style="display: none;">
<div class="bg"></div>
<div class="fg cat_MSA">
<p class="detailClose"><a href="javascript:hideDetail('idp413120');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">MAFFT</td>
<td class="detailLogo" colspan="2"><img src="http://www.eaglegenomics.com/wp-content/uploads/2012/08/Mafft.jpg"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> MAFFT is a multiple sequence alignment program for amino acid or nucleotide sequences.  It has several different options for various types of alignment problems.  Progressive alignment options can be applied to a large number of homologous sequences.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2002</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/bioinformatics/btq224" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://mafft.cbrc.jp/alignment/server/">http://mafft.cbrc.jp/alignment/server/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Kazutaka Katoh, Charles Plessy</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/bioinformatics/btq224">10.1093/bioinformatics/btq224</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, MacOSX, Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GPL and BSD</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Multiple Sequence Aligners</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_MSA " onmouseover="javascript:show('idp416928');">
<div class="magnifiable">
<p class="year">04</p>
<br><p class="label linked">Mv</p>
<br><p class="name">MAVID</p>
<div class="elementMagnify cat_MSA new" id="idp416928" style="display: none;" onmouseout="javascript:hide('idp416928');">
<p class="year">2004</p>
<br><a href="javascript:showDetail('idp416928');"><p class="label linked">Mv</p></a><br><p class="name">MAVID</p>
</div>
</div>
<div class="elementDetail" id="detail_idp416928" style="display: none;">
<div class="bg"></div>
<div class="fg cat_MSA">
<p class="detailClose"><a href="javascript:hideDetail('idp416928');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">MAVID</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> MAVID is a multiple sequence alignment program suitable for the alignment of large numbers of DNA sequences. The sequences can be small mitochondrial genomes or large genomic regions up to megabases long.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2004</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1101/gr.1960404" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://bio.math.berkeley.edu/mavid/download/">http://bio.math.berkeley.edu/mavid/download/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Nicolas Bray, Lior Pachter</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1101/gr.1960404">10.1101/gr.1960404</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Unix, Linux, MacOSX</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Open Source</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Multiple Sequence Aligners</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_AGL " onmouseover="javascript:show('idp420352');">
<div class="magnifiable">
<p class="year">02</p>
<br><p class="label linked">Ar</p>
<br><p class="name">ARACHNE</p>
<div class="elementMagnify cat_AGL new" id="idp420352" style="display: none;" onmouseout="javascript:hide('idp420352');">
<p class="year">2002</p>
<br><a href="javascript:showDetail('idp420352');"><p class="label linked">Ar</p></a><br><p class="name">ARACHNE</p>
</div>
</div>
<div class="elementDetail" id="detail_idp420352" style="display: none;">
<div class="bg"></div>
<div class="fg cat_AGL">
<p class="detailClose"><a href="javascript:hideDetail('idp420352');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">ARACHNE</td>
<td class="detailLogo" colspan="2"><img src="http://www.eaglegenomics.com/wp-content/uploads/2012/08/arachne.jpg"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> ARACHNE is a program for assembling data from whole genome shotgun sequencing experiments. It was designed for long reads from Sanger sequencing technology, and has been used extensively to assemble many genomes, including many that are large and highly repetitive.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2002</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1101/gr.208902" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.broadinstitute.org/crd/wiki/index.php/Arachne_Main_Page">http://www.broadinstitute.org/crd/wiki/index.php/Arachne_Main_Page</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Jaffe DB</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1101/gr.208902">10.1101/gr.208902</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight"></td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Assemblers Genomic (long read)</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_AGL " onmouseover="javascript:show('idp423888');">
<div class="magnifiable">
<p class="year">10</p>
<br><p class="label linked">Me</p>
<br><p class="name">MERACULOUS</p>
<div class="elementMagnify cat_AGL new" id="idp423888" style="display: none;" onmouseout="javascript:hide('idp423888');">
<p class="year">2010</p>
<br><a href="javascript:showDetail('idp423888');"><p class="label linked">Me</p></a><br><p class="name">MERACULOUS</p>
</div>
</div>
<div class="elementDetail" id="detail_idp423888" style="display: none;">
<div class="bg"></div>
<div class="fg cat_AGL">
<p class="detailClose"><a href="javascript:hideDetail('idp423888');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">MERACULOUS</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Meraculous relies on an efficient and conservative traversal of the subgraph of the k-mer (deBruijn) graph of oligonucleotides with unique high quality extensions in the dataset, avoiding an explicit error correction step as used in other short-read assemblers. </td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2010</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1371/journal.pone.0023501" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="ftp://ftp.jgi-psf.org/pub/JGI_data/meraculous/Meraculous-1.4.4-rel.tar">ftp://ftp.jgi-psf.org/pub/JGI_data/meraculous/Meraculous-1.4.4-rel.tar</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Chapman JA, Ho I, Sunkara S, Luo S, Schroth GP</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1371/journal.pone.0023501">10.1371/journal.pone.0023501</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Unix</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GNU GPL</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Assemblers Genomic (long read)</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_AGS " onmouseover="javascript:show('idp427440');">
<div class="magnifiable">
<p class="year">08</p>
<br><p class="label linked">Co</p>
<br><p class="name">CORTEX_CON_RP</p>
<div class="elementMagnify cat_AGS new" id="idp427440" style="display: none;" onmouseout="javascript:hide('idp427440');">
<p class="year">2008</p>
<br><a href="javascript:showDetail('idp427440');"><p class="label linked">Co</p></a><br><p class="name">CORTEX_CON_RP</p>
</div>
</div>
<div class="elementDetail" id="detail_idp427440" style="display: none;">
<div class="bg"></div>
<div class="fg cat_AGS">
<p class="detailClose"><a href="javascript:hideDetail('idp427440');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">CORTEX_CON_RP</td>
<td class="detailLogo" colspan="2"><img src="http://www.eaglegenomics.com/wp-content/uploads/2012/08/Cortex.jpg"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Cortex is an efficient and low-memory software framework for analysis of genomes using sequence data.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2008</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1038/ng.1028" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://cortexassembler.sourceforge.net/index.html">http://cortexassembler.sourceforge.net/index.html</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Mario Caccamo and Zamin Iqbal</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1038/ng.1028">10.1038/ng.1028</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, Mac OS X</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GPLv3</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Assemblers Genomic (short read)</td>
</tr>
</table>
</div>
</div>
</td>
 <td class="element cat_AGS " onmouseover="javascript:show('idp431136');">
<div class="magnifiable">
<p class="year">10</p>
<br><p class="label linked">Pa</p>
<br><p class="name">PE-ASSEMBLER</p>
<div class="elementMagnify cat_AGS new" id="idp431136" style="display: none;" onmouseout="javascript:hide('idp431136');">
<p class="year">2010</p>
<br><a href="javascript:showDetail('idp431136');"><p class="label linked">Pa</p></a><br><p class="name">PE-ASSEMBLER</p>
</div>
</div>
<div class="elementDetail" id="detail_idp431136" style="display: none;">
<div class="bg"></div>
<div class="fg cat_AGS">
<p class="detailClose"><a href="javascript:hideDetail('idp431136');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">PE-ASSEMBLER</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> A simple  extension approach to assembling paired-end reads and capable of parallelization.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2010</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/bioinformatics/btq626" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.comp.nus.edu.sg/~bioinfo/peasm/PE_manual.htm">http://www.comp.nus.edu.sg/~bioinfo/peasm/PE_manual.htm</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Nuwantha Ariaratne P., Sung WK.</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/bioinformatics/btq626">10.1093/bioinformatics/btq626</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Open Source</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Assemblers Genomic (short read)</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_ASM " onmouseover="javascript:show('idp434512');">
<div class="magnifiable">
<p class="year">10</p>
<br><p class="label linked">Oa</p>
<br><p class="name">OASES</p>
<div class="elementMagnify cat_ASM new" id="idp434512" style="display: none;" onmouseout="javascript:hide('idp434512');">
<p class="year">2010</p>
<br><a href="javascript:showDetail('idp434512');"><p class="label linked">Oa</p></a><br><p class="name">OASES</p>
</div>
</div>
<div class="elementDetail" id="detail_idp434512" style="display: none;">
<div class="bg"></div>
<div class="fg cat_ASM">
<p class="detailClose"><a href="javascript:hideDetail('idp434512');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">OASES</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Oases is a de novo transcriptome assembler designed to produce transcripts from short read sequencing technologies, such as Illumina, SOLiD, or 454 in the absence of any genomic assembly.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2010</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/bioinformatics/bts094" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.ebi.ac.uk/~zerbino/oases/">http://www.ebi.ac.uk/~zerbino/oases/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Marcel Schulz and Daniel Zerbino</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/bioinformatics/bts094">10.1093/bioinformatics/bts094</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GPL</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Assemblers (mRNA)</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_GPM " onmouseover="javascript:show('idp437984');">
<div class="magnifiable">
<p class="year">00</p>
<br><p class="label linked">Ep</p>
<br><p class="name">ENSEMBL PIPELINE</p>
<div class="elementMagnify cat_GPM new" id="idp437984" style="display: none;" onmouseout="javascript:hide('idp437984');">
<p class="year">2000</p>
<br><a href="javascript:showDetail('idp437984');"><p class="label linked">Ep</p></a><br><p class="name">ENSEMBL PIPELINE</p>
</div>
</div>
<div class="elementDetail" id="detail_idp437984" style="display: none;">
<div class="bg"></div>
<div class="fg cat_GPM">
<p class="detailClose"><a href="javascript:hideDetail('idp437984');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">ENSEMBL PIPELINE</td>
<td class="detailLogo" colspan="2"><img src="http://www.eaglegenomics.com/wp-content/uploads/2012/09/e-ensembl.png"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Ensembl is a joint project between EMBL - EBI and the Wellcome Trust Sanger Institute to develop a software system which produces and maintains automatic annotation on selected eukaryotic genomes.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2000</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/nar/gkq1064" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.ensembl.org/info/docs/Doxygen/pipeline-api/files.html">http://www.ensembl.org/info/docs/Doxygen/pipeline-api/files.html</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">EBI and Wellcome Trust Sanger institute.</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/nar/gkq1064">10.1093/nar/gkq1064</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, Mac OS X, Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Apache-style licence.</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Gene Prediction (mRNA)</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_ST " onmouseover="javascript:show('idp441808');">
<div class="magnifiable">
<p class="year">10</p>
<br><p class="label linked">Fq</p>
<br><p class="name">FastQC</p>
<div class="elementMagnify cat_ST new" id="idp441808" style="display: none;" onmouseout="javascript:hide('idp441808');">
<p class="year">2010</p>
<br><a href="javascript:showDetail('idp441808');"><p class="label linked">Fq</p></a><br><p class="name">FastQC</p>
</div>
</div>
<div class="elementDetail" id="detail_idp441808" style="display: none;">
<div class="bg"></div>
<div class="fg cat_ST">
<p class="detailClose"><a href="javascript:hideDetail('idp441808');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">FastQC</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> A quality control tool for high throughput sequence data.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2010</td>
<td rowspan="7">No impact data available.</td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/">http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Simon Andrews</td>
</tr>
<tr>
<td class="detailLeft">Key publication:</td>
<td class="detailRight"><a target="_blank" href="index.html"></a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, Mac OS X, Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GPL v3 or later</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Sequence Tools</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_ST new" onmouseover="javascript:show('idp444688');">
<div class="magnifiable">
<p class="year">97</p>
<br><p class="label linked">Si</p>
<br><p class="name">SignalP</p>
<div class="elementMagnify cat_ST new" id="idp444688" style="display: none;" onmouseout="javascript:hide('idp444688');">
<p class="year">1997</p>
<br><a href="javascript:showDetail('idp444688');"><p class="label linked">Si</p></a><br><p class="name">SignalP</p>
</div>
</div>
<div class="elementDetail" id="detail_idp444688" style="display: none;">
<div class="bg"></div>
<div class="fg cat_ST">
<p class="detailClose"><a href="javascript:hideDetail('idp444688');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">SignalP</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> SignalP server predicts the presence and location of signal peptide cleavage sites in amino acid sequences from different organisms: Gram-positive prokaryotes, Gram-negative prokaryotes, and eukaryotes. The method incorporates a prediction of cleavage sites and a signal peptide/non-signal peptide prediction based on a combination of several artificial neural networks.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">1997</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/protein/10.1.1" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.cbs.dtu.dk/services/SignalP/">http://www.cbs.dtu.dk/services/SignalP/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Henrik Nielsen, Jacob Engelbrecht, Søren Brunak and Gunnar von Heijne</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/protein/10.1.1">10.1093/protein/10.1.1</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Web, Max, Unix</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Web unrestricted, Download free for academics</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Sequence Tools</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_DB " onmouseover="javascript:show('idp448256');">
<div class="magnifiable">
<p class="year">10</p>
<br><p class="label linked">Ch</p>
<br><p class="name">CHADO</p>
<div class="elementMagnify cat_DB new" id="idp448256" style="display: none;" onmouseout="javascript:hide('idp448256');">
<p class="year">2010</p>
<br><a href="javascript:showDetail('idp448256');"><p class="label linked">Ch</p></a><br><p class="name">CHADO</p>
</div>
</div>
<div class="elementDetail" id="detail_idp448256" style="display: none;">
<div class="bg"></div>
<div class="fg cat_DB">
<p class="detailClose"><a href="javascript:hideDetail('idp448256');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">CHADO</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Chado is a relational database schema that underlies many GMOD installations. The Chado schema has been designed with modularity and compartmentalization of function in mind.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2010</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/database/bar051" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://gmod.org/wiki/Chado">http://gmod.org/wiki/Chado</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Chris Mungall and Dave Emmert</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/database/bar051">10.1093/database/bar051</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Unix, Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Artistic Licence</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Database/Warehouse</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_WF " onmouseover="javascript:show('idp451536');">
<div class="magnifiable">
<p class="year">08</p>
<br><p class="label linked">Kn</p>
<br><p class="name">KNIME</p>
<div class="elementMagnify cat_WF new" id="idp451536" style="display: none;" onmouseout="javascript:hide('idp451536');">
<p class="year">2008</p>
<br><a href="javascript:showDetail('idp451536');"><p class="label linked">Kn</p></a><br><p class="name">KNIME</p>
</div>
</div>
<div class="elementDetail" id="detail_idp451536" style="display: none;">
<div class="bg"></div>
<div class="fg cat_WF">
<p class="detailClose"><a href="javascript:hideDetail('idp451536');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">KNIME</td>
<td class="detailLogo" colspan="2"><img src="http://www.eaglegenomics.com/wp-content/uploads/2012/09/Knime.jpg"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> KNIME (Konstanz Information Miner) is a user-friendly and comprehensive open-source data integration, processing, analysis, and exploration platform.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2008</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/bioinformatics/btr478" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.knime.org/">http://www.knime.org/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Iris Ada, Michael Berhold and Thomas Gabriel</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/bioinformatics/btr478">10.1093/bioinformatics/btr478</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, Mac OS X, Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GNU GPL v.3</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Workflows</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_GB " onmouseover="javascript:show('idp455232');">
<div class="magnifiable">
<p class="year">02</p>
<br><p class="label linked">Gb</p>
<br><p class="name">GBROWSE</p>
<div class="elementMagnify cat_GB new" id="idp455232" style="display: none;" onmouseout="javascript:hide('idp455232');">
<p class="year">2002</p>
<br><a href="javascript:showDetail('idp455232');"><p class="label linked">Gb</p></a><br><p class="name">GBROWSE</p>
</div>
</div>
<div class="elementDetail" id="detail_idp455232" style="display: none;">
<div class="bg"></div>
<div class="fg cat_GB">
<p class="detailClose"><a href="javascript:hideDetail('idp455232');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">GBROWSE</td>
<td class="detailLogo" colspan="2"><img src="http://www.eaglegenomics.com/wp-content/uploads/2012/09/400px-GBrowseLogo.png"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> GBrowse is a combination of database and interactive web pages for manipulating and displaying annotations on genomes.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2002</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1101/gr.403602" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://gmod.org/wiki/Gbrowse">http://gmod.org/wiki/Gbrowse</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Lincoln Stein</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1101/gr.403602">10.1101/gr.403602</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, MacOSX and Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Perl Artistic Licence</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Genome Browsers</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_GB " onmouseover="javascript:show('idp459376');">
<div class="magnifiable">
<p class="year">04</p>
<br><p class="label linked">Nc</p>
<br><p class="name">NCBI MapViewer</p>
<div class="elementMagnify cat_GB new" id="idp459376" style="display: none;" onmouseout="javascript:hide('idp459376');">
<p class="year">2004</p>
<br><a href="javascript:showDetail('idp459376');"><p class="label linked">Nc</p></a><br><p class="name">NCBI MapViewer</p>
</div>
</div>
<div class="elementDetail" id="detail_idp459376" style="display: none;">
<div class="bg"></div>
<div class="fg cat_GB">
<p class="detailClose"><a href="javascript:hideDetail('idp459376');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">NCBI MapViewer</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> The Map Viewer provides special browsing capabilities for a subset of organisms in Entrez Genomes. The viewer allows you to view and search an organism's complete genome, display chromosome maps, and zoom into progressively greater levels of detail, down to the sequence data for a region of interest.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2004</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1002/0471250953.bi0105s16" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/mapview/static/MapViewerHelp.html#Overview">http://www.ncbi.nlm.nih.gov/mapview/static/MapViewerHelp.html#Overview</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight"></td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1002/0471250953.bi0105s16">10.1002/0471250953.bi0105s16</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Mac OS X, Unix and Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight"></td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Genome Browsers</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_TK " onmouseover="javascript:show('idp459056');">
<div class="magnifiable">
<p class="year">03</p>
<br><p class="label linked">By</p>
<br><p class="name">BioPython</p>
<div class="elementMagnify cat_TK new" id="idp459056" style="display: none;" onmouseout="javascript:hide('idp459056');">
<p class="year">2003</p>
<br><a href="javascript:showDetail('idp459056');"><p class="label linked">By</p></a><br><p class="name">BioPython</p>
</div>
</div>
<div class="elementDetail" id="detail_idp459056" style="display: none;">
<div class="bg"></div>
<div class="fg cat_TK">
<p class="detailClose"><a href="javascript:hideDetail('idp459056');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">BioPython</td>
<td class="detailLogo" colspan="2"><img src="http://www.eaglegenomics.com/wp-content/uploads/2012/09/Biopython.jpg"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Biopython is a set of freely available tools for biological computation written in Python by an international team of developers.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2003</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/bioinformatics/btp163" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://biopython.org/wiki/Biopython">http://biopython.org/wiki/Biopython</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Tiago Antao, Sebastian Bassi, Jeffrey Chang etc.</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/bioinformatics/btp163">10.1093/bioinformatics/btp163</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">MacOsx, Unix and Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Biopython License</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">ToolKits and APIs</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_TK " onmouseover="javascript:show('idp465744');">
<div class="magnifiable">
<p class="year">09</p>
<br><p class="label linked">Pic</p>
<br><p class="name">Picard</p>
<div class="elementMagnify cat_TK new" id="idp465744" style="display: none;" onmouseout="javascript:hide('idp465744');">
<p class="year">2009</p>
<br><a href="javascript:showDetail('idp465744');"><p class="label linked">Pic</p></a><br><p class="name">Picard</p>
</div>
</div>
<div class="elementDetail" id="detail_idp465744" style="display: none;">
<div class="bg"></div>
<div class="fg cat_TK">
<p class="detailClose"><a href="javascript:hideDetail('idp465744');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">Picard</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Picard comprises Java-based command-line utilities that manipulate SAM files, and a Java API (SAM-JDK) for creating new programs that read and write SAM files. Both SAM text format and SAM binary (BAM) format are supported.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2009</td>
<td rowspan="7">No impact data available.</td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://picard.sourceforge.net/">http://picard.sourceforge.net/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight"></td>
</tr>
<tr>
<td class="detailLeft">Key publication:</td>
<td class="detailRight"><a target="_blank" href="http://picard.sourceforge.net/">http://picard.sourceforge.net/</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight"></td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Apache License V2.0, MIT License</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">ToolKits and APIs</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_TK " onmouseover="javascript:show('idp468912');">
<div class="magnifiable">
<p class="year">09</p>
<br><p class="label linked">Vc</p>
<br><p class="name">VCFtools</p>
<div class="elementMagnify cat_TK new" id="idp468912" style="display: none;" onmouseout="javascript:hide('idp468912');">
<p class="year">2009</p>
<br><a href="javascript:showDetail('idp468912');"><p class="label linked">Vc</p></a><br><p class="name">VCFtools</p>
</div>
</div>
<div class="elementDetail" id="detail_idp468912" style="display: none;">
<div class="bg"></div>
<div class="fg cat_TK">
<p class="detailClose"><a href="javascript:hideDetail('idp468912');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">VCFtools</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Welcome to VCFtools - a program package designed for working with VCF files, such as those generated by the 1000 Genomes Project. VCFtools is split into two sections: The vcftool binary program/ The VCF perl module.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2009</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/bioinformatics/btr330" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://vcftools.sourceforge.net/">http://vcftools.sourceforge.net/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Adam Auton and Petr Danecek</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/bioinformatics/btr330">10.1093/bioinformatics/btr330</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux and Mac OS X</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GNU General Public Licence version 3.0</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">ToolKits and APIs</td>
</tr>
</table>
</div>
</div>
</td>
  </tr>
  <tr>
  <td class="element cat_ALP " onmouseover="javascript:show('idp472800');">
<div class="magnifiable">
<p class="year">88</p>
<br><p class="label linked">Fa</p>
<br><p class="name">FASTA</p>
<div class="elementMagnify cat_ALP new" id="idp472800" style="display: none;" onmouseout="javascript:hide('idp472800');">
<p class="year">1988</p>
<br><a href="javascript:showDetail('idp472800');"><p class="label linked">Fa</p></a><br><p class="name">FASTA</p>
</div>
</div>
<div class="elementDetail" id="detail_idp472800" style="display: none;">
<div class="bg"></div>
<div class="fg cat_ALP">
<p class="detailClose"><a href="javascript:hideDetail('idp472800');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">FASTA</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> FASTA is a DNA and protein sequence alignment software package first described (as FASTP). Its legacy is the FASTA format which is now ubiquitous in bioinformatics. FASTA is pronounced 'fast A', and stands for 'FAST-All', because it works with any alphabet, an extension of 'FAST-P' (protein) and 'FAST-N' (nucleotide) alignment.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">1988</td>
<td rowspan="7">No impact data available.</td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://en.wikipedia.org/wiki/FASTA">http://en.wikipedia.org/wiki/FASTA</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">David Lipman and William Pearson</td>
</tr>
<tr>
<td class="detailLeft">Key publication:</td>
<td class="detailRight"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC280013/?tool=pmcentrez">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC280013/?tool=pmcentrez</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Unix, Linux, MacOSX, Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Free for academic users.</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Aligners (pairwise)</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_ALS " onmouseover="javascript:show('idp477088');">
<div class="magnifiable">
<p class="year">10</p>
<br><p class="label linked">Sm</p>
<br><p class="name">SMALT</p>
<div class="elementMagnify cat_ALS new" id="idp477088" style="display: none;" onmouseout="javascript:hide('idp477088');">
<p class="year">2010</p>
<br><a href="javascript:showDetail('idp477088');"><p class="label linked">Sm</p></a><br><p class="name">SMALT</p>
</div>
</div>
<div class="elementDetail" id="detail_idp477088" style="display: none;">
<div class="bg"></div>
<div class="fg cat_ALS">
<p class="detailClose"><a href="javascript:hideDetail('idp477088');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">SMALT</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> SMALT efficiently aligns DNA sequencing reads with genomic reference sequences. The software employs a perfect hash index of short words (&lt; 21 nucleotides long), sampled at equidistant steps along the genomic reference sequences. For each read, potentially matching segments in the reference are identified from seed matches in the index and subsequently aligned with the read using a banded Smith-Waterman algorithm. Reads from a wide range of sequencing platforms, for example Illumina, Roche-454, Ion Torrent, PacBio or ABI-Sanger, can be processed including paired-end reads. </td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2010</td>
<td rowspan="7">No impact data available.</td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.sanger.ac.uk/resources/software/smalt/">http://www.sanger.ac.uk/resources/software/smalt/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Hannes Ponstigl</td>
</tr>
<tr>
<td class="detailLeft">Key publication:</td>
<td class="detailRight"><a target="_blank" href="index.html"></a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Unix</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GNU GPL</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Aligners (short read)</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_MSA " onmouseover="javascript:show('idp476336');">
<div class="magnifiable">
<p class="year">04</p>
<br><p class="label linked">Ml</p>
<br><p class="name">MUSCLE</p>
<div class="elementMagnify cat_MSA new" id="idp476336" style="display: none;" onmouseout="javascript:hide('idp476336');">
<p class="year">2004</p>
<br><a href="javascript:showDetail('idp476336');"><p class="label linked">Ml</p></a><br><p class="name">MUSCLE</p>
</div>
</div>
<div class="elementDetail" id="detail_idp476336" style="display: none;">
<div class="bg"></div>
<div class="fg cat_MSA">
<p class="detailClose"><a href="javascript:hideDetail('idp476336');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">MUSCLE</td>
<td class="detailLogo" colspan="2"><img src="http://www.eaglegenomics.com/wp-content/uploads/2012/08/Muscle.jpg"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> MUSCLE is a program for creating multiple alignments of amino acid or nucleotide sequences. A range of options is provided that give you the choice of optimizing accuracy, speed, or some compromise between the two.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2004</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/nar/gkh340" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.drive5.com/muscle/">http://www.drive5.com/muscle/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Robert C. Edgar</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/nar/gkh340">10.1093/nar/gkh340</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, MacOSX, Unix, Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Freeware</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Multiple Sequence Aligners</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_MSA " onmouseover="javascript:show('idp483136');">
<div class="magnifiable">
<p class="year">05</p>
<br><p class="label linked">Pb</p>
<br><p class="name">PROBCONS</p>
<div class="elementMagnify cat_MSA new" id="idp483136" style="display: none;" onmouseout="javascript:hide('idp483136');">
<p class="year">2005</p>
<br><a href="javascript:showDetail('idp483136');"><p class="label linked">Pb</p></a><br><p class="name">PROBCONS</p>
</div>
</div>
<div class="elementDetail" id="detail_idp483136" style="display: none;">
<div class="bg"></div>
<div class="fg cat_MSA">
<p class="detailClose"><a href="javascript:hideDetail('idp483136');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">PROBCONS</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> ProbCons is an open source probabilistic consistency-based multiple alignment of amino acid sequences. It is an efficient protein multiple sequence alignment program, which has demonstrated a statistically significant improvement in accuracy compared to several leading alignment tools.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2005</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1101/gr.2821705" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://probcons.stanford.edu/about.html">http://probcons.stanford.edu/about.html</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Chuong Do, Michael Brudno</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1101/gr.2821705">10.1101/gr.2821705</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Unix</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Open Source</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Multiple Sequence Aligners</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_AGL " onmouseover="javascript:show('idp486624');">
<div class="magnifiable">
<p class="year">03</p>
<br><p class="label linked">Ph</p>
<br><p class="name">PHUSION(2)</p>
<div class="elementMagnify cat_AGL new" id="idp486624" style="display: none;" onmouseout="javascript:hide('idp486624');">
<p class="year">2003</p>
<br><a href="javascript:showDetail('idp486624');"><p class="label linked">Ph</p></a><br><p class="name">PHUSION(2)</p>
</div>
</div>
<div class="elementDetail" id="detail_idp486624" style="display: none;">
<div class="bg"></div>
<div class="fg cat_AGL">
<p class="detailClose"><a href="javascript:hideDetail('idp486624');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">PHUSION(2)</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Phusion is a software package for assembling genome sequences from whole genome shotgun(WGS) reads. The Phusion assembler is involved in assembling a number of genomes, such as mouse, zebrafish, C. briggsae, Schistosoma mansoni etc.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2003</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1101/gr.731003" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.sanger.ac.uk/resources/software/phusion/">http://www.sanger.ac.uk/resources/software/phusion/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Mullikin JC and Zemin Ning</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1101/gr.731003">10.1101/gr.731003</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Unix</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Creative Commons Attribution 3.0 Unported Licence</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Assemblers Genomic (long read)</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_AGL " onmouseover="javascript:show('idp490160');">
<div class="magnifiable">
<p class="year">10</p>
<br><p class="label linked">Qu</p>
<br><p class="name">QUAKE</p>
<div class="elementMagnify cat_AGL new" id="idp490160" style="display: none;" onmouseout="javascript:hide('idp490160');">
<p class="year">2010</p>
<br><a href="javascript:showDetail('idp490160');"><p class="label linked">Qu</p></a><br><p class="name">QUAKE</p>
</div>
</div>
<div class="elementDetail" id="detail_idp490160" style="display: none;">
<div class="bg"></div>
<div class="fg cat_AGL">
<p class="detailClose"><a href="javascript:hideDetail('idp490160');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">QUAKE</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Quake is a package to correct substitution sequencing errors in experiments with deep coverage (e.g. &gt;15X), specifically intended for Illumina sequencing reads. Quake adopts the k-mer error correction framework, first introduced by the EULER genome assembly package.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2010</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1186/gb-2010-11-11-r116" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.cbcb.umd.edu/software/quake/">http://www.cbcb.umd.edu/software/quake/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">David R. Kelly, Michael C. Schatz, Steven L. Salzberg</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1186/gb-2010-11-11-r116">10.1186/gb-2010-11-11-r116</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Pearl Artistic Licence</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Assemblers Genomic (long read)</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_AGS " onmouseover="javascript:show('idp493712');">
<div class="magnifiable">
<p class="year">07</p>
<br><p class="label linked">Al</p>
<br><p class="name">ALLPATHS-LG</p>
<div class="elementMagnify cat_AGS new" id="idp493712" style="display: none;" onmouseout="javascript:hide('idp493712');">
<p class="year">2007</p>
<br><a href="javascript:showDetail('idp493712');"><p class="label linked">Al</p></a><br><p class="name">ALLPATHS-LG</p>
</div>
</div>
<div class="elementDetail" id="detail_idp493712" style="display: none;">
<div class="bg"></div>
<div class="fg cat_AGS">
<p class="detailClose"><a href="javascript:hideDetail('idp493712');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">ALLPATHS-LG</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> ALLPATHS LG is a whole-genome shotgun assembler that can generate high-quality genome assemblies using short reads (~100bp) such as those produced by the new generation of sequencers. The significant difference between ALLPATHS and traditional assemblers such as Arachne is that ALLPATHS assemblies are not necessarily linear, but instead are presented in the form of a graph.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2007</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1101/gr.7337908" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.broadinstitute.org/science/programs/genome-biology/crd">http://www.broadinstitute.org/science/programs/genome-biology/crd</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Gnerre S.</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1101/gr.7337908">10.1101/gr.7337908</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, Unix</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Open Source Licence</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Assemblers Genomic (short read)</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_AGS " onmouseover="javascript:show('idp497360');">
<div class="magnifiable">
<p class="year">09</p>
<br><p class="label linked">Ba</p>
<br><p class="name">BAMBUS 2 09</p>
<div class="elementMagnify cat_AGS new" id="idp497360" style="display: none;" onmouseout="javascript:hide('idp497360');">
<p class="year">2009</p>
<br><a href="javascript:showDetail('idp497360');"><p class="label linked">Ba</p></a><br><p class="name">BAMBUS 2 09</p>
</div>
</div>
<div class="elementDetail" id="detail_idp497360" style="display: none;">
<div class="bg"></div>
<div class="fg cat_AGS">
<p class="detailClose"><a href="javascript:hideDetail('idp497360');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">BAMBUS 2 09</td>
<td class="detailLogo" colspan="2"><img src="http://www.eaglegenomics.com/wp-content/uploads/2012/08/bambus2logo.jpg"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Bambus 2.0, the second generation Bambus scaffolder for polymorphic and metagenomic data.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2009</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1101/gr.1536204" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.cbcb.umd.edu/software/bambus/">http://www.cbcb.umd.edu/software/bambus/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Segey Koren and Mihai Pop</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1101/gr.1536204">10.1101/gr.1536204</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Open source</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Assemblers Genomic (short read)</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_ASM new" onmouseover="javascript:show('idp501040');">
<div class="magnifiable">
<p class="year">11</p>
<br><p class="label linked">Ty</p>
<br><p class="name">Trinity</p>
<div class="elementMagnify cat_ASM new" id="idp501040" style="display: none;" onmouseout="javascript:hide('idp501040');">
<p class="year">2011</p>
<br><a href="javascript:showDetail('idp501040');"><p class="label linked">Ty</p></a><br><p class="name">Trinity</p>
</div>
</div>
<div class="elementDetail" id="detail_idp501040" style="display: none;">
<div class="bg"></div>
<div class="fg cat_ASM">
<p class="detailClose"><a href="javascript:hideDetail('idp501040');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">Trinity</td>
<td class="detailLogo" colspan="2"><img src="http://trinityrnaseq.sourceforge.net/images/TrinityCompositeLogo.png"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Trinity, developed at the Broad Institute and the Hebrew University of Jerusalem, represents a novel method for the efficient and robust de novo reconstruction of transcriptomes from RNA-seq data. Trinity combines three independent software modules: Inchworm, Chrysalis, and Butterfly, applied sequentially to process large volumes of RNA-seq reads. Trinity partitions the sequence data into many individual de Bruijn graphs, each representing the transcriptional complexity at at a given gene or locus, and then processes each graph independently to extract full-length splicing isoforms and to tease apart transcripts derived from paralogous genes.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2011</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1038/nbt.1883" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://trinityrnaseq.sourceforge.net/">http://trinityrnaseq.sourceforge.net/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Grabherr MG, Haas BJ, Yassour M, Levin JZ, Thompson DA, Amit I, Adiconis X, Fan L, Raychowdhury R, Zeng Q, Chen Z, Mauceli E, Hacohen N, Gnirke A, Rhind N, di Palma F, Birren BW, Nusbaum C, Lindblad-Toh K, Friedman N, Regev A. </td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1038/nbt.1883">10.1038/nbt.1883</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">BSD</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Assemblers (mRNA)</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_GPM " onmouseover="javascript:show('idp505408');">
<div class="magnifiable">
<p class="year">11</p>
<br><p class="label linked">Sn</p>
<br><p class="name">SNAP</p>
<div class="elementMagnify cat_GPM new" id="idp505408" style="display: none;" onmouseout="javascript:hide('idp505408');">
<p class="year">2011</p>
<br><a href="javascript:showDetail('idp505408');"><p class="label linked">Sn</p></a><br><p class="name">SNAP</p>
</div>
</div>
<div class="elementDetail" id="detail_idp505408" style="display: none;">
<div class="bg"></div>
<div class="fg cat_GPM">
<p class="detailClose"><a href="javascript:hideDetail('idp505408');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">SNAP</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> The SNAP gene finder is HMM-based like Genscan and attempts to be more adaptable to different organisms, addressing problems related to using a gene finder on a genome sequence that it was not trained against.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2011</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1186/1471-2105-5-59" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://korflab.ucdavis.edu/software.html">http://korflab.ucdavis.edu/software.html</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Ian Korf</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1186/1471-2105-5-59">10.1186/1471-2105-5-59</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GNU GPL</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Gene Prediction (mRNA)</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_ST new" onmouseover="javascript:show('idp509760');">
<div class="magnifiable">
<p class="year">96</p>
<br><p class="label linked">Rk</p>
<br><p class="name">RepeatMasker</p>
<div class="elementMagnify cat_ST new" id="idp509760" style="display: none;" onmouseout="javascript:hide('idp509760');">
<p class="year">1996</p>
<br><a href="javascript:showDetail('idp509760');"><p class="label linked">Rk</p></a><br><p class="name">RepeatMasker</p>
</div>
</div>
<div class="elementDetail" id="detail_idp509760" style="display: none;">
<div class="bg"></div>
<div class="fg cat_ST">
<p class="detailClose"><a href="javascript:hideDetail('idp509760');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">RepeatMasker</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> RepeatMasker is a program that screens DNA sequences for interspersed repeats and low complexity DNA sequences. The output of the program is a detailed annotation of the repeats that are present in the query sequence as well as a modified version of the query sequence in which all the annotated repeats have been masked (default: replaced by Ns). On average, almost 50% of a human genomic DNA sequence currently will be masked by the program. Sequence comparisons in RepeatMasker are performed by one of several popular search engines including, cross_match, ABBlast/WUBlast, RMBlast and Decypher.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">1996</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1002/0471250953.bi0410s05" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.repeatmasker.org/">http://www.repeatmasker.org/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Smit, AFA, Hubley, R, Green, P.</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1002/0471250953.bi0410s05">10.1002/0471250953.bi0410s05</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Universal</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">OSL v2.1</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Sequence Tools</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_ST new" onmouseover="javascript:show('idp509008');">
<div class="magnifiable">
<p class="year">11</p>
<br><p class="label linked">Vt</p>
<br><p class="name">VariantTools</p>
<div class="elementMagnify cat_ST new" id="idp509008" style="display: none;" onmouseout="javascript:hide('idp509008');">
<p class="year">2011</p>
<br><a href="javascript:showDetail('idp509008');"><p class="label linked">Vt</p></a><br><p class="name">VariantTools</p>
</div>
</div>
<div class="elementDetail" id="detail_idp509008" style="display: none;">
<div class="bg"></div>
<div class="fg cat_ST">
<p class="detailClose"><a href="javascript:hideDetail('idp509008');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">VariantTools</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Variant tools is a software tool for the annotation, selection, and analysis of variants in the context of next-gen sequencing analysis. Unlike some other tools used for Next-Gen sequencing analysis, variant tools is project based and provide a whole set of tools to manipulate and analyze genetic variants.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2011</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/bioinformatics/btr667" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://varianttools.sourceforge.net/">http://varianttools.sourceforge.net/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">F Anthony San Lucas, Gao Wang, Paul Scheet, Bo Peng</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/bioinformatics/btr667">10.1093/bioinformatics/btr667</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Mac, Linux</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GPL v3</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Sequence Tools</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_DB " onmouseover="javascript:show('idp516144');">
<div class="magnifiable">
<p class="year">71</p>
<br><p class="label linked">Pd</p>
<br><p class="name">PDB</p>
<div class="elementMagnify cat_DB new" id="idp516144" style="display: none;" onmouseout="javascript:hide('idp516144');">
<p class="year">1971</p>
<br><a href="javascript:showDetail('idp516144');"><p class="label linked">Pd</p></a><br><p class="name">PDB</p>
</div>
</div>
<div class="elementDetail" id="detail_idp516144" style="display: none;">
<div class="bg"></div>
<div class="fg cat_DB">
<p class="detailClose"><a href="javascript:hideDetail('idp516144');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">PDB</td>
<td class="detailLogo" colspan="2"><img src="http://www.rcsb.org/pdb/skins/web20/default/images/pdbLogo.png"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> The Protein Data Bank (PDB) archive is the single worldwide repository of information about the 3D structures of large biological molecules, including proteins and nucleic acids. These are the molecules of life that are found in all organisms including bacteria, yeast, plants, flies, other animals, and humans. Understanding the shape of a molecule helps to understand how it works. This knowledge can be used to help deduce a structure's role in human health and disease, and in drug development. The structures in the archive range from tiny proteins and bits of DNA to complex molecular machines like the ribosome.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">1971</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/nar/28.1.235" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.rcsb.org/pdb">http://www.rcsb.org/pdb</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Consortium.</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/nar/28.1.235">10.1093/nar/28.1.235</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, Mac OS X and Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Data contained in PDF is free of all copyright restrictions and made fully and freely available for both non-commercial and commercial use.</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Database/Warehouse</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_DB " onmouseover="javascript:show('idp520416');">
<div class="magnifiable">
<p class="year">03</p>
<br><p class="label linked">Up</p>
<br><p class="name">UniProt</p>
<div class="elementMagnify cat_DB new" id="idp520416" style="display: none;" onmouseout="javascript:hide('idp520416');">
<p class="year">2003</p>
<br><a href="javascript:showDetail('idp520416');"><p class="label linked">Up</p></a><br><p class="name">UniProt</p>
</div>
</div>
<div class="elementDetail" id="detail_idp520416" style="display: none;">
<div class="bg"></div>
<div class="fg cat_DB">
<p class="detailClose"><a href="javascript:hideDetail('idp520416');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">UniProt</td>
<td class="detailLogo" colspan="2"><img src="http://www.uniprot.org/images/logo.gif"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> The mission of UniProt is to provide the scientific community with a comprehensive, high-quality and freely accessible resource of protein sequence and functional information.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2003</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/nar/gkr981" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.uniprot.org/">http://www.uniprot.org/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Consortium led by Rolf Apweiler, Cathy Wu, and Ioannis Xenarios.</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/nar/gkr981">10.1093/nar/gkr981</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, Mac OS X and Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Creative Commons Attribution-NoDerivs</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Database/Warehouse</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_GB " onmouseover="javascript:show('idp524176');">
<div class="magnifiable">
<p class="year">08</p>
<br><p class="label linked">Iv</p>
<br><p class="name">IGV</p>
<div class="elementMagnify cat_GB new" id="idp524176" style="display: none;" onmouseout="javascript:hide('idp524176');">
<p class="year">2008</p>
<br><a href="javascript:showDetail('idp524176');"><p class="label linked">Iv</p></a><br><p class="name">IGV</p>
</div>
</div>
<div class="elementDetail" id="detail_idp524176" style="display: none;">
<div class="bg"></div>
<div class="fg cat_GB">
<p class="detailClose"><a href="javascript:hideDetail('idp524176');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">IGV</td>
<td class="detailLogo" colspan="2"><img src="http://www.eaglegenomics.com/wp-content/uploads/2012/09/IGVlogo.png"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> The Integrative Genomics Viewer (IGV) is a high-performance visualization tool for interactive exploration of large, integrated datasets. It supports a wide variety of data types including sequence alignments, microarrays, and genomic annotations.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2008</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1038/nbt.1754" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.broadinstitute.org/igv/home">http://www.broadinstitute.org/igv/home</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Jim Robinson, Helga Thorvaldsdottir, Jacob Silterra and Marc-Danie Nazaire.</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1038/nbt.1754">10.1038/nbt.1754</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, Mac OS X and Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GNU Lesser General Public Licence</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Genome Browsers</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_GB " onmouseover="javascript:show('idp264560');">
<div class="magnifiable">
<p class="year">07</p>
<br><p class="label linked">Ag</p>
<br><p class="name">ARGO</p>
<div class="elementMagnify cat_GB new" id="idp264560" style="display: none;" onmouseout="javascript:hide('idp264560');">
<p class="year">2007</p>
<br><a href="javascript:showDetail('idp264560');"><p class="label linked">Ag</p></a><br><p class="name">ARGO</p>
</div>
</div>
<div class="elementDetail" id="detail_idp264560" style="display: none;">
<div class="bg"></div>
<div class="fg cat_GB">
<p class="detailClose"><a href="javascript:hideDetail('idp264560');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">ARGO</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> The Argo Genome Browser is the Broad Institute's production tool for visualizing and manually annotating whole genomes.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2007</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/bioinformatics/btl193" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.broadinstitute.org/annotation/argo/">http://www.broadinstitute.org/annotation/argo/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Reinhard Engels</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/bioinformatics/btl193">10.1093/bioinformatics/btl193</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Mac OS X and Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GNU General Public Licence</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Genome Browsers</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_GB " onmouseover="javascript:show('idp267968');">
<div class="magnifiable">
<p class="year">02</p>
<br><p class="label linked">At</p>
<br><p class="name">ARTEMIS</p>
<div class="elementMagnify cat_GB new" id="idp267968" style="display: none;" onmouseout="javascript:hide('idp267968');">
<p class="year">2002</p>
<br><a href="javascript:showDetail('idp267968');"><p class="label linked">At</p></a><br><p class="name">ARTEMIS</p>
</div>
</div>
<div class="elementDetail" id="detail_idp267968" style="display: none;">
<div class="bg"></div>
<div class="fg cat_GB">
<p class="detailClose"><a href="javascript:hideDetail('idp267968');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">ARTEMIS</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Artemis is a free genome browser and annotation tool that allows visualisation of sequence features, next generation data and the results of analyses within the context of the sequence, and also its six-frame translation.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2002</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/bioinformatics/16.10.944" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.sanger.ac.uk/resources/software/artemis/">http://www.sanger.ac.uk/resources/software/artemis/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Tim Carver, Giles Velarde, Matthew Berriman, Julian Parkhill and Jacqueline McQuillan.</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/bioinformatics/16.10.944">10.1093/bioinformatics/16.10.944</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, Mac OS X, Unix and Windows.</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GNU General Public Licence v. 2.0</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Genome Browsers</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_TK " onmouseover="javascript:show('idp271552');">
<div class="magnifiable">
<p class="year">05</p>
<br><p class="label linked">As</p>
<br><p class="name">APSampler</p>
<div class="elementMagnify cat_TK new" id="idp271552" style="display: none;" onmouseout="javascript:hide('idp271552');">
<p class="year">2005</p>
<br><a href="javascript:showDetail('idp271552');"><p class="label linked">As</p></a><br><p class="name">APSampler</p>
</div>
</div>
<div class="elementDetail" id="detail_idp271552" style="display: none;">
<div class="bg"></div>
<div class="fg cat_TK">
<p class="detailClose"><a href="javascript:hideDetail('idp271552');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">APSampler</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> APSampler is a tool that allows multi-locus and multi-level association analysis of genotypic and phenotypic data. The goal is to find the allelic sets (patterns) that are associated with phenotype.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2005</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1534/genetics.105.048090" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://code.google.com/p/apsampler/">http://code.google.com/p/apsampler/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Dmitrijs Lvov and A.V. Favorov</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1534/genetics.105.048090">10.1534/genetics.105.048090</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux and Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">MIT License</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">ToolKits and APIs</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_TK " onmouseover="javascript:show('idp275008');">
<div class="magnifiable">
<p class="year">12</p>
<br><p class="label linked">Ge</p>
<br><p class="name">GenABEL</p>
<div class="elementMagnify cat_TK new" id="idp275008" style="display: none;" onmouseout="javascript:hide('idp275008');">
<p class="year">2012</p>
<br><a href="javascript:showDetail('idp275008');"><p class="label linked">Ge</p></a><br><p class="name">GenABEL</p>
</div>
</div>
<div class="elementDetail" id="detail_idp275008" style="display: none;">
<div class="bg"></div>
<div class="fg cat_TK">
<p class="detailClose"><a href="javascript:hideDetail('idp275008');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">GenABEL</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> The mission of the GenABEL project is to provide a free framework for collaborative, robust, transparent, open-source based development of statistical genomics methodology.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2012</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/bioinformatics/btm108 " data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.genabel.org/">http://www.genabel.org/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">GenABEL project developers</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/bioinformatics/btm108%20">10.1093/bioinformatics/btm108 </a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GPL v2</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">ToolKits and APIs</td>
</tr>
</table>
</div>
</div>
</td>
  </tr>
  <tr>
<td class="element cat_ALP new" onmouseover="javascript:show('idp278864');">
<div class="magnifiable">
<p class="year">05</p>
<br><p class="label linked">Ac</p>
<br><p class="name">ACT</p>
<div class="elementMagnify cat_ALP new" id="idp278864" style="display: none;" onmouseout="javascript:hide('idp278864');">
<p class="year">2005</p>
<br><a href="javascript:showDetail('idp278864');"><p class="label linked">Ac</p></a><br><p class="name">ACT</p>
</div>
</div>
<div class="elementDetail" id="detail_idp278864" style="display: none;">
<div class="bg"></div>
<div class="fg cat_ALP">
<p class="detailClose"><a href="javascript:hideDetail('idp278864');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">ACT</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> ACT is a free tool for displaying pairwise comparisons between two or more DNA sequences. It can be used to identify and analyse regions of similarity and difference between genomes and to explore conservation of synteny, in the context of the entire sequences and their annotation.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2005</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/bioinformatics/bti553" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.sanger.ac.uk/resources/software/act/">http://www.sanger.ac.uk/resources/software/act/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Carver TJ, Rutherford KM, Berriman M, Rajandream MA, Barrell BG and Parkhill J</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/bioinformatics/bti553">10.1093/bioinformatics/bti553</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Universal</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GNU GPL</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Aligners (pairwise)</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_ALS new" onmouseover="javascript:show('idp562320');">
<div class="magnifiable">
<p class="year">12</p>
<br><p class="label linked">Rs</p>
<br><p class="name">rna-star</p>
<div class="elementMagnify cat_ALS new" id="idp562320" style="display: none;" onmouseout="javascript:hide('idp562320');">
<p class="year">2012</p>
<br><a href="javascript:showDetail('idp562320');"><p class="label linked">Rs</p></a><br><p class="name">rna-star</p>
</div>
</div>
<div class="elementDetail" id="detail_idp562320" style="display: none;">
<div class="bg"></div>
<div class="fg cat_ALS">
<p class="detailClose"><a href="javascript:hideDetail('idp562320');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">rna-star</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> To align our large (&gt;80 billon reads) ENCODE Transcriptome RNA-seq dataset, we developed the Spliced Transcripts Alignment to a Reference (STAR) software based on a previously undescribed RNA-seq alignment algorithm that uses sequential maximum mappable seed search in uncompressed suffix arrays followed by seed clustering and stitching procedure.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2012</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/bioinformatics/bts635" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="https://code.google.com/p/rna-star/">https://code.google.com/p/rna-star/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Alexander Dobin, Carrie A. Davis, Felix Schlesinger, Jorg Drenkow, Chris Zaleski, Sonali Jha, Philippe Batut, Mark Chaisson and Thomas R. Gingeras</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/bioinformatics/bts635">10.1093/bioinformatics/bts635</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Universal</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GPL v3</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Aligners (short read)</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_MSA " onmouseover="javascript:show('idp565984');">
<div class="magnifiable">
<p class="year">12</p>
<br><p class="label linked">Po</p>
<br><p class="name">PROT PAL</p>
<div class="elementMagnify cat_MSA new" id="idp565984" style="display: none;" onmouseout="javascript:hide('idp565984');">
<p class="year">2012</p>
<br><a href="javascript:showDetail('idp565984');"><p class="label linked">Po</p></a><br><p class="name">PROT PAL</p>
</div>
</div>
<div class="elementDetail" id="detail_idp565984" style="display: none;">
<div class="bg"></div>
<div class="fg cat_MSA">
<p class="detailClose"><a href="javascript:hideDetail('idp565984');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">PROT PAL</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Prot Pal is a software tool for multiple sequence alignment, ancestral reconstruction, and measurement of indel rates on a phylogenetic tree.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2012</td>
<td rowspan="7">No impact data available.</td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.biowiki.org/ProtPal">http://www.biowiki.org/ProtPal</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Oscar Westesson and Ian Holmes</td>
</tr>
<tr>
<td class="detailLeft">Key publication:</td>
<td class="detailRight"><a target="_blank" href="index.html"></a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Free software</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Multiple Sequence Aligners</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_MSA " onmouseover="javascript:show('idp569216');">
<div class="magnifiable">
<p class="year">08</p>
<br><p class="label linked">Fs</p>
<br><p class="name">FSA</p>
<div class="elementMagnify cat_MSA new" id="idp569216" style="display: none;" onmouseout="javascript:hide('idp569216');">
<p class="year">2008</p>
<br><a href="javascript:showDetail('idp569216');"><p class="label linked">Fs</p></a><br><p class="name">FSA</p>
</div>
</div>
<div class="elementDetail" id="detail_idp569216" style="display: none;">
<div class="bg"></div>
<div class="fg cat_MSA">
<p class="detailClose"><a href="javascript:hideDetail('idp569216');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">FSA</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> FSA is a probabilistic multiple sequence alignment algorithm which uses a "distance-based" approach to aligning homologous protein, RNA or DNA sequences.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2008</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1371/journal.pcbi.1000392" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://fsa.sourceforge.net/">http://fsa.sourceforge.net/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Robert Bradley, Colin Dewey, Jaeyoung Do, Sudeep Juvekar, Lior Pachter, Adam Roberts, and Michael Smoot.</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1371/journal.pcbi.1000392">10.1371/journal.pcbi.1000392</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux and Mac OS X</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GNU GPL</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Multiple Sequence Aligners</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_AGL " onmouseover="javascript:show('idp572640');">
<div class="magnifiable">
<p class="year">11</p>
<br><p class="label linked">Pi</p>
<br><p class="name">PRICE</p>
<div class="elementMagnify cat_AGL new" id="idp572640" style="display: none;" onmouseout="javascript:hide('idp572640');">
<p class="year">2011</p>
<br><a href="javascript:showDetail('idp572640');"><p class="label linked">Pi</p></a><br><p class="name">PRICE</p>
</div>
</div>
<div class="elementDetail" id="detail_idp572640" style="display: none;">
<div class="bg"></div>
<div class="fg cat_AGL">
<p class="detailClose"><a href="javascript:hideDetail('idp572640');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">PRICE</td>
<td class="detailLogo" colspan="2"><img src="http://www.eaglegenomics.com/wp-content/uploads/2012/08/PRICE-Logo-Font.jpg"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> We are pleased to release PRICE (Paired-Read Iterative Contig Extension), a de novo genome assembler implemented in C++. Its name describes the strategy that it implements for genome assembly: PRICE uses paired-read information to iteratively increase the size of existing contigs. Initially, those contigs can be individual reads from a subset of the paired-read dataset, non-paired reads from sequencing technologies that provide non-paired data, or contigs that were output from a prior run of PRICE or any other assembler.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2011</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1534/g3.113.005967" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://derisilab.ucsf.edu/software/price/index.html">http://derisilab.ucsf.edu/software/price/index.html</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight"></td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1534/g3.113.005967">10.1534/g3.113.005967</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, MacOS</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Open Source</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Assemblers Genomic (long read)</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_AGS " onmouseover="javascript:show('idp576368');">
<div class="magnifiable">
<p class="year">07</p>
<br><p class="label linked">Ve</p>
<br><p class="name">VELVET</p>
<div class="elementMagnify cat_AGS new" id="idp576368" style="display: none;" onmouseout="javascript:hide('idp576368');">
<p class="year">2007</p>
<br><a href="javascript:showDetail('idp576368');"><p class="label linked">Ve</p></a><br><p class="name">VELVET</p>
</div>
</div>
<div class="elementDetail" id="detail_idp576368" style="display: none;">
<div class="bg"></div>
<div class="fg cat_AGS">
<p class="detailClose"><a href="javascript:hideDetail('idp576368');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">VELVET</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Velvet is short read assembler which uses de Bruijn graphs to assemble next-generation sequencing reads into useable contigs. It handles mixed length reads as well as paired-end reads.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2007</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1101/gr.074492.107" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.ebi.ac.uk/~zerbino/velvet/">http://www.ebi.ac.uk/~zerbino/velvet/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Daniel Zerbino and Ewan Birney</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1101/gr.074492.107">10.1101/gr.074492.107</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, Mac OS X</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GPL v2.0</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Assemblers Genomic (short read)</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_AGS " onmouseover="javascript:show('idp579808');">
<div class="magnifiable">
<p class="year">12</p>
<br><p class="label linked">Sp</p>
<br><p class="name">SPAdes</p>
<div class="elementMagnify cat_AGS new" id="idp579808" style="display: none;" onmouseout="javascript:hide('idp579808');">
<p class="year">2012</p>
<br><a href="javascript:showDetail('idp579808');"><p class="label linked">Sp</p></a><br><p class="name">SPAdes</p>
</div>
</div>
<div class="elementDetail" id="detail_idp579808" style="display: none;">
<div class="bg"></div>
<div class="fg cat_AGS">
<p class="detailClose"><a href="javascript:hideDetail('idp579808');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">SPAdes</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> SPAdes stands for St. Petersburg genome assembler. It is intended for both standard (multicell) and single-cell MDA bacteria assemblies.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2012</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1089/cmb.2012.0021" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://bioinf.spbau.ru/en/spades">http://bioinf.spbau.ru/en/spades</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Anton Bankevich, Sergey Nurk, Dmitry Antipov etc.'c </td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1089/cmb.2012.0021">10.1089/cmb.2012.0021</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GPLv2</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Assemblers Genomic (short read)</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_AGS new" onmouseover="javascript:show('idp583232');">
<div class="magnifiable">
<p class="year">09</p>
<br><p class="label linked">Cn</p>
<br><p class="name">Contrail</p>
<div class="elementMagnify cat_AGS new" id="idp583232" style="display: none;" onmouseout="javascript:hide('idp583232');">
<p class="year">2009</p>
<br><a href="javascript:showDetail('idp583232');"><p class="label linked">Cn</p></a><br><p class="name">Contrail</p>
</div>
</div>
<div class="elementDetail" id="detail_idp583232" style="display: none;">
<div class="bg"></div>
<div class="fg cat_AGS">
<p class="detailClose"><a href="javascript:hideDetail('idp583232');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">Contrail</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Similar to other leading short read assembler, Contrail relies on the graph-theoretic framework of de Bruijn graphs. However, unlike these programs, which require large RAM resources, Contrail relies on Hadoop to iteratively transform an on-disk representation of the assembly graph, allowing an in depth analysis even for large genomes.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2009</td>
<td rowspan="7">No impact data available.</td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://contrail-bio.sf.net/">http://contrail-bio.sf.net/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Michael Schatz, Jeremy Chambers, Avijit Gupta, Rushil Gupta, David Kelley, Jeremy Lewi, Deepak Nettem, Dan Sommer, and Mihai Pop</td>
</tr>
<tr>
<td class="detailLeft">Key publication:</td>
<td class="detailRight"><a target="_blank" href="index.html"></a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">All</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Unknown</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Assemblers Genomic (short read)</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_GPM " onmouseover="javascript:show('idp586816');">
<div class="magnifiable">
<p class="year">09</p>
<br><p class="label linked">Au</p>
<br><p class="name">AUGUSTUS</p>
<div class="elementMagnify cat_GPM new" id="idp586816" style="display: none;" onmouseout="javascript:hide('idp586816');">
<p class="year">2009</p>
<br><a href="javascript:showDetail('idp586816');"><p class="label linked">Au</p></a><br><p class="name">AUGUSTUS</p>
</div>
</div>
<div class="elementDetail" id="detail_idp586816" style="display: none;">
<div class="bg"></div>
<div class="fg cat_GPM">
<p class="detailClose"><a href="javascript:hideDetail('idp586816');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">AUGUSTUS</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> AUGUSTUS is a program that predicts genes in eukaryotic genomic sequences. It can be run on this web server, on a new web server for larger input files or be downloaded and run locally.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2009</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/bioinformatics/btn013" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://augustus.gobics.de/">http://augustus.gobics.de/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Mario Stanke and Oliver Keller</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/bioinformatics/btn013">10.1093/bioinformatics/btn013</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Artistic licence 2</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Gene Prediction (mRNA)</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_GPM " onmouseover="javascript:show('idp590192');">
<div class="magnifiable">
<p class="year">08</p>
<br><p class="label linked">Mg</p>
<br><p class="name">MGENE</p>
<div class="elementMagnify cat_GPM new" id="idp590192" style="display: none;" onmouseout="javascript:hide('idp590192');">
<p class="year">2008</p>
<br><a href="javascript:showDetail('idp590192');"><p class="label linked">Mg</p></a><br><p class="name">MGENE</p>
</div>
</div>
<div class="elementDetail" id="detail_idp590192" style="display: none;">
<div class="bg"></div>
<div class="fg cat_GPM">
<p class="detailClose"><a href="javascript:hideDetail('idp590192');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">MGENE</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> mGene is a computational tool for the genome-wide prediction of protein coding genes from eukaryotic DNA sequences.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2008</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1101/gr.090597.108" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.mgene.org/">http://www.mgene.org/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Gunnar Raetsch, Gabriele Schweikert, Jonas Behr</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1101/gr.090597.108">10.1101/gr.090597.108</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Open Source</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Gene Prediction (mRNA)</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_ST new" onmouseover="javascript:show('idp593472');">
<div class="magnifiable">
<p class="year">09</p>
<br><p class="label linked">Mm</p>
<br><p class="name">MetaBioME</p>
<div class="elementMagnify cat_ST new" id="idp593472" style="display: none;" onmouseout="javascript:hide('idp593472');">
<p class="year">2009</p>
<br><a href="javascript:showDetail('idp593472');"><p class="label linked">Mm</p></a><br><p class="name">MetaBioME</p>
</div>
</div>
<div class="elementDetail" id="detail_idp593472" style="display: none;">
<div class="bg"></div>
<div class="fg cat_ST">
<p class="detailClose"><a href="javascript:hideDetail('idp593472');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">MetaBioME</td>
<td class="detailLogo" colspan="2"><img src="http://metasystems.riken.jp/metabiome/images/banner.png"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> MetaBioME is a web resource to find novel homologs for known Commercially Useful Enzymes (CUEs) in metagenomic datasets and completed bacterial genomes.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2009</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/nar/gkp1001" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://metasystems.riken.jp/metabiome/">http://metasystems.riken.jp/metabiome/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Vineet Kumar Sharma, Naveen Kumar, Tulika Prakash, Todd D. Taylor</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/nar/gkp1001">10.1093/nar/gkp1001</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Online</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Free to use</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Sequence Tools</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_ST new" onmouseover="javascript:show('idp597152');">
<div class="magnifiable">
<p class="year">04</p>
<br><p class="label linked">Jv</p>
<br><p class="name">JalView</p>
<div class="elementMagnify cat_ST new" id="idp597152" style="display: none;" onmouseout="javascript:hide('idp597152');">
<p class="year">2004</p>
<br><a href="javascript:showDetail('idp597152');"><p class="label linked">Jv</p></a><br><p class="name">JalView</p>
</div>
</div>
<div class="elementDetail" id="detail_idp597152" style="display: none;">
<div class="bg"></div>
<div class="fg cat_ST">
<p class="detailClose"><a href="javascript:hideDetail('idp597152');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">JalView</td>
<td class="detailLogo" colspan="2"><img src="http://www.jalview.org/sites/jalview.aegir.lifesci.dundee.ac.uk/files/styles/adaptive/adaptive-image/public/jalviewHome_0.jpg"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Jalview is a free program developed for the interactive editing, analysis and visualization of multiple sequence alignments. It can also work with sequence annotation, secondary structure information, phylogenetic trees and 3D molecular structures.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2004</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/bioinformatics/btp033" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.jalview.org/">http://www.jalview.org/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Waterhouse AM, Procter JB, Martin DMA, Clamp M, Barton GJ</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/bioinformatics/btp033">10.1093/bioinformatics/btp033</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">All</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GPL3</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Sequence Tools</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_DB new" onmouseover="javascript:show('idp600976');">
<div class="magnifiable">
<p class="year">97</p>
<br><p class="label linked">Ct</p>
<br><p class="name">CATH</p>
<div class="elementMagnify cat_DB new" id="idp600976" style="display: none;" onmouseout="javascript:hide('idp600976');">
<p class="year">1997</p>
<br><a href="javascript:showDetail('idp600976');"><p class="label linked">Ct</p></a><br><p class="name">CATH</p>
</div>
</div>
<div class="elementDetail" id="detail_idp600976" style="display: none;">
<div class="bg"></div>
<div class="fg cat_DB">
<p class="detailClose"><a href="javascript:hideDetail('idp600976');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">CATH</td>
<td class="detailLogo" colspan="2"><img src="http://www.cathdb.info/static/images/banner-cath-logo-h20.png"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> The CATH database is a hierarchical domain classification of protein structures in the Protein Data Bank. Protein structures are classified using a combination of automated and manual procedures. </td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">1997</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/nar/gks1211" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.cathdb.info/">http://www.cathdb.info/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Sillitoe I, Cuff AL, Dessailly BH, Dawson NL, Furnham N, Lee D, Lees JG, Lewis TE, Studer RA, Rentzsch R, Yeats C, Thornton JM, Orengo CA</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/nar/gks1211">10.1093/nar/gks1211</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Universal</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Free to use</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Database/Warehouse</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_DB " onmouseover="javascript:show('idp604768');">
<div class="magnifiable">
<p class="year">12</p>
<br><p class="label linked">Tg</p>
<br><p class="name">MISO</p>
<div class="elementMagnify cat_DB new" id="idp604768" style="display: none;" onmouseout="javascript:hide('idp604768');">
<p class="year">2012</p>
<br><a href="javascript:showDetail('idp604768');"><p class="label linked">Tg</p></a><br><p class="name">MISO</p>
</div>
</div>
<div class="elementDetail" id="detail_idp604768" style="display: none;">
<div class="bg"></div>
<div class="fg cat_DB">
<p class="detailClose"><a href="javascript:hideDetail('idp604768');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">MISO</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> MISO (Managing Information for Sequencing Operations) is a new open-source Lab Information Management System (LIMS) under development at TGAC, specifically designed for tracking next-generation sequencing experiments.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2012</td>
<td rowspan="7">No impact data available.</td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.tgac.ac.uk/miso/">http://www.tgac.ac.uk/miso/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Robert Davey and Mario Caccamo</td>
</tr>
<tr>
<td class="detailLeft">Key publication:</td>
<td class="detailRight"><a target="_blank" href="index.html"></a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, Mac OS X and Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GPL v3</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Database/Warehouse</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_GB " onmouseover="javascript:show('idp607920');">
<div class="magnifiable">
<p class="year">10</p>
<br><p class="label linked">Da</p>
<br><p class="name">DALLIANCE BROWSER</p>
<div class="elementMagnify cat_GB new" id="idp607920" style="display: none;" onmouseout="javascript:hide('idp607920');">
<p class="year">2010</p>
<br><a href="javascript:showDetail('idp607920');"><p class="label linked">Da</p></a><br><p class="name">DALLIANCE BROWSER</p>
</div>
</div>
<div class="elementDetail" id="detail_idp607920" style="display: none;">
<div class="bg"></div>
<div class="fg cat_GB">
<p class="detailClose"><a href="javascript:hideDetail('idp607920');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">DALLIANCE BROWSER</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Dalliance is a track-oriented genome viewer similar to Ensembl, UCSC, or GBrowse. However, it is a little different in operation because it uses recent extensions to web standards (HTML 5) to offer a higher level of interactivity than most previous genome viewers.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2010</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/bioinformatics/btr020" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.biodalliance.org/">http://www.biodalliance.org/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Thomas Down</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/bioinformatics/btr020">10.1093/bioinformatics/btr020</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, Mac OS X and Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">BSD-Style licence.</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Genome Browsers</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_GB " onmouseover="javascript:show('idp611456');">
<div class="magnifiable">
<p class="year">07</p>
<br><p class="label linked">Jb</p>
<br><p class="name">JBROWSE</p>
<div class="elementMagnify cat_GB new" id="idp611456" style="display: none;" onmouseout="javascript:hide('idp611456');">
<p class="year">2007</p>
<br><a href="javascript:showDetail('idp611456');"><p class="label linked">Jb</p></a><br><p class="name">JBROWSE</p>
</div>
</div>
<div class="elementDetail" id="detail_idp611456" style="display: none;">
<div class="bg"></div>
<div class="fg cat_GB">
<p class="detailClose"><a href="javascript:hideDetail('idp611456');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">JBROWSE</td>
<td class="detailLogo" colspan="2"><img src="http://www.eaglegenomics.com/wp-content/uploads/2012/09/JBrowseLogo.png"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> JBrowse is a JavaScript genome browser with an emphasis on portability. Browse is the official successor to GBrowse.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2007</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1101/gr.094607.109" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://jbrowse.org/">http://jbrowse.org/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Rober Buels and Mitch Skinner</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1101/gr.094607.109">10.1101/gr.094607.109</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux and Mac OS X</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GNU Lesser GPL v 2.1</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Genome Browsers</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_TK " onmouseover="javascript:show('idp615120');">
<div class="magnifiable">
<p class="year">XX</p>
<br><p class="label linked">Mb</p>
<br><p class="name">MrBayes</p>
<div class="elementMagnify cat_TK new" id="idp615120" style="display: none;" onmouseout="javascript:hide('idp615120');">
<p class="year">XX</p>
<br><a href="javascript:showDetail('idp615120');"><p class="label linked">Mb</p></a><br><p class="name">MrBayes</p>
</div>
</div>
<div class="elementDetail" id="detail_idp615120" style="display: none;">
<div class="bg"></div>
<div class="fg cat_TK">
<p class="detailClose"><a href="javascript:hideDetail('idp615120');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">MrBayes</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> MrBayes is a program for Bayesian inference and model choice across a wide range of phylogenetic and evolutionary models. MrBayes uses Markov chain Monte Carlo (MCMC) methods to estimate the posterior distribution of model parameters.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">XX</td>
<td rowspan="7">No impact data available.</td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://mrbayes.sourceforge.net/">http://mrbayes.sourceforge.net/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">John Huelsenbeck, Bret Larget, Paul van der Mark, Fredrik Ronquist, Donald Simon and Maxim Teslenko.</td>
</tr>
<tr>
<td class="detailLeft">Key publication:</td>
<td class="detailRight"><a target="_blank" href="http://mrbayes.sourceforge.net/commref_mb3.2.pdf">http://mrbayes.sourceforge.net/commref_mb3.2.pdf</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Macintosh, Windows, and UNIX</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Free software</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">ToolKits and APIs</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_TK " onmouseover="javascript:show('idp173808');">
<div class="magnifiable">
<p class="year">04</p>
<br><p class="label linked">Pn</p>
<br><p class="name">PhastCons</p>
<div class="elementMagnify cat_TK new" id="idp173808" style="display: none;" onmouseout="javascript:hide('idp173808');">
<p class="year">2004</p>
<br><a href="javascript:showDetail('idp173808');"><p class="label linked">Pn</p></a><br><p class="name">PhastCons</p>
</div>
</div>
<div class="elementDetail" id="detail_idp173808" style="display: none;">
<div class="bg"></div>
<div class="fg cat_TK">
<p class="detailClose"><a href="javascript:hideDetail('idp173808');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">PhastCons</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> PHAST is a freely available software package for comparative and evolutionary genomics. It consists of about half a dozen major programs, plus more than a dozen utilities for manipulating sequence alignments, phylogenetic trees, and genomic annotations (see left panel).</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2004</td>
<td rowspan="7">No impact data available.</td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://compgen.bscb.cornell.edu/phast/">http://compgen.bscb.cornell.edu/phast/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Adam Siepel, Melissa Hubisz and Katie Pollard</td>
</tr>
<tr>
<td class="detailLeft">Key publication:</td>
<td class="detailRight"><a target="_blank" href="http://compgen.bscb.cornell.edu/~acs/phyhmm_with_apdx.pdf">http://compgen.bscb.cornell.edu/~acs/phyhmm_with_apdx.pdf</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, Mac OS X and Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">BSD-style licence</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">ToolKits and APIs</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_TK new" onmouseover="javascript:show('idp173488');">
<div class="magnifiable">
<p class="year">08</p>
<br><p class="label linked">Ug</p>
<br><p class="name">UGENE</p>
<div class="elementMagnify cat_TK new" id="idp173488" style="display: none;" onmouseout="javascript:hide('idp173488');">
<p class="year">2008</p>
<br><a href="javascript:showDetail('idp173488');"><p class="label linked">Ug</p></a><br><p class="name">UGENE</p>
</div>
</div>
<div class="elementDetail" id="detail_idp173488" style="display: none;">
<div class="bg"></div>
<div class="fg cat_TK">
<p class="detailClose"><a href="javascript:hideDetail('idp173488');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">UGENE</td>
<td class="detailLogo" colspan="2"><img src="http://ugene.unipro.ru/images/ugene_logo.png"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Unipro UGENE is a multiplatform open-source software with the main goal of assisting molecular biologists without much expertise in bioinformatics to manage, analyze and visualize their data. UGENE integrates widely used bioinformatics tools within a common user interface. The toolkit supports multiple biological data formats and allows the retrieval of data from remote data sources. It provides visualization modules for biological objects such as annotated genome sequences, Next Generation Sequencing (NGS) assembly data, multiple sequence alignments, phylogenetic trees and 3D structures. Most of the integrated algorithms are tuned for maximum performance by the usage of multithreading and special processor instructions. UGENE includes a visual environment for creating reusable workflows that can be launched on local resources or in a High Performance Computing (HPC) environment.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2008</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/bioinformatics/bts091" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://ugene.unipro.ru/">http://ugene.unipro.ru/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Konstantin Okonechnikov; Olga Golosova; Mikhail Fursov</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/bioinformatics/bts091">10.1093/bioinformatics/bts091</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Universal</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GPL v2</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">ToolKits and APIs</td>
</tr>
</table>
</div>
</div>
</td>
  </tr>
  <tr>
  <td class="empty" colspan="1"></td>
<td class="element cat_ALS new" onmouseover="javascript:show('idp647488');">
<div class="magnifiable">
<p class="year">11</p>
<br><p class="label linked">Gm</p>
<br><p class="name">GMAP/GSNAP</p>
<div class="elementMagnify cat_ALS new" id="idp647488" style="display: none;" onmouseout="javascript:hide('idp647488');">
<p class="year">2011</p>
<br><a href="javascript:showDetail('idp647488');"><p class="label linked">Gm</p></a><br><p class="name">GMAP/GSNAP</p>
</div>
</div>
<div class="elementDetail" id="detail_idp647488" style="display: none;">
<div class="bg"></div>
<div class="fg cat_ALS">
<p class="detailClose"><a href="javascript:hideDetail('idp647488');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">GMAP/GSNAP</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> GMAP: A Genomic Mapping and Alignment Program for mRNA and EST Sequences, and GSNAP: Genomic Short-read Nucleotide Alignment Program</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2011</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/bioinformatics/bti310" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://research-pub.gene.com/gmap/">http://research-pub.gene.com/gmap/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Thomas D. Wu, Colin K. Watanabe, Serban Nacu</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/bioinformatics/bti310">10.1093/bioinformatics/bti310</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Unix, Linux</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Free (custom Genentech) but do not redistribute modifications</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Aligners (short read)</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="empty" colspan="6"></td>
  <td class="element cat_GPM " onmouseover="javascript:show('idp651232');">
<div class="magnifiable">
<p class="year">05</p>
<br><p class="label linked">Ji</p>
<br><p class="name">JIGSAW</p>
<div class="elementMagnify cat_GPM new" id="idp651232" style="display: none;" onmouseout="javascript:hide('idp651232');">
<p class="year">2005</p>
<br><a href="javascript:showDetail('idp651232');"><p class="label linked">Ji</p></a><br><p class="name">JIGSAW</p>
</div>
</div>
<div class="elementDetail" id="detail_idp651232" style="display: none;">
<div class="bg"></div>
<div class="fg cat_GPM">
<p class="detailClose"><a href="javascript:hideDetail('idp651232');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">JIGSAW</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> JIGSAW is a program designed to use the output from gene finders, splice site prediction programs and sequence alignments to predict gene models. JIGSAW is available for all species. We have tested JIGSAW on Human, Rice (Oryza sativa), Arabidopsis thaliana, C. elegans, Brugia malayi, Cryptococcus neoformans, Entamoeba histolytica, Theileria parva, Aspergillus fumigatus, Plasmodium falciparum and Plasmodium yoelii.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2005</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1186/gb-2006-7-s1-s9" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.cbcb.umd.edu/software/jigsaw/">http://www.cbcb.umd.edu/software/jigsaw/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Jonathan Edward Allen</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1186/gb-2006-7-s1-s9">10.1186/gb-2006-7-s1-s9</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GNU gcc3.2</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Gene Prediction (mRNA)</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_ST new" onmouseover="javascript:show('idp654784');">
<div class="magnifiable">
<p class="year">08</p>
<br><p class="label linked">Th</p>
<br><p class="name">Tophat</p>
<div class="elementMagnify cat_ST new" id="idp654784" style="display: none;" onmouseout="javascript:hide('idp654784');">
<p class="year">2008</p>
<br><a href="javascript:showDetail('idp654784');"><p class="label linked">Th</p></a><br><p class="name">Tophat</p>
</div>
</div>
<div class="elementDetail" id="detail_idp654784" style="display: none;">
<div class="bg"></div>
<div class="fg cat_ST">
<p class="detailClose"><a href="javascript:hideDetail('idp654784');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">Tophat</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> TopHat is a fast splice junction mapper for RNA-Seq reads. It aligns RNA-Seq reads to mammalian-sized genomes using the ultra high-throughput short read aligner Bowtie, and then analyzes the mapping results to identify splice junctions between exons.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2008</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1186/gb-2013-14-4-r36" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://tophat.cbcb.umd.edu/">http://tophat.cbcb.umd.edu/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Daehwan Kim, Geo Pertea, Cole Trapnell, Harold Pimentel, Ryan Kelley and Steven L Salzberg</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1186/gb-2013-14-4-r36">10.1186/gb-2013-14-4-r36</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, Mac</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Artistic</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Sequence Tools</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_ST new" onmouseover="javascript:show('idp659040');">
<div class="magnifiable">
<p class="year">12</p>
<br><p class="label linked">Re</p>
<br><p class="name">RSEM</p>
<div class="elementMagnify cat_ST new" id="idp659040" style="display: none;" onmouseout="javascript:hide('idp659040');">
<p class="year">2012</p>
<br><a href="javascript:showDetail('idp659040');"><p class="label linked">Re</p></a><br><p class="name">RSEM</p>
</div>
</div>
<div class="elementDetail" id="detail_idp659040" style="display: none;">
<div class="bg"></div>
<div class="fg cat_ST">
<p class="detailClose"><a href="javascript:hideDetail('idp659040');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">RSEM</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> RSEM is a software package for estimating gene and isoform expression levels from RNA-Seq data. The RSEM package provides an user-friendly interface, supports threads for parallel computation of the EM algorithm, single-end and paired-end read data, quality scores, variable-length reads and RSPD estimation. In addition, it provides posterior mean and 95% credibility interval estimates for expression levels. For visualization, It can generate BAM and Wiggle files in both transcript-coordinate and genomic-coordinate.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2012</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1186/1471-2105-12-323" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://deweylab.biostat.wisc.edu/rsem/">http://deweylab.biostat.wisc.edu/rsem/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Bo Li</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1186/1471-2105-12-323">10.1186/1471-2105-12-323</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GPL</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Sequence Tools</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_ST new" onmouseover="javascript:show('idp658288');">
<div class="magnifiable">
<p class="year">12</p>
<br><p class="label linked">Fb</p>
<br><p class="name">Flexbar</p>
<div class="elementMagnify cat_ST new" id="idp658288" style="display: none;" onmouseout="javascript:hide('idp658288');">
<p class="year">2012</p>
<br><a href="javascript:showDetail('idp658288');"><p class="label linked">Fb</p></a><br><p class="name">Flexbar</p>
</div>
</div>
<div class="elementDetail" id="detail_idp658288" style="display: none;">
<div class="bg"></div>
<div class="fg cat_ST">
<p class="detailClose"><a href="javascript:hideDetail('idp658288');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">Flexbar</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Flexbar preprocesses high-throughput sequencing data efficiently. It demultiplexes barcoded runs and removes adapter sequences. Moreover, trimming and filtering features are provided. Flexbar increases read mapping rates and improves genome and transcriptome assemblies. It supports next-generation sequencing data in fasta/q and csfasta/q format from Illumina, Roche 454, and the SOLiD platform.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2012</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.3390/biology1030895" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://sourceforge.net/p/flexbar/wiki/Manual/">http://sourceforge.net/p/flexbar/wiki/Manual/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Matthias Dodt, Johannes T. Roehr, Rina Ahmed, Christoph Dieterich</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.3390/biology1030895">10.3390/biology1030895</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">All</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GPL3</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Sequence Tools</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_DB new" onmouseover="javascript:show('idp665248');">
<div class="magnifiable">
<p class="year">12</p>
<br><p class="label linked">Ea</p>
<br><p class="name">ENA</p>
<div class="elementMagnify cat_DB new" id="idp665248" style="display: none;" onmouseout="javascript:hide('idp665248');">
<p class="year">2012</p>
<br><a href="javascript:showDetail('idp665248');"><p class="label linked">Ea</p></a><br><p class="name">ENA</p>
</div>
</div>
<div class="elementDetail" id="detail_idp665248" style="display: none;">
<div class="bg"></div>
<div class="fg cat_DB">
<p class="detailClose"><a href="javascript:hideDetail('idp665248');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">ENA</td>
<td class="detailLogo" colspan="2"><img src="http://www.ebi.ac.uk/ena/sites/ebi.ac.uk.ena/files/images/ena_header.png"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> The European Nucleotide Archive (ENA) captures and presents information relating to experimental workflows that are based around nucleotide sequencing. A typical workflow includes the isolation and preparation of material for sequencing, a run of a sequencing machine in which sequencing data are produced and a subsequent bioinformatic analysis pipeline. ENA records this information in a data model that covers input information (sample, experimental setup, machine configuration), output machine data (sequence traces, reads and quality scores) and interpreted information (assembly, mapping, functional annotation).</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2012</td>
<td rowspan="7">No impact data available.</td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.ebi.ac.uk/ena/home">http://www.ebi.ac.uk/ena/home</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight"></td>
</tr>
<tr>
<td class="detailLeft">Key publication:</td>
<td class="detailRight"><a target="_blank" href="index.html"></a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Online</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Free</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Database/Warehouse</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_DB " onmouseover="javascript:show('idp668704');">
<div class="magnifiable">
<p class="year">11</p>
<br><p class="label linked">Pf</p>
<br><p class="name">Pfam</p>
<div class="elementMagnify cat_DB new" id="idp668704" style="display: none;" onmouseout="javascript:hide('idp668704');">
<p class="year">2011</p>
<br><a href="javascript:showDetail('idp668704');"><p class="label linked">Pf</p></a><br><p class="name">Pfam</p>
</div>
</div>
<div class="elementDetail" id="detail_idp668704" style="display: none;">
<div class="bg"></div>
<div class="fg cat_DB">
<p class="detailClose"><a href="javascript:hideDetail('idp668704');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">Pfam</td>
<td class="detailLogo" colspan="2"><img src="http://www.eaglegenomics.com/wp-content/uploads/2012/09/Pfam.png"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> The Pfam database is a large collection of protein families, each represented by multiple sequence alignments and hidden Markov models (HMMs).</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2011</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/nar/gkr1065" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://pfam.sanger.ac.uk/">http://pfam.sanger.ac.uk/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">L. Aravind, Adam Godik and Val Wood</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/nar/gkr1065">10.1093/nar/gkr1065</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Creative Commons Zero</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Database/Warehouse</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_GB " onmouseover="javascript:show('idp672272');">
<div class="magnifiable">
<p class="year">07</p>
<br><p class="label linked">Sk</p>
<br><p class="name">SeqMonk</p>
<div class="elementMagnify cat_GB new" id="idp672272" style="display: none;" onmouseout="javascript:hide('idp672272');">
<p class="year">2007</p>
<br><a href="javascript:showDetail('idp672272');"><p class="label linked">Sk</p></a><br><p class="name">SeqMonk</p>
</div>
</div>
<div class="elementDetail" id="detail_idp672272" style="display: none;">
<div class="bg"></div>
<div class="fg cat_GB">
<p class="detailClose"><a href="javascript:hideDetail('idp672272');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">SeqMonk</td>
<td class="detailLogo" colspan="2"><img src="http://seqanswers.com/w/images/8/89/SeqMonk.png"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> </td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2007</td>
<td rowspan="7">No impact data available.</td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.bioinformatics.babraham.ac.uk/projects/seqmonk/">http://www.bioinformatics.babraham.ac.uk/projects/seqmonk/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Simon Andrews</td>
</tr>
<tr>
<td class="detailLeft">Key publication:</td>
<td class="detailRight"><a target="_blank" href="index.html"></a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, Mac OS X, Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">GPL v3 or later</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Genome Browsers</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_GB new" onmouseover="javascript:show('idp675168');">
<div class="magnifiable">
<p class="year">10</p>
<br><p class="label linked">Sv</p>
<br><p class="name">Savant</p>
<div class="elementMagnify cat_GB new" id="idp675168" style="display: none;" onmouseout="javascript:hide('idp675168');">
<p class="year">2010</p>
<br><a href="javascript:showDetail('idp675168');"><p class="label linked">Sv</p></a><br><p class="name">Savant</p>
</div>
</div>
<div class="elementDetail" id="detail_idp675168" style="display: none;">
<div class="bg"></div>
<div class="fg cat_GB">
<p class="detailClose"><a href="javascript:hideDetail('idp675168');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">Savant</td>
<td class="detailLogo" colspan="2"><img src="http://genomesavant.com/p/assets/img/logo.png"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Savant is a next-generation genome browser designed for the latest generation of genome data.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2010</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/nar/gks427" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://genomesavant.com/p/savant/index">http://genomesavant.com/p/savant/index</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Fiume M, Smith EJ, Brook A, Strbenac D, Turner B, Mezlini AM, Robinson MD, Wodak SJ, Brudno M.</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/nar/gks427">10.1093/nar/gks427</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">All</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Apache 2.0</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Genome Browsers</td>
</tr>
</table>
</div>
</div>
</td>
  </tr>
<tr>
<td class="empty" colspan="11"></td>
<td class="element cat_DB new" onmouseover="javascript:show('idp680448');">
<div class="magnifiable">
<p class="year">82</p>
<br><p class="label linked">Gk</p>
<br><p class="name">Genbank</p>
<div class="elementMagnify cat_DB new" id="idp680448" style="display: none;" onmouseout="javascript:hide('idp680448');">
<p class="year">1982</p>
<br><a href="javascript:showDetail('idp680448');"><p class="label linked">Gk</p></a><br><p class="name">Genbank</p>
</div>
</div>
<div class="elementDetail" id="detail_idp680448" style="display: none;">
<div class="bg"></div>
<div class="fg cat_DB">
<p class="detailClose"><a href="javascript:hideDetail('idp680448');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">Genbank</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> GenBank ® is the NIH genetic sequence database, an annotated collection of all publicly available DNA sequences (Nucleic Acids Research, 2013 Jan;41(D1):D36-42).  GenBank is part of the International Nucleotide Sequence Database Collaboration , which comprises the DNA DataBank of Japan (DDBJ), the European Molecular Biology Laboratory (EMBL), and GenBank at NCBI. These three organizations exchange data on a daily basis.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">1982</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/nar/gks1195" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/genbank/">http://www.ncbi.nlm.nih.gov/genbank/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight"></td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/nar/gks1195">10.1093/nar/gks1195</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Online</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Free</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Database/Warehouse</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_DB new" onmouseover="javascript:show('idp679696');">
<div class="magnifiable">
<p class="year">05</p>
<br><p class="label linked">Ip</p>
<br><p class="name">Interpro</p>
<div class="elementMagnify cat_DB new" id="idp679696" style="display: none;" onmouseout="javascript:hide('idp679696');">
<p class="year">2005</p>
<br><a href="javascript:showDetail('idp679696');"><p class="label linked">Ip</p></a><br><p class="name">Interpro</p>
</div>
</div>
<div class="elementDetail" id="detail_idp679696" style="display: none;">
<div class="bg"></div>
<div class="fg cat_DB">
<p class="detailClose"><a href="javascript:hideDetail('idp679696');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">Interpro</td>
<td class="detailLogo" colspan="2"><img src="http://www.ebi.ac.uk/interpro/resources/images/logo_interpro_white_342.png"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> InterPro is a resource that provides functional analysis of protein sequences by classifying them into families and predicting the presence of domains and important sites. To classify proteins in this way, InterPro uses predictive models, known as signatures, provided by several different databases (referred to as member databases) that make up the InterPro consortium.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2005</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/nar/gkr948" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.ebi.ac.uk/interpro/">http://www.ebi.ac.uk/interpro/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight"></td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/nar/gkr948">10.1093/nar/gkr948</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Online</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Free</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Database/Warehouse</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_DB new" onmouseover="javascript:show('idp686304');">
<div class="magnifiable">
<p class="year">08</p>
<br><p class="label linked">Cm</p>
<br><p class="name">ChEMBL</p>
<div class="elementMagnify cat_DB new" id="idp686304" style="display: none;" onmouseout="javascript:hide('idp686304');">
<p class="year">2008</p>
<br><a href="javascript:showDetail('idp686304');"><p class="label linked">Cm</p></a><br><p class="name">ChEMBL</p>
</div>
</div>
<div class="elementDetail" id="detail_idp686304" style="display: none;">
<div class="bg"></div>
<div class="fg cat_DB">
<p class="detailClose"><a href="javascript:hideDetail('idp686304');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">ChEMBL</td>
<td class="detailLogo" colspan="2"><img src="https://www.ebi.ac.uk/chembldb/images/new_chembl_logo.png"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> ChEMBL is a database of bioactive drug-like small molecules, it contains 2-D structures, calculated properties (e.g. logP, Molecular Weight, Lipinski Parameters, etc.) and abstracted bioactivities (e.g. binding constants, pharmacology and ADMET data).</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2008</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/nar/gkr777" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="https://www.ebi.ac.uk/chembl/">https://www.ebi.ac.uk/chembl/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Anna Gaulton, Louisa J. Bellis, A. Patricia Bento, Jon Chambers, Mark Davies, Anne Hersey, Yvonne Light, Shaun McGlinchey, David Michalovich, Bissan Al-Lazikani and John P. Overington</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/nar/gkr777">10.1093/nar/gkr777</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Online</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Free</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Database/Warehouse</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_DB new" onmouseover="javascript:show('idp690992');">
<div class="magnifiable">
<p class="year">10</p>
<br><p class="label linked">Cb</p>
<br><p class="name">ChEBI</p>
<div class="elementMagnify cat_DB new" id="idp690992" style="display: none;" onmouseout="javascript:hide('idp690992');">
<p class="year">2010</p>
<br><a href="javascript:showDetail('idp690992');"><p class="label linked">Cb</p></a><br><p class="name">ChEBI</p>
</div>
</div>
<div class="elementDetail" id="detail_idp690992" style="display: none;">
<div class="bg"></div>
<div class="fg cat_DB">
<p class="detailClose"><a href="javascript:hideDetail('idp690992');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">ChEBI</td>
<td class="detailLogo" colspan="2"><img src="http://www.ebi.ac.uk/chebi/images/ChEBI_logo.png"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Chemical Entities of Biological Interest (ChEBI) is a freely available dictionary of molecular entities focused on &#8216;small&#8217; chemical compounds. The term &#8216;molecular entity&#8217; refers to any constitutionally or isotopically distinct atom, molecule, ion, ion pair, radical, radical ion, complex, conformer, etc., identifiable as a separately distinguishable entity. The molecular entities in question are either products of nature or synthetic products used to intervene in the processes of living organisms.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2010</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/nar/gks1146" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.ebi.ac.uk/chebi/">http://www.ebi.ac.uk/chebi/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Hastings, J., de Matos, P., Dekker, A., Ennis, M., Harsha, B., Kale, N., Muthukrishnan, V., Owen, G., Turner, S., Williams, M., and Steinbeck, C.</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/nar/gks1146">10.1093/nar/gks1146</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Online</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Free</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Database/Warehouse</td>
</tr>
</table>
</div>
</div>
</td>
</tr>
 </table>
 <p class="tablelabel acadfree">Tools Free for Academics Only</p>
<table id="acadfree">
  <tr>
  <td class="element cat_ALP " onmouseover="javascript:show('idp695008');">
<div class="magnifiable">
<p class="year">02</p>
<br><p class="label linked">Bl</p>
<br><p class="name">BLAT</p>
<div class="elementMagnify cat_ALP new" id="idp695008" style="display: none;" onmouseout="javascript:hide('idp695008');">
<p class="year">2002</p>
<br><a href="javascript:showDetail('idp695008');"><p class="label linked">Bl</p></a><br><p class="name">BLAT</p>
</div>
</div>
<div class="elementDetail" id="detail_idp695008" style="display: none;">
<div class="bg"></div>
<div class="fg cat_ALP">
<p class="detailClose"><a href="javascript:hideDetail('idp695008');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">BLAT</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> BLAT is a bioinformatics software a tool which performs rapid mRNA/DNA and cross-species protein alignments. BLAT is more accurate and 500 times faster than popular existing tools for mRNA/DNA alignments and 50 times faster for protein alignments at sensitivity settings typically used when comparing vertebrate sequences.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2002</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1101/gr.229202" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.kentinformatics.com/">http://www.kentinformatics.com/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Jim Kent</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1101/gr.229202">10.1101/gr.229202</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, MacOSX</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">For non-commercial use only</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Aligners (pairwise)</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_ALP " onmouseover="javascript:show('idp698320');">
<div class="magnifiable">
<p class="year">00</p>
<br><p class="label linked">Vm</p>
<br><p class="name">VMATCH</p>
<div class="elementMagnify cat_ALP new" id="idp698320" style="display: none;" onmouseout="javascript:hide('idp698320');">
<p class="year">2000</p>
<br><a href="javascript:showDetail('idp698320');"><p class="label linked">Vm</p></a><br><p class="name">VMATCH</p>
</div>
</div>
<div class="elementDetail" id="detail_idp698320" style="display: none;">
<div class="bg"></div>
<div class="fg cat_ALP">
<p class="detailClose"><a href="javascript:hideDetail('idp698320');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">VMATCH</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Vmatch, a versatile software tool for efficiently solving large scale sequence matching tasks.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2000</td>
<td rowspan="7">No impact data available.</td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.vmatch.de/">http://www.vmatch.de/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Stefan Kurtz</td>
</tr>
<tr>
<td class="detailLeft">Key publication:</td>
<td class="detailRight"><a target="_blank" href="index.html"></a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Unix, Linux</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">For non-commercial use only.</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Aligners (pairwise)</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_ST new" onmouseover="javascript:show('idp701856');">
<div class="magnifiable">
<p class="year">11</p>
<br><p class="label linked">Rt</p>
<br><p class="name">Rnnotator</p>
<div class="elementMagnify cat_ST new" id="idp701856" style="display: none;" onmouseout="javascript:hide('idp701856');">
<p class="year">2011</p>
<br><a href="javascript:showDetail('idp701856');"><p class="label linked">Rt</p></a><br><p class="name">Rnnotator</p>
</div>
</div>
<div class="elementDetail" id="detail_idp701856" style="display: none;">
<div class="bg"></div>
<div class="fg cat_ST">
<p class="detailClose"><a href="javascript:hideDetail('idp701856');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">Rnnotator</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Comprehensive annotation and quantification of transcriptomes are outstanding problems in functional genomics. Rnnotator is an automated software pipeline that generates transcript models by de novo assembly of RNA-Seq data without the need for a reference genome. The contigs produced by Rnnotator are highly accurate and reconstruct fulllength genes when transcripts are sequenced sufficiently deep, roughly 30X for a given transcript. Rnnotator was designed to assemble Illumina single or paired-end reads.  Rnnotator is also able to incorporate strand-specific RNA-Seq reads into the assembly in order to further improve the assembly.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2011</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1186/1471-2164-11-663" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="ftp://ftp.jgi-psf.org/pub/rnd/Rnnotator/">ftp://ftp.jgi-psf.org/pub/rnd/Rnnotator/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Martin J, Bruno VM, Fang Z, Meng X, Blow M, Zhang T, Sherlock G, Snyder M, Wang Z</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1186/1471-2164-11-663">10.1186/1471-2164-11-663</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Free for academics</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Sequence Tools</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_ST new" onmouseover="javascript:show('idp701104');">
<div class="magnifiable">
<p class="year">11</p>
<br><p class="label linked">Av</p>
<br><p class="name">ANNOVAR</p>
<div class="elementMagnify cat_ST new" id="idp701104" style="display: none;" onmouseout="javascript:hide('idp701104');">
<p class="year">2011</p>
<br><a href="javascript:showDetail('idp701104');"><p class="label linked">Av</p></a><br><p class="name">ANNOVAR</p>
</div>
</div>
<div class="elementDetail" id="detail_idp701104" style="display: none;">
<div class="bg"></div>
<div class="fg cat_ST">
<p class="detailClose"><a href="javascript:hideDetail('idp701104');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">ANNOVAR</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> ANNOVAR is an efficient software tool to utilize update-to-date information to functionally annotate genetic variants detected from diverse genomes (including human genome hg18, hg19, as well as mouse, worm, fly, yeast and many others).</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2011</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1002/humu.21517" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.openbioinformatics.org/annovar/">http://www.openbioinformatics.org/annovar/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Liu X, JIan X, Boerwinkle E</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1002/humu.21517">10.1002/humu.21517</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Universal</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Free for academics only</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Sequence Tools</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_GPM " onmouseover="javascript:show('idp708240');">
<div class="magnifiable">
<p class="year">99</p>
<br><p class="label linked">Fg</p>
<br><p class="name">FGENESH</p>
<div class="elementMagnify cat_GPM new" id="idp708240" style="display: none;" onmouseout="javascript:hide('idp708240');">
<p class="year">1999</p>
<br><a href="javascript:showDetail('idp708240');"><p class="label linked">Fg</p></a><br><p class="name">FGENESH</p>
</div>
</div>
<div class="elementDetail" id="detail_idp708240" style="display: none;">
<div class="bg"></div>
<div class="fg cat_GPM">
<p class="detailClose"><a href="javascript:hideDetail('idp708240');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">FGENESH</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> HMM-based gene structure prediction (multiple genes, both chains).In recent rice genome sequencing projects, it was cited 'the most successful' (gene finding) program (Yu et al. (2002) Science 296:79) and was used to produce 87% of all high-evidence predicted genes (Goff et al. (2002) Science 296:79).</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">1999</td>
<td rowspan="7">No impact data available.</td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://linux1.softberry.com/berry.phtml?topic=fgenesh&amp;group=programs&amp;subgroup=gfind">http://linux1.softberry.com/berry.phtml?topic=fgenesh&amp;group=programs&amp;subgroup=gfind</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Asaf Salamov and Victor Solovyev</td>
</tr>
<tr>
<td class="detailLeft">Key publication:</td>
<td class="detailRight"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC310882/">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC310882/</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, Mac OS X and Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Free for academics</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Gene Prediction (mRNA)</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_GPM " onmouseover="javascript:show('idp711440');">
<div class="magnifiable">
<p class="year">97</p>
<br><p class="label linked">Gn</p>
<br><p class="name">GENSCAN</p>
<div class="elementMagnify cat_GPM new" id="idp711440" style="display: none;" onmouseout="javascript:hide('idp711440');">
<p class="year">1997</p>
<br><a href="javascript:showDetail('idp711440');"><p class="label linked">Gn</p></a><br><p class="name">GENSCAN</p>
</div>
</div>
<div class="elementDetail" id="detail_idp711440" style="display: none;">
<div class="bg"></div>
<div class="fg cat_GPM">
<p class="detailClose"><a href="javascript:hideDetail('idp711440');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">GENSCAN</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> GENSCAN is an program to identify complete gene structures in genomic DNA.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">1997</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1006/jmbi.1997.0951" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://genes.mit.edu/GENSCAN.html">http://genes.mit.edu/GENSCAN.html</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Chris Burge</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1006/jmbi.1997.0951">10.1006/jmbi.1997.0951</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Free for academic use.</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Gene Prediction (mRNA)</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_DB " onmouseover="javascript:show('idp714544');">
<div class="magnifiable">
<p class="year">06</p>
<br><p class="label linked">Bh</p>
<br><p class="name">BioWarehouse</p>
<div class="elementMagnify cat_DB new" id="idp714544" style="display: none;" onmouseout="javascript:hide('idp714544');">
<p class="year">2006</p>
<br><a href="javascript:showDetail('idp714544');"><p class="label linked">Bh</p></a><br><p class="name">BioWarehouse</p>
</div>
</div>
<div class="elementDetail" id="detail_idp714544" style="display: none;">
<div class="bg"></div>
<div class="fg cat_DB">
<p class="detailClose"><a href="javascript:hideDetail('idp714544');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">BioWarehouse</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> BioWarehouse is an open-source software environment for integrating a set of biological databases into a single physical database management system for data management, mining, and exploration.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2006</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1186/1471-2105-7-170" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://biowarehouse.ai.sri.com/">http://biowarehouse.ai.sri.com/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Peter D Karp, Thomas J Lee, Yannick Pouliot, Valerie Wagner, Tomer Altman, Nan Guo and David Dunkely.</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1186/1471-2105-7-170">10.1186/1471-2105-7-170</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">MOZILLA PUBLIC LICENSE Version 1.1</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Database/Warehouse</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_TK " onmouseover="javascript:show('idp718368');">
<div class="magnifiable">
<p class="year">09</p>
<br><p class="label linked">Ga</p>
<br><p class="name">GATK</p>
<div class="elementMagnify cat_TK new" id="idp718368" style="display: none;" onmouseout="javascript:hide('idp718368');">
<p class="year">2009</p>
<br><a href="javascript:showDetail('idp718368');"><p class="label linked">Ga</p></a><br><p class="name">GATK</p>
</div>
</div>
<div class="elementDetail" id="detail_idp718368" style="display: none;">
<div class="bg"></div>
<div class="fg cat_TK">
<p class="detailClose"><a href="javascript:hideDetail('idp718368');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">GATK</td>
<td class="detailLogo" colspan="2"><img src="http://www.eaglegenomics.com/wp-content/uploads/2012/09/gatk-logo-333.png"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Say you have ten exomes and you want to identify the rare mutations they all have in common &#8211; the GATK can do that. Or you need to know which mutations are specific to a group of patients, as opposed to a healthy cohort &#8211; the GATK can do that too. In fact, the GATK is the industry standard for such analyses.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2009</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1101/gr.107524.110" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.broadinstitute.org/gatk/index.php">http://www.broadinstitute.org/gatk/index.php</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Aaron McKenna, Matthew Hanna, Eric Banks, Andrey Sivachenko, Kristian Cibulskis, Andrew Kernytsky, Kiran Garimella, David Altshuler, Stacey Gabriel, Mark Daly and Mark DePristo</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1101/gr.107524.110">10.1101/gr.107524.110</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, MacOSX and Unix</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">BSD style licence: For non-commercial use only. GATK-lite can used for commercial operations</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">ToolKits and APIs</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_TK " onmouseover="javascript:show('idp718048');">
<div class="magnifiable">
<p class="year">97</p>
<br><p class="label linked">Pm</p>
<br><p class="name">PAML</p>
<div class="elementMagnify cat_TK new" id="idp718048" style="display: none;" onmouseout="javascript:hide('idp718048');">
<p class="year">1997</p>
<br><a href="javascript:showDetail('idp718048');"><p class="label linked">Pm</p></a><br><p class="name">PAML</p>
</div>
</div>
<div class="elementDetail" id="detail_idp718048" style="display: none;">
<div class="bg"></div>
<div class="fg cat_TK">
<p class="detailClose"><a href="javascript:hideDetail('idp718048');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">PAML</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> PAML is a package of programs for phylogenetic analyses of DNA or protein sequences using maximum likelihood.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">1997</td>
<td rowspan="7">No impact data available.</td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://abacus.gene.ucl.ac.uk/software/paml.html">http://abacus.gene.ucl.ac.uk/software/paml.html</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Ziheng Yang</td>
</tr>
<tr>
<td class="detailLeft">Key publication:</td>
<td class="detailRight"><a target="_blank" href="http://abacus.gene.ucl.ac.uk/software/pamlDOC.pdf">http://abacus.gene.ucl.ac.uk/software/pamlDOC.pdf</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">UNIX, Linux, Mac OSX and Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Free for academics</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">ToolKits and APIs</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_SM " onmouseover="javascript:show('idp725056');">
<div class="magnifiable">
<p class="year">96</p>
<br><p class="label linked">Ro</p>
<br><p class="name">ROSETTA</p>
<div class="elementMagnify cat_SM new" id="idp725056" style="display: none;" onmouseout="javascript:hide('idp725056');">
<p class="year">1996</p>
<br><a href="javascript:showDetail('idp725056');"><p class="label linked">Ro</p></a><br><p class="name">ROSETTA</p>
</div>
</div>
<div class="elementDetail" id="detail_idp725056" style="display: none;">
<div class="bg"></div>
<div class="fg cat_SM">
<p class="detailClose"><a href="javascript:hideDetail('idp725056');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">ROSETTA</td>
<td class="detailLogo" colspan="2"><img src="https://www.rosettacommons.org/sites/default/files/logo.png"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> ROSETTA is a software suite for predicting and designing protein structures, protein folding mechanisms, and protein-protein interactions.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">1996</td>
<td rowspan="7">No impact data available.</td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="https://www.rosettacommons.org/">https://www.rosettacommons.org/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">David Baker, et al</td>
</tr>
<tr>
<td class="detailLeft">Key publication:</td>
<td class="detailRight"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/10526365?dopt=Citation">http://www.ncbi.nlm.nih.gov/pubmed/10526365?dopt=Citation</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, MacOSX and Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">WashU Express Licensing Program</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Structure Modelling</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_SV " onmouseover="javascript:show('idp728336');">
<div class="magnifiable">
<p class="year">04</p>
<br><p class="label linked">Cr</p>
<br><p class="name">Chimera</p>
<div class="elementMagnify cat_SV new" id="idp728336" style="display: none;" onmouseout="javascript:hide('idp728336');">
<p class="year">2004</p>
<br><a href="javascript:showDetail('idp728336');"><p class="label linked">Cr</p></a><br><p class="name">Chimera</p>
</div>
</div>
<div class="elementDetail" id="detail_idp728336" style="display: none;">
<div class="bg"></div>
<div class="fg cat_SV">
<p class="detailClose"><a href="javascript:hideDetail('idp728336');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">Chimera</td>
<td class="detailLogo" colspan="2"><img src="http://www.cgl.ucsf.edu/chimera/docs/titleChimera.png"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> UCSF Chimera is a highly extensible program for interactive visualization and analysis of molecular structures and related data, including density maps, supramolecular assemblies, sequence alignments, docking results, trajectories, and conformational ensembles. High-quality images and animations can be generated. Chimera includes complete documentation and several tutorials, and can be downloaded free of charge for academic, government, non-profit, and personal use.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2004</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1002/jcc.20084" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.cgl.ucsf.edu/chimera/">http://www.cgl.ucsf.edu/chimera/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Pettersen, Goddard, Huang, Couch, Greenblatt, Meng, and Ferrin</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1002/jcc.20084">10.1002/jcc.20084</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Windows, Linux, Mac OS X</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Custom, free for academic/non-profit/government use.</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Structure Visualisation</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_OA " onmouseover="javascript:show('idp732288');">
<div class="magnifiable">
<p class="year">03</p>
<br><p class="label linked">Dl</p>
<br><p class="name">Dali</p>
<div class="elementMagnify cat_OA new" id="idp732288" style="display: none;" onmouseout="javascript:hide('idp732288');">
<p class="year">2003</p>
<br><a href="javascript:showDetail('idp732288');"><p class="label linked">Dl</p></a><br><p class="name">Dali</p>
</div>
</div>
<div class="elementDetail" id="detail_idp732288" style="display: none;">
<div class="bg"></div>
<div class="fg cat_OA">
<p class="detailClose"><a href="javascript:hideDetail('idp732288');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">Dali</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> This server computes optimal and suboptimal structural alignments between two protein structures using the DaliLite -pairwise option (version 3.1). It will compare all chains in the first structure against all chains in the second (unless chain IDs are given). The resulting superimposed coordinate files can be downloaded or viewed interactively in Jmol (requires Java).</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2003</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1016/j.sbi.2009.04.003" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://ekhidna.biocenter.helsinki.fi/dali_lite/start">http://ekhidna.biocenter.helsinki.fi/dali_lite/start</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Hasegawa, Holm</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1016/j.sbi.2009.04.003">10.1016/j.sbi.2009.04.003</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Custom licence, commercial users must pay</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Other Aligners</td>
</tr>
</table>
</div>
</div>
</td>
 <td class="element cat_AGL " onmouseover="javascript:show('idp735856');">
<div class="magnifiable">
<p class="year">01</p>
<br><p class="label linked">Pc</p>
<br><p class="name">PCAP</p>
<div class="elementMagnify cat_AGL new" id="idp735856" style="display: none;" onmouseout="javascript:hide('idp735856');">
<p class="year">2001</p>
<br><a href="javascript:showDetail('idp735856');"><p class="label linked">Pc</p></a><br><p class="name">PCAP</p>
</div>
</div>
<div class="elementDetail" id="detail_idp735856" style="display: none;">
<div class="bg"></div>
<div class="fg cat_AGL">
<p class="detailClose"><a href="javascript:hideDetail('idp735856');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">PCAP</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Parallel contig assembly program (PCAP) for assembling tens of millions of reads into long sequences.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2001</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1101/gr.1390403" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://seq.cs.iastate.edu/pcap.html">http://seq.cs.iastate.edu/pcap.html</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Xiaoqui Huang</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1101/gr.1390403">10.1101/gr.1390403</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, Mac OS X</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Free for academics</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Assemblers Genomic (long read)</td>
</tr>
</table>
</div>
</div>
</td>
  </tr>
<tr>
  <td class="element cat_ALP " onmouseover="javascript:show('idp740352');">
<div class="magnifiable">
<p class="year">92</p>
<br><p class="label linked">Cm</p>
<br><p class="name">CROSSMATCH</p>
<div class="elementMagnify cat_ALP new" id="idp740352" style="display: none;" onmouseout="javascript:hide('idp740352');">
<p class="year">1992</p>
<br><a href="javascript:showDetail('idp740352');"><p class="label linked">Cm</p></a><br><p class="name">CROSSMATCH</p>
</div>
</div>
<div class="elementDetail" id="detail_idp740352" style="display: none;">
<div class="bg"></div>
<div class="fg cat_ALP">
<p class="detailClose"><a href="javascript:hideDetail('idp740352');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">CROSSMATCH</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Cross_match is a program for rapid protein and nucleic acid sequence comparison and database search, based on an efficient implementation of the Smith-Waterman-Gotoh algorithm (Smith &amp; Waterman 1981, Gotoh 1982). By revising the recursion relations slightly and making efficient use of word-packing, we have been able to significantly reduce the number of machine instructions executed per Smith-Waterman matrix cell, resulting in a speed enhancement of 2-3 fold over the previous best implementation (Pearson and Miller, 1992).</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">1992</td>
<td rowspan="7">No impact data available.</td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.incogen.com/public_documents/vibe/details/crossmatch.html">http://www.incogen.com/public_documents/vibe/details/crossmatch.html</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Phil Green</td>
</tr>
<tr>
<td class="detailLeft">Key publication:</td>
<td class="detailRight"><a target="_blank" href="index.html"></a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Unix</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">For non-commercial use only.</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Aligners (pairwise)</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_ALP " onmouseover="javascript:show('idp739600');">
<div class="magnifiable">
<p class="year">01</p>
<br><p class="label linked">Sh</p>
<br><p class="name">SSAHA2</p>
<div class="elementMagnify cat_ALP new" id="idp739600" style="display: none;" onmouseout="javascript:hide('idp739600');">
<p class="year">2001</p>
<br><a href="javascript:showDetail('idp739600');"><p class="label linked">Sh</p></a><br><p class="name">SSAHA2</p>
</div>
</div>
<div class="elementDetail" id="detail_idp739600" style="display: none;">
<div class="bg"></div>
<div class="fg cat_ALP">
<p class="detailClose"><a href="javascript:hideDetail('idp739600');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">SSAHA2</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> SSAHA2 (Sequence Search and Alignment by Hashing Algorithm) is a pairwise sequence alignment program designed for the efficient mapping of sequencing reads onto genomic reference sequences. SSAHA2 reads of most sequencing platforms (ABI-Sanger, Roche 454, Illumina-Solexa) and a range of output formats (SAM, CIGAR, PSL etc.) are supported. A pile-up pipeline for analysis and genotype calling is available as a separate package.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2001</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1101/gr.194201" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.sanger.ac.uk/resources/software/ssaha2">http://www.sanger.ac.uk/resources/software/ssaha2</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Adam Spargo, Zemin Ning, James Mullikin, Tony Cox and Nikolai Ivanov.</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1101/gr.194201">10.1101/gr.194201</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, MacOSX</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">All rights reserved.</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Aligners (pairwise)</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_ALP new" onmouseover="javascript:show('idp746288');">
<div class="magnifiable">
<p class="year">00</p>
<br><p class="label unlinked">Md</p>
<br><p class="name">MaskerAid</p>
<div class="elementMagnify cat_ALP new" id="idp746288" style="display: none;" onmouseout="javascript:hide('idp746288');">
<p class="year">2000</p>
<br><a href="javascript:showDetail('idp746288');"><p class="label unlinked">Md</p></a><br><p class="name">MaskerAid</p>
</div>
</div>
<div class="elementDetail" id="detail_idp746288" style="display: none;">
<div class="bg"></div>
<div class="fg cat_ALP">
<p class="detailClose"><a href="javascript:hideDetail('idp746288');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">MaskerAid</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Identifying and masking repetitive elements is usually the first step when analyzing vertebrate genomic sequence. Current repeat identification software is sensitive but slow, creating a costly bottleneck in large-scale analyses. We have developed MaskerAid, a software enhancement to RepeatMasker that increased the speed of masking more than 30-fold at the most sensitive setting.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2000</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/bioinformatics/16.11.1040" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="index.html"></a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Bedell JA, Korf I, Gish W.</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/bioinformatics/16.11.1040">10.1093/bioinformatics/16.11.1040</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Universal</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Free for academics only</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Aligners (pairwise)</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_AGS " onmouseover="javascript:show('idp749440');">
<div class="magnifiable">
<p class="year">09</p>
<br><p class="label linked">Ab</p>
<br><p class="name">ABYSS</p>
<div class="elementMagnify cat_AGS new" id="idp749440" style="display: none;" onmouseout="javascript:hide('idp749440');">
<p class="year">2009</p>
<br><a href="javascript:showDetail('idp749440');"><p class="label linked">Ab</p></a><br><p class="name">ABYSS</p>
</div>
</div>
<div class="elementDetail" id="detail_idp749440" style="display: none;">
<div class="bg"></div>
<div class="fg cat_AGS">
<p class="detailClose"><a href="javascript:hideDetail('idp749440');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">ABYSS</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> ABySS(Assembly By Short Sequencing) is a de novo, parallel, paired-end sequence assembler that is designed for short reads.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2009</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1101/gr.089532.108" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.bcgsc.ca/platform/bioinfo/software/abyss">http://www.bcgsc.ca/platform/bioinfo/software/abyss</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Shuan Jackman, Tony Raymond and Jared Simpson</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1101/gr.089532.108">10.1101/gr.089532.108</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, MacOSX</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">BCCA Academic use</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Assemblers Genomic (short read)</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_GPM " onmouseover="javascript:show('idp752800');">
<div class="magnifiable">
<p class="year">11</p>
<br><p class="label linked">Mk</p>
<br><p class="name">MAKER</p>
<div class="elementMagnify cat_GPM new" id="idp752800" style="display: none;" onmouseout="javascript:hide('idp752800');">
<p class="year">2011</p>
<br><a href="javascript:showDetail('idp752800');"><p class="label linked">Mk</p></a><br><p class="name">MAKER</p>
</div>
</div>
<div class="elementDetail" id="detail_idp752800" style="display: none;">
<div class="bg"></div>
<div class="fg cat_GPM">
<p class="detailClose"><a href="javascript:hideDetail('idp752800');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">MAKER</td>
<td class="detailLogo" colspan="2"><img src="http://www.eaglegenomics.com/wp-content/uploads/2012/09/Maker-Logo.png"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> MAKER is a portable and easily configurable genome annotation pipeline. It's purpose is to allow smaller eukaryotic and prokaryotic genome projects to independently annotate their genomes and to create genome databases.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2011</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1101/gr.6743907" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.yandell-lab.org/software/maker.html">http://www.yandell-lab.org/software/maker.html</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Moj Eram and Mark Yandell</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1101/gr.6743907">10.1101/gr.6743907</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Academic use under Artistic licence 2.0</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Gene Prediction (mRNA)</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_GPN " onmouseover="javascript:show('idp756480');">
<div class="magnifiable">
<p class="year">08</p>
<br><p class="label linked">Mr</p>
<br><p class="name">MiRDeep</p>
<div class="elementMagnify cat_GPN new" id="idp756480" style="display: none;" onmouseout="javascript:hide('idp756480');">
<p class="year">2008</p>
<br><a href="javascript:showDetail('idp756480');"><p class="label linked">Mr</p></a><br><p class="name">MiRDeep</p>
</div>
</div>
<div class="elementDetail" id="detail_idp756480" style="display: none;">
<div class="bg"></div>
<div class="fg cat_GPN">
<p class="detailClose"><a href="javascript:hideDetail('idp756480');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">MiRDeep</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> The miRDeep package was developed to discover active known or novel miRNAs from deep sequencing data (Solexa/Illumina, 454). The package consists of everything you need to analyze your own deep sequencing data after removal of ligation adapters: a number of scripts to preprocess the mapped data, and the core miRDeep algorithm that will analyze and score these data. </td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2008</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1038/nbt1394" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://seqanswers.com/wiki/MiRDeep">http://seqanswers.com/wiki/MiRDeep</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Marc Friedlander and Nikolaus Rajewsky</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1038/nbt1394">10.1038/nbt1394</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Free for non-commercial use.</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Gene Prediction (ncRNA)</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_GB " onmouseover="javascript:show('idp760384');">
<div class="magnifiable">
<p class="year">01</p>
<br><p class="label linked">Uc</p>
<br><p class="name">UCSC</p>
<div class="elementMagnify cat_GB new" id="idp760384" style="display: none;" onmouseout="javascript:hide('idp760384');">
<p class="year">2001</p>
<br><a href="javascript:showDetail('idp760384');"><p class="label linked">Uc</p></a><br><p class="name">UCSC</p>
</div>
</div>
<div class="elementDetail" id="detail_idp760384" style="display: none;">
<div class="bg"></div>
<div class="fg cat_GB">
<p class="detailClose"><a href="javascript:hideDetail('idp760384');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">UCSC</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> The UCSC Genome Browser and associated tools provide access to the sequences of human and other vertebrate genomes, as well as selected invertebrate model organisms.  The Browser shows annotations of many types mapped to the coordinates of the reference assembly for each organism.  Annotations include the locations of RNAs from the organism and from other organisms, DNA modifications (ENCODE project on human and mouse), SNPs, repeats and structural  variations.  The BLAT alignment tool allows alignment of sequences to the reference assemblies. The Table Browser offers an interface for access to the large underlying database.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2001</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/nar/gks1048" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://genome.ucsc.edu/cgi-bin/hgTracks?org=human">http://genome.ucsc.edu/cgi-bin/hgTracks?org=human</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Meyer LR, Zweig AS, Hinrichs AS, Karolchik D, Kuhn RM, Wong&#8232;M, Sloan CA, Rosenbloom KR, Roe G, Rhead B, Raney BJ, Pohl A, Malladi&#8232;VS, Li CH, Lee BT, Learned K, Kirkup V, Hsu F, Heitner S, Harte RA,&#8232;Haeussler M, Guruvadoo L, Goldman M, Giardine BM, Fujita PA, Dreszer&#8232;TR, Diekhans M, Cline MS, Clawson H, Barber GP, Haussler D, and Kent&#8232;WJ.</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/nar/gks1048">10.1093/nar/gks1048</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, Mac OS X and Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Free with some exceptions</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Genome Browsers</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_TK " onmouseover="javascript:show('idp760064');">
<div class="magnifiable">
<p class="year">08</p>
<br><p class="label linked">Pt</p>
<br><p class="name">Pathway Tools</p>
<div class="elementMagnify cat_TK new" id="idp760064" style="display: none;" onmouseout="javascript:hide('idp760064');">
<p class="year">2008</p>
<br><a href="javascript:showDetail('idp760064');"><p class="label linked">Pt</p></a><br><p class="name">Pathway Tools</p>
</div>
</div>
<div class="elementDetail" id="detail_idp760064" style="display: none;">
<div class="bg"></div>
<div class="fg cat_TK">
<p class="detailClose"><a href="javascript:hideDetail('idp760064');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">Pathway Tools</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Pathway Tools is a comprehensive software environment that supports construction of organism-specific databases called Pathway/Genome Databases (PGDBs).</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2008</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/bib/bbp043" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://gmod.org/wiki/Pathway_Tools">http://gmod.org/wiki/Pathway_Tools</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Peter D. Karp</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/bib/bbp043">10.1093/bib/bbp043</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, Windows, and Macintosh</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Free to academics</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">ToolKits and APIs</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_TK " onmouseover="javascript:show('idp767024');">
<div class="magnifiable">
<p class="year">11</p>
<br><p class="label linked">Sw</p>
<br><p class="name">sRNA workbench UEA</p>
<div class="elementMagnify cat_TK new" id="idp767024" style="display: none;" onmouseout="javascript:hide('idp767024');">
<p class="year">2011</p>
<br><a href="javascript:showDetail('idp767024');"><p class="label linked">Sw</p></a><br><p class="name">sRNA workbench UEA</p>
</div>
</div>
<div class="elementDetail" id="detail_idp767024" style="display: none;">
<div class="bg"></div>
<div class="fg cat_TK">
<p class="detailClose"><a href="javascript:hideDetail('idp767024');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">sRNA workbench UEA</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> The UEA sRNA workbench is a new simple to use, downloadable sRNA software package based on algorithms developed for the original UEA sRNA Toolkit that will perform a complete analysis of single or multiple-sample sRNA datasets from both plants and animals to identify interesting landmarks (such as detection of novel micro RNA sequences) or other tasks such as profiling small RNA expression patterns in genetic data.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2011</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/bioinformatics/bts311" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://srna-workbench.cmp.uea.ac.uk/">http://srna-workbench.cmp.uea.ac.uk/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Dr. Matthew Stocks, Dr Daniel Mapleson, Dr. Hugh Woolfenden, Mr. Leighton Folkes and Irina-Ioana Mohorianu.</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/bioinformatics/bts311">10.1093/bioinformatics/bts311</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, Mac OS X and Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">For non-commercial use</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">ToolKits and APIs</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_SM " onmouseover="javascript:show('idp770624');">
<div class="magnifiable">
<p class="year">03</p>
<br><p class="label linked">Mo</p>
<br><p class="name">Modeller</p>
<div class="elementMagnify cat_SM new" id="idp770624" style="display: none;" onmouseout="javascript:hide('idp770624');">
<p class="year">2003</p>
<br><a href="javascript:showDetail('idp770624');"><p class="label linked">Mo</p></a><br><p class="name">Modeller</p>
</div>
</div>
<div class="elementDetail" id="detail_idp770624" style="display: none;">
<div class="bg"></div>
<div class="fg cat_SM">
<p class="detailClose"><a href="javascript:hideDetail('idp770624');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">Modeller</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> MODELLER is used for homology or comparative modeling of protein three-dimensional structures (1,2). The user provides an alignment of a sequence to be modeled with known related structures and MODELLER automatically calculates a model containing all non-hydrogen atoms. MODELLER implements comparative protein structure modeling by satisfaction of spatial restraints (3,4), and can perform many additional tasks, including de novo modeling of loops in protein structures, optimization of various models of protein structure with respect to a flexibly defined objective function, multiple alignment of protein sequences and/or structures, clustering, searching of sequence databases, comparison of protein structures, etc.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2003</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1016/S0076-6879(03)74020-8" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://salilab.org/modeller/">http://salilab.org/modeller/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Ben Webb, et al</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1016/S0076-6879(03)74020-8">10.1016/S0076-6879(03)74020-8</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Unix, Linux, MacOSX and Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Free for non-commercial non-government use.</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Structure Modelling</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_SV " onmouseover="javascript:show('idp774464');">
<div class="magnifiable">
<p class="year">XX</p>
<br><p class="label linked">Pm</p>
<br><p class="name">PyMOL</p>
<div class="elementMagnify cat_SV new" id="idp774464" style="display: none;" onmouseout="javascript:hide('idp774464');">
<p class="year">XX</p>
<br><a href="javascript:showDetail('idp774464');"><p class="label linked">Pm</p></a><br><p class="name">PyMOL</p>
</div>
</div>
<div class="elementDetail" id="detail_idp774464" style="display: none;">
<div class="bg"></div>
<div class="fg cat_SV">
<p class="detailClose"><a href="javascript:hideDetail('idp774464');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">PyMOL</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> PyMOL is a user-sponsored molecular visualization system on an open-source foundation. Please support development of this open, effective, and affordable software by purchasing an incentive copy, which is pre-built and comes with maintenance and support.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">XX</td>
<td rowspan="7">No impact data available.</td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.pymol.org/">http://www.pymol.org/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Warren Lyford DeLano</td>
</tr>
<tr>
<td class="detailLeft">Key publication:</td>
<td class="detailRight"><a target="_blank" href="index.html"></a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Windows, Mac OS X, Linux</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Custom, older versions free for educational use</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Structure Visualisation</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_ALS " onmouseover="javascript:show('idp777312');">
<div class="magnifiable">
<p class="year">09</p>
<br><p class="label linked">No</p>
<br><p class="name">NOVOALIGN</p>
<div class="elementMagnify cat_ALS new" id="idp777312" style="display: none;" onmouseout="javascript:hide('idp777312');">
<p class="year">2009</p>
<br><a href="javascript:showDetail('idp777312');"><p class="label linked">No</p></a><br><p class="name">NOVOALIGN</p>
</div>
</div>
<div class="elementDetail" id="detail_idp777312" style="display: none;">
<div class="bg"></div>
<div class="fg cat_ALS">
<p class="detailClose"><a href="javascript:hideDetail('idp777312');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">NOVOALIGN</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Novoalign is a short-read mapper designed to be fast and sensitive on small to large genome databases. It's primary design is based on the use of read quality information and the need to assemble genomes from resequencing experiments. Novoalign supports fragment, paired-end and mate-pair reads from major sequencing platforms such as Illumina, SOLiD and Roche 454. NovoalignCS is the version of Novoalign developed for SOLiD colourspace reads.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2009</td>
<td rowspan="7">No impact data available.</td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.novocraft.com/main/index.php">http://www.novocraft.com/main/index.php</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Colin Hercus</td>
</tr>
<tr>
<td class="detailLeft">Key publication:</td>
<td class="detailRight"><a target="_blank" href="index.html"></a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, MacOSX</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Not-for-Profit Licence</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Aligners (short read)</td>
</tr>
</table>
</div>
</div>
</td>
</tr>
 </table>
 <p class="tablelabel commercial">Commercial Tools</p>
<table id="commercial">
  <tr>
  <td class="element cat_ALP " onmouseover="javascript:show('idp781840');">
<div class="magnifiable">
<p class="year">08</p>
<br><p class="label linked">Zm</p>
<br><p class="name">ZOOM</p>
<div class="elementMagnify cat_ALP new" id="idp781840" style="display: none;" onmouseout="javascript:hide('idp781840');">
<p class="year">2008</p>
<br><a href="javascript:showDetail('idp781840');"><p class="label linked">Zm</p></a><br><p class="name">ZOOM</p>
</div>
</div>
<div class="elementDetail" id="detail_idp781840" style="display: none;">
<div class="bg"></div>
<div class="fg cat_ALP">
<p class="detailClose"><a href="javascript:hideDetail('idp781840');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">ZOOM</td>
<td class="detailLogo" colspan="2"><img src="http://www.eaglegenomics.com/wp-content/uploads/2012/08/Zoom.jpg"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Next generation sequencing researchers require accurate results, fast. ZOOM exceeds these needs. Using ZOOM, zillions of short reads are mapped back to reference genomes, including post-analysis at unparalleled in speed, at full sensitivity. Throughout the next few pages, ZOOM's features, capabilities, even benchmarking will be presented, to provide researchers with as much information about the software as possible. For unconvinced until truly relevant results are presented, ZOOM offers a free 30-day licenses to process private and public data from Illumina/SOLiD instruments.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2008</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/bioinformatics/btn416" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.bioinfor.com/zoom/general/overview.html">http://www.bioinfor.com/zoom/general/overview.html</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Lin H., Zhang Z., Zhang MQ., Ma B., Li M.</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/bioinformatics/btn416">10.1093/bioinformatics/btn416</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Unix, Linux</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight"></td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Aligners (pairwise)</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_ALS " onmouseover="javascript:show('idp781088');">
<div class="magnifiable">
<p class="year">XX</p>
<br><p class="label linked">Ca</p>
<br><p class="name">Casava</p>
<div class="elementMagnify cat_ALS new" id="idp781088" style="display: none;" onmouseout="javascript:hide('idp781088');">
<p class="year">XX</p>
<br><a href="javascript:showDetail('idp781088');"><p class="label linked">Ca</p></a><br><p class="name">Casava</p>
</div>
</div>
<div class="elementDetail" id="detail_idp781088" style="display: none;">
<div class="bg"></div>
<div class="fg cat_ALS">
<p class="detailClose"><a href="javascript:hideDetail('idp781088');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">Casava</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Illumina's Consensus Assessment of Sequence and Variation (CASAVA) software captures summary information for resequencing and counting studies and places the data in a compact structure for visualization within GenomeStudio Software or publicly available analysis tools. CASAVA can create genomic builds, call SNPs, detects indels, and count reads from data generated from one or more runs of the Genome Analyzer across a broad range of sequencing applications.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">XX</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1186/gb-2011-12-11-r112" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.illumina.com/software/genome_analyzer_software.ilmn">http://www.illumina.com/software/genome_analyzer_software.ilmn</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Illumina</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1186/gb-2011-12-11-r112">10.1186/gb-2011-12-11-r112</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Commercial tool</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Aligners (short read)</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_AGL " onmouseover="javascript:show('idp788528');">
<div class="magnifiable">
<p class="year">07</p>
<br><p class="label linked">Oz</p>
<br><p class="name">OLIGOZIP</p>
<div class="elementMagnify cat_AGL new" id="idp788528" style="display: none;" onmouseout="javascript:hide('idp788528');">
<p class="year">2007</p>
<br><a href="javascript:showDetail('idp788528');"><p class="label linked">Oz</p></a><br><p class="name">OLIGOZIP</p>
</div>
</div>
<div class="elementDetail" id="detail_idp788528" style="display: none;">
<div class="bg"></div>
<div class="fg cat_AGL">
<p class="detailClose"><a href="javascript:hideDetail('idp788528');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">OLIGOZIP</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> OligoZip tool for processing short reads generated by Illumina-Solexa or similar sequencing machines. The tool does de novo reconstruction of genomc sequence, reconstruction of genomic sequences based on a reference genome for same or close species, mutation profiling and SNP discovery in a given set of genes.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2007</td>
<td rowspan="7">No impact data available.</td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://linux1.softberry.com/berry.phtml?topic=OligoZip">http://linux1.softberry.com/berry.phtml?topic=OligoZip</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Victor Solovyev, Igor Seledtsov, Peter Kosarev, Vladimir Molodstov</td>
</tr>
<tr>
<td class="detailLeft">Key publication:</td>
<td class="detailRight"><a target="_blank" href="http://2010symposium.crg.es/perfiles/victor-solovyev/">http://2010symposium.crg.es/perfiles/victor-solovyev/</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Commercial licence</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Assemblers Genomic (long read)</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_AGL " onmouseover="javascript:show('idp791856');">
<div class="magnifiable">
<p class="year">05</p>
<br><p class="label linked">Ne</p>
<br><p class="name">NEWBLER</p>
<div class="elementMagnify cat_AGL new" id="idp791856" style="display: none;" onmouseout="javascript:hide('idp791856');">
<p class="year">2005</p>
<br><a href="javascript:showDetail('idp791856');"><p class="label linked">Ne</p></a><br><p class="name">NEWBLER</p>
</div>
</div>
<div class="elementDetail" id="detail_idp791856" style="display: none;">
<div class="bg"></div>
<div class="fg cat_AGL">
<p class="detailClose"><a href="javascript:hideDetail('idp791856');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">NEWBLER</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> The 454 Newbler Assembly is a de novo assembler that allows you to assemble your raw reads. Newbler was designed with the intent to be used with 454 data.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2005</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1186/1471-2164-11-571" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://wiki.genomequest.com/index.php/Newbler_Assembler">http://wiki.genomequest.com/index.php/Newbler_Assembler</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Florent E Angly 'Roche'</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1186/1471-2164-11-571">10.1186/1471-2164-11-571</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, Mac OS X</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Commercial tool</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Assemblers Genomic (long read)</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_ST new" onmouseover="javascript:show('idp795280');">
<div class="magnifiable">
<p class="year">08</p>
<br><p class="label linked">Gd</p>
<br><p class="name">Genedata</p>
<div class="elementMagnify cat_ST new" id="idp795280" style="display: none;" onmouseout="javascript:hide('idp795280');">
<p class="year">2008</p>
<br><a href="javascript:showDetail('idp795280');"><p class="label linked">Gd</p></a><br><p class="name">Genedata</p>
</div>
</div>
<div class="elementDetail" id="detail_idp795280" style="display: none;">
<div class="bg"></div>
<div class="fg cat_ST">
<p class="detailClose"><a href="javascript:hideDetail('idp795280');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">Genedata</td>
<td class="detailLogo" colspan="2"><img src="http://www.genedata.com/fileadmin/templates/img/genedata_logo.gif"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> The Genedata portfolio of advanced software solutions is built on an open, enterprise-level client-server architecture, which supports large-scale experimental research processes. Genedata solutions deliver a high degree of built-in-business logic, integration, performance, and scalability to researchers, scientists, and managers.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2008</td>
<td rowspan="7">No impact data available.</td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.genedata.com/">http://www.genedata.com/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight"></td>
</tr>
<tr>
<td class="detailLeft">Key publication:</td>
<td class="detailRight"><a target="_blank" href="index.html"></a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Various</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Commercial</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Sequence Tools</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_ST new" onmouseover="javascript:show('idp798560');">
<div class="magnifiable">
<p class="year">03</p>
<br><p class="label linked">It</p>
<br><p class="name">Integromics</p>
<div class="elementMagnify cat_ST new" id="idp798560" style="display: none;" onmouseout="javascript:hide('idp798560');">
<p class="year">2003</p>
<br><a href="javascript:showDetail('idp798560');"><p class="label linked">It</p></a><br><p class="name">Integromics</p>
</div>
</div>
<div class="elementDetail" id="detail_idp798560" style="display: none;">
<div class="bg"></div>
<div class="fg cat_ST">
<p class="detailClose"><a href="javascript:hideDetail('idp798560');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">Integromics</td>
<td class="detailLogo" colspan="2"><img src="https://www.integromics.com/wp-content/uploads/2012/02/Integromics_logo.png"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Integromics S.L. provides state-of-the-art bioinformatics software solutions for data management and data analysis in genomics and proteomics. Our advanced software technology, supported by an experienced and knowledgeable team of life science professionals, enhances your ability to effectively organize your data, perform validated analyses, discover new biomedical knowledge and gain well-founded insights into various areas of research.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2003</td>
<td rowspan="7">No impact data available.</td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="https://www.integromics.com/">https://www.integromics.com/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight"></td>
</tr>
<tr>
<td class="detailLeft">Key publication:</td>
<td class="detailRight"><a target="_blank" href="index.html"></a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Various</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Commercial</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Sequence Tools</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_ST new" onmouseover="javascript:show('idp801920');">
<div class="magnifiable">
<p class="year">97</p>
<br><p class="label linked">Sr</p>
<br><p class="name">SRS</p>
<div class="elementMagnify cat_ST new" id="idp801920" style="display: none;" onmouseout="javascript:hide('idp801920');">
<p class="year">1997</p>
<br><a href="javascript:showDetail('idp801920');"><p class="label linked">Sr</p></a><br><p class="name">SRS</p>
</div>
</div>
<div class="elementDetail" id="detail_idp801920" style="display: none;">
<div class="bg"></div>
<div class="fg cat_ST">
<p class="detailClose"><a href="javascript:hideDetail('idp801920');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">SRS</td>
<td class="detailLogo" colspan="2"><img src="http://www.instem.com/images/products/srs.png"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> SRS is a proven and scalable data integration platform that is used at over 300 commercial and academic sites delivering genomic data to thousands of users. SRS facilitates the rapid development of applications and algorithms, as well as bioinformatics portals for the Internet or Intranet, making the data efficiently available to entire organizations.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">1997</td>
<td rowspan="7">No impact data available.</td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.instem.com/solutions/srs.html">http://www.instem.com/solutions/srs.html</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight"></td>
</tr>
<tr>
<td class="detailLeft">Key publication:</td>
<td class="detailRight"><a target="_blank" href="index.html"></a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Online</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Commercial</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Sequence Tools</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_ST new" onmouseover="javascript:show('idp805184');">
<div class="magnifiable">
<p class="year">10</p>
<br><p class="label linked">Gp</p>
<br><p class="name">GenePool</p>
<div class="elementMagnify cat_ST new" id="idp805184" style="display: none;" onmouseout="javascript:hide('idp805184');">
<p class="year">2010</p>
<br><a href="javascript:showDetail('idp805184');"><p class="label linked">Gp</p></a><br><p class="name">GenePool</p>
</div>
</div>
<div class="elementDetail" id="detail_idp805184" style="display: none;">
<div class="bg"></div>
<div class="fg cat_ST">
<p class="detailClose"><a href="javascript:hideDetail('idp805184');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">GenePool</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> GenePool is a software system for analyzing and visualizing large-scale genomic data for biomarker discovery in clinical research and development.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2010</td>
<td rowspan="7">No impact data available.</td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.stationxinc.com/genepool-overview.html">http://www.stationxinc.com/genepool-overview.html</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight"></td>
</tr>
<tr>
<td class="detailLeft">Key publication:</td>
<td class="detailRight"><a target="_blank" href="index.html"></a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">All</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Commercial</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Sequence Tools</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_ST new" onmouseover="javascript:show('idp808112');">
<div class="magnifiable">
<p class="year">03</p>
<br><p class="label linked">Mh</p>
<br><p class="name">Match</p>
<div class="elementMagnify cat_ST new" id="idp808112" style="display: none;" onmouseout="javascript:hide('idp808112');">
<p class="year">2003</p>
<br><a href="javascript:showDetail('idp808112');"><p class="label linked">Mh</p></a><br><p class="name">Match</p>
</div>
</div>
<div class="elementDetail" id="detail_idp808112" style="display: none;">
<div class="bg"></div>
<div class="fg cat_ST">
<p class="detailClose"><a href="javascript:hideDetail('idp808112');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">Match</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Match is a weight matrix-based program for predicting transcription factor binding sites (TFBS) in DNA sequences. It uses a library of positional weight matrices from TRANSFAC® Public 6.0.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2003</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1093/nar/gkg585" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.gene-regulation.com/pub/programs.html">http://www.gene-regulation.com/pub/programs.html</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Kel, A. E.; Goessling, E.; Reuter, I.; Cheremushkin, E.; Kel-Margoulis, O. V.; Wingender, E.</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1093/nar/gkg585">10.1093/nar/gkg585</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Online (free), Linux, Unix, Windows (commercial)</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Basic version free, Updated data commercial</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Sequence Tools</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_WF new" onmouseover="javascript:show('idp811504');">
<div class="magnifiable">
<p class="year">10</p>
<br><p class="label linked">Gp</p>
<br><p class="name">geneXplain</p>
<div class="elementMagnify cat_WF new" id="idp811504" style="display: none;" onmouseout="javascript:hide('idp811504');">
<p class="year">2010</p>
<br><a href="javascript:showDetail('idp811504');"><p class="label linked">Gp</p></a><br><p class="name">geneXplain</p>
</div>
</div>
<div class="elementDetail" id="detail_idp811504" style="display: none;">
<div class="bg"></div>
<div class="fg cat_WF">
<p class="detailClose"><a href="javascript:hideDetail('idp811504');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">geneXplain</td>
<td class="detailLogo" colspan="2"><img src="http://genexplain.com/sites/all/themes/genexplain/logo.png"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> The geneXplain platform is an online toolbox and workflow management system for a broad range of bioinformatic and systems biology applications. The individual modules, or Bricks, are unified under a standardized interface, with a consistent look-and-feel and can flexibly be put together to comprehensive workflows. The workflow management is intuitively handled through a simple drag-and-drop system. With this system, you can edit the predefined workflows or compose your own workflows from scratch. Own Bricks can easily be added as scripts or plug-ins and can be used in combination with pre-existing analyses.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2010</td>
<td rowspan="7">No impact data available.</td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://genexplain.com/genexplain-platform-1">http://genexplain.com/genexplain-platform-1</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight"></td>
</tr>
<tr>
<td class="detailLeft">Key publication:</td>
<td class="detailRight"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3186619/">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3186619/</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Online</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Commercial</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Workflows</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_WF " onmouseover="javascript:show('idp815680');">
<div class="magnifiable">
<p class="year">99</p>
<br><p class="label linked">Pp</p>
<br><p class="name">PIPELINE PILOT</p>
<div class="elementMagnify cat_WF new" id="idp815680" style="display: none;" onmouseout="javascript:hide('idp815680');">
<p class="year">1999</p>
<br><a href="javascript:showDetail('idp815680');"><p class="label linked">Pp</p></a><br><p class="name">PIPELINE PILOT</p>
</div>
</div>
<div class="elementDetail" id="detail_idp815680" style="display: none;">
<div class="bg"></div>
<div class="fg cat_WF">
<p class="detailClose"><a href="javascript:hideDetail('idp815680');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">PIPELINE PILOT</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Pipeline Pilot is Accelrys&#8217; graphical scientific workflow authoring application. Built on the scientifically aware Accelrys Enterprise Platform, Pipeline Pilot enables scientists to rapidly create, test and publish scientific services that automate the process of accessing, analyzing and reporting scientific data, either for the scientist&#8217;s personal use or for sharing across the scientific community.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">1999</td>
<td rowspan="7"><div class="impactstory-embed" data-id="10.1007/s10822-012-9577-7" data-id-type="doi" data-api-key="elementsofbioinformatics-7vuvfr"></div></td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://accelrys.com/products/pipeline-pilot/">http://accelrys.com/products/pipeline-pilot/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">SciTegic</td>
</tr>
<tr>
<td class="detailLeft">DOI:</td>
<td class="detailRight"><a target="_blank" href="http://dx.doi.org/10.1007/s10822-012-9577-7">10.1007/s10822-012-9577-7</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Commercial licence</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">Workflows</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_TK " onmouseover="javascript:show('idp819232');">
<div class="magnifiable">
<p class="year">XX</p>
<br><p class="label linked">Gs</p>
<br><p class="name">GENESPRING</p>
<div class="elementMagnify cat_TK new" id="idp819232" style="display: none;" onmouseout="javascript:hide('idp819232');">
<p class="year">XX</p>
<br><a href="javascript:showDetail('idp819232');"><p class="label linked">Gs</p></a><br><p class="name">GENESPRING</p>
</div>
</div>
<div class="elementDetail" id="detail_idp819232" style="display: none;">
<div class="bg"></div>
<div class="fg cat_TK">
<p class="detailClose"><a href="javascript:hideDetail('idp819232');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">GENESPRING</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> GeneSpring provides powerful, accessible statistical tools for fast visualization and analysis of microarray data. Designed specifically for the needs of biologists, GeneSpring offers an interactive computing environment that promotes investigation and enables understanding of microarray data within a biological context.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">XX</td>
<td rowspan="7">No impact data available.</td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.genomics.agilent.com/CollectionSubpage.aspx?PageType=Product&amp;SubPageType=ProductDetail&amp;PageID=1675">http://www.genomics.agilent.com/CollectionSubpage.aspx?PageType=Product&amp;SubPageType=ProductDetail&amp;PageID=1675</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Agilent Technologies</td>
</tr>
<tr>
<td class="detailLeft">Key publication:</td>
<td class="detailRight"><a target="_blank" href="http://www.chem.agilent.com/Library/applications/5990-7505en_hi.pdf">http://www.chem.agilent.com/Library/applications/5990-7505en_hi.pdf</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Linux, Mac OS X and Windows</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Yearly Licence</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">ToolKits and APIs</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_TK new" onmouseover="javascript:show('idp822448');">
<div class="magnifiable">
<p class="year">05</p>
<br><p class="label linked">Gu</p>
<br><p class="name">Geneious Pro</p>
<div class="elementMagnify cat_TK new" id="idp822448" style="display: none;" onmouseout="javascript:hide('idp822448');">
<p class="year">2005</p>
<br><a href="javascript:showDetail('idp822448');"><p class="label linked">Gu</p></a><br><p class="name">Geneious Pro</p>
</div>
</div>
<div class="elementDetail" id="detail_idp822448" style="display: none;">
<div class="bg"></div>
<div class="fg cat_TK">
<p class="detailClose"><a href="javascript:hideDetail('idp822448');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">Geneious Pro</td>
<td class="detailLogo" colspan="2"><img src="http://upload.wikimedia.org/wikipedia/en/8/89/Geneious_Logo.gif"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Geneious Pro is an integrated, cross-platform bioinformatics software suite for manipulating, finding, sharing, and exploring biological data such as DNA sequences or proteins, phylogenies, 3D structure information, publications, etc. It features sequence alignment and phylogenetic analysis, contig assembly, primer design and cloning, access to NCBI and UniProt, BLAST, protein structure viewing, automated PubMed searching, and more. It even includes an API for creating your own plugins.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2005</td>
<td rowspan="7">No impact data available.</td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.geneious.com/">http://www.geneious.com/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Drummond AJ, Ashton B, Buxton S, Cheung M, Cooper A, Duran C, Field M, Heled J, Kearse M, Markowitz S, Moir R, Stones-Havas S, Sturrock S, Thierer T, Wilson A (2011)</td>
</tr>
<tr>
<td class="detailLeft">Key publication:</td>
<td class="detailRight"><a target="_blank" href="index.html"></a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Universal</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Commercial licence</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">ToolKits and APIs</td>
</tr>
</table>
</div>
</div>
</td>
  <td class="element cat_TK " onmouseover="javascript:show('idp826224');">
<div class="magnifiable">
<p class="year">05</p>
<br><p class="label linked">Cb</p>
<br><p class="name">CLC BIO</p>
<div class="elementMagnify cat_TK new" id="idp826224" style="display: none;" onmouseout="javascript:hide('idp826224');">
<p class="year">2005</p>
<br><a href="javascript:showDetail('idp826224');"><p class="label linked">Cb</p></a><br><p class="name">CLC BIO</p>
</div>
</div>
<div class="elementDetail" id="detail_idp826224" style="display: none;">
<div class="bg"></div>
<div class="fg cat_TK">
<p class="detailClose"><a href="javascript:hideDetail('idp826224');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">CLC BIO</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> CLC bio is the world's leading bioinformatics solution provider and the only one that provides both desktop and server software seamlessly integrated and optimized for best performance. By developing their own proprietary algorithms, the scientists at CLC bio have successfully parallelized the data calculations to achieve remarkable improvements in speed over comparable solutions.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2005</td>
<td rowspan="7">No impact data available.</td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.clcbio.com/">http://www.clcbio.com/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">CLC Bio</td>
</tr>
<tr>
<td class="detailLeft">Key publication:</td>
<td class="detailRight"><a target="_blank" href="index.html"></a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight">Universal</td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Commercial licence</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">ToolKits and APIs</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_TK " onmouseover="javascript:show('idp829440');">
<div class="magnifiable">
<p class="year">10</p>
<br><p class="label linked">An</p>
<br><p class="name">Avadis NGS</p>
<div class="elementMagnify cat_TK new" id="idp829440" style="display: none;" onmouseout="javascript:hide('idp829440');">
<p class="year">2010</p>
<br><a href="javascript:showDetail('idp829440');"><p class="label linked">An</p></a><br><p class="name">Avadis NGS</p>
</div>
</div>
<div class="elementDetail" id="detail_idp829440" style="display: none;">
<div class="bg"></div>
<div class="fg cat_TK">
<p class="detailClose"><a href="javascript:hideDetail('idp829440');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">Avadis NGS</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Avadis NGS is an integrated platform that provides analysis, management and visualization tools for next-generation sequencing data. It supports extensive workflows for alignment, RNA-Seq, small RNA-Seq, DNA-Seq, and ChIP-Seq experiments.</td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">2010</td>
<td rowspan="7">No impact data available.</td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.avadis-ngs.com/">http://www.avadis-ngs.com/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Strand Scientific Intelligence</td>
</tr>
<tr>
<td class="detailLeft">Key publication:</td>
<td class="detailRight"><a target="_blank" href="http://www.nature.com/srep/2012/120214/srep00264/full/srep00264.html">http://www.nature.com/srep/2012/120214/srep00264/full/srep00264.html</a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight"></td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Commercial</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">ToolKits and APIs</td>
</tr>
</table>
</div>
</div>
</td>
<td class="element cat_TK " onmouseover="javascript:show('idp832592');">
<div class="magnifiable">
<p class="year">XX</p>
<br><p class="label linked">Pk</p>
<br><p class="name">Partek</p>
<div class="elementMagnify cat_TK new" id="idp832592" style="display: none;" onmouseout="javascript:hide('idp832592');">
<p class="year">XX</p>
<br><a href="javascript:showDetail('idp832592');"><p class="label linked">Pk</p></a><br><p class="name">Partek</p>
</div>
</div>
<div class="elementDetail" id="detail_idp832592" style="display: none;">
<div class="bg"></div>
<div class="fg cat_TK">
<p class="detailClose"><a href="javascript:hideDetail('idp832592');"><img src="close.png" height="25" align="center" alt="Close details."></a></p>
<table class="detailGrid">
<tr>
<td class="detailName">Partek</td>
<td class="detailLogo" colspan="2"><img src="index.html"></td>
</tr>
<tr><td class="detailAbout" colspan="3">
<b>About:</b> Partek's statistical analysis and data mining tools are based on proven methods that undergo rigorous testing to verify mathematical correctness, and our interactive 2-D and 3-D visualizations are linked together to facilitate powerful and intuitive exploratory data analysis. </td></tr>
<tr>
<td class="detailLeft">Year of release:</td>
<td class="detailRight">XX</td>
<td rowspan="7">No impact data available.</td>
</tr>
<tr>
<td class="detailLeft">Download:</td>
<td class="detailRight"><a target="_blank" href="http://www.partek.com/">http://www.partek.com/</a></td>
</tr>
<tr>
<td class="detailLeft">Author:</td>
<td class="detailRight">Partek</td>
</tr>
<tr>
<td class="detailLeft">Key publication:</td>
<td class="detailRight"><a target="_blank" href="index.html"></a></td>
</tr>
<tr>
<td class="detailLeft">OS:</td>
<td class="detailRight"></td>
</tr>
<tr>
<td class="detailLeft">Licence:</td>
<td class="detailRight">Commercial licence</td>
</tr>
<tr>
<td class="detailLeft">Category:</td>
<td class="detailRight">ToolKits and APIs</td>
</tr>
</table>
</div>
</div>
</td>
  </tr>
 </table>
</div></div>
<div id="twitfeed">
<a class="twitter-timeline" href="https://twitter.com/search?q=%23egelements" data-widget-id="345191898161479681">Tweets about "#egelements"</a><script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)0,p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+"://platform.twitter.com/widgets.js";fjs.parentNode.insertBefore(js,fjs);}}(document,"script","twitter-wjs");</script>
</div>
<div id="footer">
<div id="footertext">
<p class="flyers"><a href="http://www.eaglegenomics.com/">Eagle</a> integrates any of these tools into any bioinformatics analysis or pipeline. We combine your and all public data: on the cloud, on your internal system and, coming soon, through a simple web-interface. <a href="flyers.pdf">Learn more</a>.</p>
<p class="initial">The initial selection focused on genomics but we are now opening it up to cover the whole of bioinformatics - proteomics, metabolomics, etc. Please <a href="mailto:ivan.karabaliev@eaglegenomics.com">send us your suggestions</a> for additional tools - we need your input!</p>
</div>
<p class="contact"><a href="flyers.pdf"><img src="flyers.png" alt="Eagle brochures."></a><a href="mailto:ivan.karabaliev@eaglegenomics.com">Make a Suggestion</a> | <a href="elements-about.html">About</a> | <a href="elements-72dpi.pdf" alt="72dpi PDF of the Bioinformatics Elements.">PDF (web)</a> | <a href="elements-1200dpi.pdf" alt="1200dpi PDF of the Bioinformatics Elements.">PDF (print)</a> | <a href="elements.rdf">RDF</a> | <a href="elements.xml">XML</a></p>
<p class="copyright"><a href="http://creativecommons.org/licenses/by-nc-sa/3.0/" rel="license"><img alt="Creative Commons License." src="http://i.creativecommons.org/l/by-nc-sa/3.0/88x31.png"></a> ImpactStory data is not under Eagle's control. Table, design, and underlying data published under the <a href="http://creativecommons.org/licenses/by-nc-sa/3.0/" rel="license">Creative Commons Attribution-NonCommercial-ShareAlike 3.0 Unported License</a>.<br><a href="http://dx.doi.org/10.6084/m9.figshare.726124">doi:10.6084/m9.figshare.726124</a>. Copyright © Eagle Genomics Ltd 2012-2014. No responsibility is assumed for the accuracy of information.<br>
Eagle Genomics is a company registered in England and Wales. company no. 6587071.</p>
</div>
</body>
</html>
