# Restriction Enzyme

* [Highlights of the DNA cutters: a short history of the restriction enzymes](http://nar.oxfordjournals.org/content/early/2013/10/18/nar.gkt990)
* [Type I restriction enzymes and their relatives](http://nar.oxfordjournals.org/content/42/1/20.long)
* [Type III restriction-modification enzymes: a historical perspective](http://nar.oxfordjournals.org/content/42/1/45.full)
* [Type II restriction endonucleases—a historical perspective and more](http://nar.oxfordjournals.org/content/42/12/7489.full)
* [The other face of restriction: modification-dependent enzymes](http://nar.oxfordjournals.org/content/42/1/56.full)
* [To be or not to be: regulation of restriction-modification systems and other toxin-antitoxin systems](http://nar.oxfordjournals.org/content/42/1/70.long)
* [Structure and function of type II restriction endonucleases](http://nar.oxfordjournals.org/content/29/18/3705.full)
* [Common patterns in type II restriction enzyme binding sites](http://nar.oxfordjournals.org/content/33/8/2726.full)



# Hidden Markov Model

* [What is a hidden Markov model?](http://www.nature.com/nbt/journal/v22/n10/full/nbt1004-1315.html)
* [An introduction to hidden Markov models and Bayesian networks](mlg.eng.cam.ac.uk/zoubin/papers/ijprai.pdf)
* [How does eukaryotic gene prediction work?](http://www.nature.com/nbt/journal/v25/n8/full/nbt0807-883.html)
* [Hidden Markov Models, grammars, and biology: a tutorial](http://www.worldscientific.com/doi/abs/10.1142/S0219720005001077)
* [A review of feature selection techniques in bioinformatics](http://bioinformatics.oxfordjournals.org/content/23/19/2507.full)
* [What is Bayesian statistics?](http://www.nature.com/nbt/journal/v22/n9/full/nbt0904-1177.html)



# Gene Prediction

* [Computational prediction of eukaryotic protein-coding genes](http://www.nature.com/nrg/journal/v3/n9/execsumm/nrg890.html)
* [How does eukaryotic gene prediction work?](http://www.nature.com/nbt/journal/v25/n8/full/nbt0807-883.html)
* [Identifying protein-coding genes in genomic sequences](https://genomebiology.biomedcentral.com/articles/10.1186/gb-2009-10-1-201)
* [Computational discovery of human coding and non-coding transcripts with conserved splice sites](http://bioinformatics.oxfordjournals.org/content/27/14/1894.full)
* [Gene Prediction](http://link.springer.com/content/pdf/10.1007/978-1-61779-582-4_6.pdf)



# SNV annotation
* [Targeted capture and massively parallel sequencing of 12 human exomes](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2844771/)
* [Integrated annotation and analysis of genetic variants from next-generation sequencing studies with variant tools](http://bioinformatics.oxfordjournals.org/content/28/3/421.long)
* [A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3.](http://www.tandfonline.com/doi/full/10.4161/fly.19695)
* [Predicting the effects of coding non-synonymous variants on protein function using the SIFT algorithm](http://www.nature.com/nprot/journal/v4/n7/full/nprot.2009.86.html)
* [Predicting the effects of amino acid substitutions on protein function](http://www.annualreviews.org/doi/full/10.1146/annurev.genom.7.080505.115630)
* [SIFT: Predicting amino acid changes that affect protein function](http://nar.oxfordjournals.org/content/31/13/3812.long)
* [Accounting for human polymorphisms predicted to affect protein function](http://genome.cshlp.org/content/12/3/436.long)
* [Predicting deleterious amino acid substitutions](http://genome.cshlp.org/content/11/5/863.long)
* [A method and server for predicting damaging missense mutations](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2855889/)
* [Predicting functional effect of human missense mutations using PolyPhen-2](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4480630/)
* [Human non-synonymous SNPs: server and survey](http://nar.oxfordjournals.org/content/30/17/3894.long)
* [PSIC: profile extraction from sequence alignments with position-specific counts of independent observations](http://peds.oxfordjournals.org/content/12/5/387.long)
* [SNPs3D: candidate gene and SNP selection for association studies](http://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-7-166)
* [Identification and analysis of deleterious human SNPs](linkinghub.elsevier.com/retrieve/pii/S0022-2836(05)01587-1)
* [Loss of protein structure stability as a major causative factor in monogenic disease](http://www.sciencedirect.com/science/article/pii/S0022283605009575)
* [Three-dimensional structural location and molecular functional effects of missense SNPs in the T cell receptor Vbeta domain](http://onlinelibrary.wiley.com/doi/10.1002/prot.10522/abstract)
* [SNPs, protein structure, and disease](http://onlinelibrary.wiley.com/doi/10.1002/humu.22/abstract)
* [Predicting the functional effect of amino acid substitutions and indels](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0046688)
* [A fast computation of pairwise sequence alignment scores between a protein and a set of single-locus variants of another protein](http://provean.jcvi.org/downloads/provean.acmbcb.pdf)



# Enrichment Analysis
* [How does multiple testing correction work?](http://www.nature.com/nbt/journal/v27/n12/full/nbt1209-1135.html)
* [Gene ontology: tool for the unification of biology](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3037419/)
* [The Gene Ontology (GO) project: structured vocabularies for molecular biology and their application to genome and expression analysis](http://onlinelibrary.wiley.com/doi/10.1002/0471250953.bi0702s23/abstract)
* [Use and misuse of the gene ontology annotations](http://www.nature.com/nrg/journal/v9/n7/full/nrg2363.html)
* [The what, where, how and why of gene ontology--a primer for bioinformaticians](http://bib.oxfordjournals.org/content/12/6/723.long)
* [Ten quick tips for using the gene ontology](http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1003343)
* [Gene Ontology annotations and resources](http://nar.oxfordjournals.org/content/41/D1/D530.long)
* [DAVID Bioinformatics Resources: expanded annotation database and novel algorithms to better extract biology from large gene lists](http://nar.oxfordjournals.org/content/35/suppl_2/W169.long)
* [DAVID gene ID conversion tool](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2561161/)
* [Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources](http://www.nature.com/nprot/journal/v4/n1/full/nprot.2008.211.html)
* [Bioinformatics enrichment tools: paths toward the comprehensive functional analysis of large gene lists](http://nar.oxfordjournals.org/content/37/1/1.long)



# Sequence Logo
* [What are DNA sequence motifs?](http://www.nature.com/nbt/journal/v24/n4/full/nbt0406-423.html)
* [Sequence logos: a new way to display consensus sequences](http://nar.oxfordjournals.org/content/18/20/6097.long)
* [WebLogo: a sequence logo generator](http://genome.cshlp.org/content/14/6/1188.long)
* [enoLOGOS: a versatile web tool for energy normalized sequence logos](http://nar.oxfordjournals.org/content/33/suppl_2/W389.long)
* [Skylign: a tool for creating informative, interactive logos representing sequence alignments and profile hidden Markov models](http://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-7)
* [Two Sample Logo: a graphical representation of the differences between two sets of sequence alignments](http://bioinformatics.oxfordjournals.org/content/22/12/1536.long)
* [Automated construction and graphical presentation of protein blocks from unaligned sequences](http://www.sciencedirect.com/science/article/pii/037811199500486P)
* [Discovering sequence motifs with arbitrary insertions and deletions](http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1000071)



# Box Plot
* [Graphical data presentation](http://www.sciencedirect.com/science/article/pii/S0020138308001095)
* [Points of View: Bar charts and box plots](http://www.nature.com/nmeth/journal/v11/n2/full/nmeth.2807.html)
* [Kick the bar chart habit](http://www.nature.com/nmeth/journal/v11/n2/full/nmeth.2837.html)
* [Points of Significance: Visualizing samples with box plots](http://www.nature.com/nmeth/journal/v11/n2/full/nmeth.2813.html)
* [BoxPlotR: a web tool for generation of box plots](http://www.nature.com/nmeth/journal/v11/n2/full/nmeth.2811.html)
* [Beanplot: A Boxplot Alternative for Visual Comparison of Distributions](https://www.jstatsoft.org/article/view/v028c01)
* [Methods for Presenting Statistical Information: The Box Plot](http://pages.uoregon.edu/kpotter/publications/potter-2006-MPSI.pdf)
* [Visual Presentation of Data by Means of Box Plots](http://www.csee.umbc.edu/courses/graduate/CMSC601/Spring11/papers/box_charts.pdf)
* [“Mean ± SEM” or “Mean (SD)”?](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2959222/)
* [Standard deviations and standard errors](http://www.bmj.com/content/331/7521/903.long)
* [Explorations in statistics: standard deviations and standard errors](http://advan.physiology.org/content/32/3/203.long)
* [Error bars in experimental biology](http://jcb.rupress.org/content/177/1/7.long)
* [Misuse of standard error of the mean (SEM) when reporting variability of a sample. A critical evaluation of four anaesthesia journal](http://bja.oxfordjournals.org/content/90/4/514.long)
* [Inappropriate use of standard error of the mean when reporting variability of study samples: a critical evaluation of four selected journals of obstetrics and gynecology](http://www.sciencedirect.com/science/article/pii/S1028455914000084)
* [Box Plot: Display of Distribution](http://www.physics.csbsju.edu/stats/box2.html)
* [40 years of boxplots](http://vita.had.co.nz/papers/boxplots.html)


# UCSC
* [The UCSC Genome Browser Database](http://nar.oxfordjournals.org/content/31/1/51.long)
* [The UCSC Table Browser data retrieval tool](http://nar.oxfordjournals.org/content/32/suppl_1/D493.long)
* [The UCSC genome browser and associated tools](http://bib.oxfordjournals.org/content/14/2/144.long)
* [The UCSC Genome Browser database: 2016 update](http://nar.oxfordjournals.org/content/44/D1/D717.long)
* [Visualizing Genomic Annotations with the UCSC Genome Browser](http://cshprotocols.cshlp.org/content/early/2016/08/27/pdb.prot093062.long)
* [Track data hubs enable visualization of user-defined genome-wide annotations on the UCSC Genome Browser](http://bioinformatics.oxfordjournals.org/content/30/7/1003.long)
* [The UCSC Genome Browser](http://onlinelibrary.wiley.com/doi/10.1002/0471250953.bi0104s40/abstract)
* [Comparative genomic analysis using the UCSC genome browser](http://link.springer.com/protocol/10.1007%2F978-1-59745-514-5_2)
* [A brief introduction to web-based genome browsers](http://bib.oxfordjournals.org/content/14/2/131.long)
* [UCSC genome browser: deep support for molecular biomedical research](http://www.sciencedirect.com/science/article/pii/S1387265608000033)



# Galaxy
* [Galaxy: a web-based genome analysis tool for experimentalists](http://onlinelibrary.wiley.com/doi/10.1002/0471142727.mb1910s89/abstract)
* [Using Galaxy to perform large-scale interactive data analyses](http://onlinelibrary.wiley.com/doi/10.1002/0471250953.bi1005s38/abstract)
* [Galaxy: A Gateway to Tools in e-Science](http://link.springer.com/chapter/10.1007%2F978-0-85729-439-5_6)
* [Galaxy: a platform for interactive large-scale genome analysis](http://genome.cshlp.org/content/15/10/1451.long)
* [Manipulation of FASTQ data with Galaxy](http://bioinformatics.oxfordjournals.org/content/26/14/1783.long)
* [Galaxy: a comprehensive approach for supporting accessible, reproducible, and transparent computational research in the life sciences](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2945788/)
* [Integrating diverse databases into an unified analysis framework: a Galaxy approach](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3092608/)
* [Tavaxy: integrating Taverna and Galaxy workflows with cloud computing support](http://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-77)
* [Galaxy tools and workflows for sequence analysis with applications in molecular plant pathology](https://peerj.com/articles/167/)
* [iReport: a generalised Galaxy solution for integrated experimental reporting](http://gigascience.biomedcentral.com/articles/10.1186/2047-217X-3-19)
* [Galaxy tools to study genome diversity](http://gigascience.biomedcentral.com/articles/10.1186/2047-217X-2-17)
* [Some phenotype association tools in Galaxy: looking for disease SNPs in a full genome](http://onlinelibrary.wiley.com/doi/10.1002/0471250953.bi1502s39/abstract)



