<!DOCTYPE HTML>


    












<html>
    <!--base_panels.mako-->
    

    
    <head>
            <title>
    Galaxy | 
Published Page | 
    Galaxy 101: The first thing you need to try
</title>
        <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
        <meta name = "viewport" content = "maximum-scale=1.0">
        <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
        
    
    
    <link href="/static/style/base.css?v=1396981224" media="screen" rel="stylesheet" type="text/css" />
<link href="/static/style/jquery.rating.css?v=1396981224" media="screen" rel="stylesheet" type="text/css" />
    <style type="text/css">
    #center {
            left: 0 !important;
    }
    </style>

    <link href="/static/style/autocomplete_tagging.css?v=1396981224" media="screen" rel="stylesheet" type="text/css" />
<link href="/static/style/embed_item.css?v=1396981224" media="screen" rel="stylesheet" type="text/css" />
<link href="/static/style/jquery.rating.css?v=1396981224" media="screen" rel="stylesheet" type="text/css" />
    <link href="/static/style/autocomplete_tagging.css?v=1396981224" media="screen" rel="stylesheet" type="text/css" />
<link href="/static/style/trackster.css?v=1396981224" media="screen" rel="stylesheet" type="text/css" />
<link href="/static/style/library.css?v=1396981224" media="screen" rel="stylesheet" type="text/css" />
<link href="/static/style/jquery-ui/smoothness/jquery-ui.css?v=1396981224" media="screen" rel="stylesheet" type="text/css" />
    
    <style type="text/css">
        .page-body {
            padding: 10px;
        }
        .page-meta {
            float: right;
            width: 27%;
            padding: 0.5em;
            margin: 0.25em;
            vertical-align: text-top;
            border: 2px solid #DDDDDD;
            border-top: 4px solid #DDDDDD;
        }
        
        .historyItemContainer, .toolForm {
            max-width: 500px;
        }
        
        div.toolForm{
            margin-top: 10px;
            margin-bottom: 10px;
        }
        
        .historyItemContainer {
            padding-right: 3px;
            border-right-style: solid;
            border-right-color: #66AA66;
        }
    </style>

    <link href="/static/style/base.css?v=1396981224" media="screen" rel="stylesheet" type="text/css" />
<link href="/static/style/autocomplete_tagging.css?v=1396981224" media="screen" rel="stylesheet" type="text/css" />
    <style type="text/css">
        .toggle { display: none; }
        .embedded-item .title {
            padding-top: 1px;
        }
        .embedded-item h4 {
            margin: 0px;
        }
        .page-body > table {
            padding: 8px 5px 5px;
            min-width: 500px; 
            border: none;
            margin-top: 1em;
            margin-bottom: 1em;
        }
        .page-body caption { 
            text-align: left;
            background: #E4E4B0; 
            padding: 5px; 
            font-weight: bold; 
        }
        .page-body > table td {
            width: 25%;
            padding: 0.2em 0.8em;
        }
        .embedded-item .trackster-nav-container {
            height: inherit;
        }
        .embedded-item .trackster-nav {
            position: inherit;
        }

        /* ---------------------------- histories */
        .embedded-item.history .toggle {
            display: inline;
        }
        .embedded-item.history .expanded-content {
            /* generates a fake wide border */
            padding: 4px;
        }
        /** wraps around the history */
        .embedded-item.history .item-content {
            background-color: white;
            padding: 8px;
            border-radius: 0px 0px 4px 4px;
        }
        .embedded-item.history .history-panel .datasets-list {
            margin-bottom: 8px;
        }
        .embedded-item.history .history-panel .errormessage {
            margin-top: 8px;
        }
        .annotated-history-panel .history-controls {
            margin: 0px 0px 16px 0px;
        }
    </style>

        
    
    



    <script src="/static/scripts/libs/jquery/jquery.js?v=1396981224" type="text/javascript"></script>
<script src="/static/scripts/libs/jquery/jquery.migrate.js?v=1396981224" type="text/javascript"></script>
<script src="/static/scripts/libs/jquery/select2.js?v=1396981224" type="text/javascript"></script>
<script src="/static/scripts/libs/bootstrap.js?v=1396981224" type="text/javascript"></script>
<script src="/static/scripts/libs/underscore.js?v=1396981224" type="text/javascript"></script>
<script src="/static/scripts/libs/backbone/backbone.js?v=1396981224" type="text/javascript"></script>
<script src="/static/scripts/libs/handlebars.runtime.js?v=1396981224" type="text/javascript"></script>
<script src="/static/scripts/galaxy.base.js?v=1396981224" type="text/javascript"></script>
<script src="/static/scripts/libs/require.js?v=1396981224" type="text/javascript"></script>
<script src="/static/scripts/mvc/ui.js?v=1396981224" type="text/javascript"></script>

    <script type="text/javascript">
        var galaxy_config =
        {
            root: '/'
        };

        if (window != window.top)
            $('<link href="' + galaxy_config.root + 'static/style/galaxy.frame.masthead.css" rel="stylesheet">').appendTo('head');

        // start a Galaxy namespace for objects created
        window.Galaxy = window.Galaxy || {};

        // console protection
        window.console = window.console || {
            log     : function(){},
            debug   : function(){},
            info    : function(){},
            warn    : function(){},
            error   : function(){},
            assert  : function(){}
        };

        require.config({
            baseUrl: "/static/scripts",
            shim: {
                "libs/underscore": { exports: "_" },
                "libs/backbone/backbone": { exports: "Backbone" }
            }
        });
    </script>


    <script src="/static/scripts/libs/jquery/jstorage.js?v=1396981224" type="text/javascript"></script>
<script src="/static/scripts/libs/jquery/jquery.autocomplete.js?v=1396981224" type="text/javascript"></script>
<script src="/static/scripts/libs/jquery/jquery.rating.js?v=1396981224" type="text/javascript"></script>
<script src="/static/scripts/galaxy.autocom_tagging.js?v=1396981224" type="text/javascript"></script>
    
<script src="/static/scripts/utils/localization.js?v=1396981224" type="text/javascript"></script>


<script type="text/javascript">
    GalaxyLocalization.setLocalizedString(
        {"View data": "View data", "Edit attributes": "\u7f16\u8f91\u5c5e\u6027", "hide deleted": "\u9690\u85cf\u5df2\u5220\u9664\u7684\u6570\u636e", "Info: ": "\u4fe1\u606f\uff1a ", "Your history is empty. Click 'Get Data' on the left pane to start": "\u5386\u53f2\u5df2\u7a7a\uff0c\u8bf7\u5355\u51fb\u5de6\u8fb9\u7a97\u683c\u4e2d\u2018\u83b7\u53d6\u6570\u636e\u2019", "You are currently viewing a deleted history!": "\u60a8\u5728\u67e5\u770b\u4e00\u4e2a\u5df2\u5220\u9664\u7684\u5386\u53f2\uff01", "No data: ": "\u6ca1\u6709\u6570\u636e\uff1a ", "refresh": "\u5237\u65b0", "database: ": "\u6570\u636e\u5e93\uff1a ", "collapse all": "\u5168\u90e8\u6536\u7f29", "Job is currently running": "\u6b63\u5728\u8fd0\u884c\u7684\u8fdb\u7a0b", "Display Data": "Display Data", "Job is waiting to run": "\u7b49\u5f85\u8fd0\u884c\u7684\u8fdb\u7a0b", "hide hidden": "hide hidden", "Download": "\u4e0b\u8f7d", "Error: unknown dataset state": "Error: unknown dataset state", "Metadata is being Auto-Detected.": "Metadata is being Auto-Detected.", "View Details": "\u67e5\u770b\u8be6\u7ec6\u4fe1\u606f", "format: ": "\u683c\u5f0f\uff1a ", "Delete": "\u5220\u9664"}
    );
</script>


<script type="text/javascript">
var //debugging = JSON.parse( sessionStorage.getItem( 'debugging' ) ) || false,
    // use deferred to allow multiple callbacks (.done())
    onhistoryready = jQuery.Deferred();

require.config({
    baseUrl : "/static/scripts"
});

// requirejs optimizer:
   //r.js -o baseUrl='/Users/carleberhard/galaxy/iframe-2-hpanel/static/scripts'    //        name=./mvc/history/history-panel.js out=history-panel.min.js
//TODO: can't get either to work - historyPanel is undefined
//require([ "history-panel.min" ], function( historyPanel ){
//require([ "/static/scripts/history-panel.min.js" ], function( historyPanel ){

require([ "mvc/history/history-panel" ], function( historyPanel ){
    $(function(){
        onhistoryready.resolve( historyPanel )
    });
});
</script>

    <script src="/static/scripts/galaxy.panels.js?v=1396981224" type="text/javascript"></script>
<script src="/static/scripts/libs/jquery/jstorage.js?v=1396981224" type="text/javascript"></script>
<script src="/static/scripts/libs/jquery/jquery.event.drag.js?v=1396981224" type="text/javascript"></script>
<script src="/static/scripts/libs/jquery/jquery.event.hover.js?v=1396981224" type="text/javascript"></script>
<script src="/static/scripts/libs/jquery/jquery.mousewheel.js?v=1396981224" type="text/javascript"></script>
<script src="/static/scripts/libs/jquery/jquery-ui.js?v=1396981224" type="text/javascript"></script>
<script src="/static/scripts/libs/require.js?v=1396981224" type="text/javascript"></script>
<script src="/static/scripts/libs/farbtastic.js?v=1396981224" type="text/javascript"></script>

    <script type="text/javascript">
        
        // Handle click on community tag.
        function community_tag_click(tag_name, tag_value) {
            
    
            var href = '/page/list_published';
            href = href + "?f-tags=" + tag_name;
            if (tag_value != undefined && tag_value != "") {
                href = href + ":" + tag_value;
            }
            self.location = href;
        }
        
        // Map item rating to number of stars to show.
        function map_rating_to_num_stars(rating) {
            if (rating <= 0)
                return 0;
            else if (rating > 0 && rating <= 1.5)
                return 1;
            else if (rating > 1.5 && rating <= 2.5)
                return 2;
            else if (rating > 2.5 && rating <= 3.5)
                return 3;
            else if (rating > 3.5 && rating <= 4.5)
                return 4;
            else if (rating > 4.5)
                return 5;
        }
        
        // Init. on document load.
        $(function() {
            // Set links to Galaxy screencasts to open in overlay.
            $(this).find("a[href^='http://screencast.g2.bx.psu.edu/']").each( function() {
                $(this).click( function() {
                    var href = $(this).attr('href');
                    show_in_overlay(
                        {
                            url: href,        
                            width: 640,
                            height: 480,
                            scroll: 'no'  
                        }
                    );
                    return false;
                });
            });
            
            // Init history boxes.
            //init_history_items( $("div.historyItemWrapper"), false, "nochanges" );
            
            // Init user item rating.
            $('.user_rating_star').rating({
                callback: function(rating, link) {
                    $.ajax({
                        type: "GET",
                        url: "/page/rate_async",
                        data: { id : "924b1a46d03eaf1e", rating : rating },
                        dataType: 'json',
                        error: function() { alert( "Rating submission failed" ); },
                        success: function( community_data ) {
                            $('#rating_feedback').show();
                            $('#num_ratings').text(Math.round(community_data[1]*10)/10);
                            $('#ave_rating').text(community_data[0]);
                            $('.community_rating_star').rating('readOnly', false);
                            $('.community_rating_star').rating('select', map_rating_to_num_stars(community_data[0])-1);
                            $('.community_rating_star').rating('readOnly', true);
                        }
                    });
                },
                required: true // Hide cancel button.
            });
        });    
    </script>

    <script type="text/javascript">
    
        $(function() {
            // Setup embedded content:
            //  (a) toggles for showing/hiding embedded content;
            //  (b) ...
            $('.embedded-item').each( function() {
                var container = $(this);
                if( container.hasClass( 'history' ) ){ return; }
            
                // Show embedded item.
                var show_embedded_item = function() {
                    var ajax_url = container.find("input[type=hidden]").val();
                    // Only get item content if it's not already there.
                    var item_content = $.trim(container.find(".item-content").text());
                    if (!item_content) {
                        $.ajax({
                            type: "GET",
                            url: ajax_url,
                            error: function() { alert("Getting item content failed."); },
                            success: function( item_content ) {
                                container.find(".summary-content").hide("fast");
                                container.find(".item-content").html(item_content);
                                container.find(".expanded-content").show("fast");
                                container.find(".toggle-expand").hide();
                                container.find(".toggle").show();

                                // Init needed for history items.
                                init_history_items( container.find("div.historyItemWrapper"), "noinit", "nochanges" ); 
                                container.find( "div.historyItemBody:visible" ).each( function() {
                                    if ( $.browser.mozilla ) {
                                        $(this).find( "pre.peek" ).css( "overflow", "hidden" );
                                    }
                                    $(this).hide();
                                });
                                make_popup_menus();
                            }
                        });
                    } else {
                        container.find(".summary-content").hide("fast");
                        container.find(".expanded-content").show("fast");
                        container.find(".toggle-expand").hide();
                        container.find(".toggle").show();
                    }
                };
            
                // Hide embedded item.
                var hide_embedded_item = function() {
                    container.find(".expanded-content").hide("fast");
                    container.find(".summary-content").show("fast");
                    container.find(".toggle").hide();
                    container.find(".toggle-expand").show();
                };
            
                // Setup toggle expand.
                var toggle_expand = $(this).find('.toggle-expand');
                toggle_expand.click( function() {
                    show_embedded_item();
                    return false;
                });
            
                // Setup toggle contract.
                var toggle_contract = $(this).find('.toggle');
                toggle_contract.click( function() {
                    hide_embedded_item();
                    return false;
                });
            
                // Setup toggle embed.
                var toggle_embed = $(this).find('.toggle-embed');
                toggle_embed.click( function() {
                    if (container.find(".expanded-content").is(":visible")) {
                        hide_embedded_item();
                    } else {
                        show_embedded_item();
                    }
                    return false;
                });
            });
        });
    
    </script>

    </head>
    
    

    <body scroll="no" class="full-content ">
        <div id="everything" style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;">
            <div id="background"></div>
            <div id="masthead" class="navbar navbar-fixed-top navbar-inverse">
                
    
    


    <script src="/static/scripts/utils/localization.js?v=1396981224" type="text/javascript"></script>

    <script type="text/javascript">
        if( !window.Galaxy ){
            Galaxy = {};
        }

        if (window != window.top){
            $('<link href="' + galaxy_config.root + 'static/style/galaxy.frame.masthead.css" rel="stylesheet">')
                .appendTo('head');
        }

        require([
            'galaxy.masthead', 'galaxy.menu', 'mvc/ui/ui-modal', 'galaxy.frame', 'mvc/upload/upload-view',
            'mvc/user/user-model',
            'mvc/user/user-quotameter'
        ], function( mod_masthead, mod_menu, mod_modal, mod_frame, GalaxyUpload, user, quotameter ){
            $(function() {
                if (Galaxy.masthead)
                    return;

                var masthead_config = {"ftp_upload_site": "usegalaxy.org", "mailing_lists": "http://wiki.galaxyproject.org/MailingLists", "nginx_upload_path": "/_upload", "terms_url": "https://usegalaxy.org/static/terms.html", "wiki_url": "http://wiki.galaxyproject.org/", "support_url": "http://wiki.galaxyproject.org/Support", "remote_user_logout_href": null, "biostar_url_redirect": "https://usegalaxy.org/biostar/biostar_redirect?biostar_action=show_tags", "ftp_upload_dir": "/galaxy/main/ftp", "logo_url": "/", "search_url": "http://galaxyproject.org/search/usegalaxy/", "screencasts_url": "http://vimeo.com/galaxyproject", "brand": "", "lims_doc_url": "http://main.g2.bx.psu.edu/u/rkchak/p/sts", "active_view": "shared", "citation_url": "http://wiki.galaxyproject.org/CitingGalaxy", "allow_user_creation": true, "is_admin_user": false, "biostar_url": null, "datatypes_disable_auto": false, "use_remote_user": false, "enable_cloud_launch": true, "user": {"requests": false, "valid": true, "email": "xfyin@sibs.ac.cn", "json": {"username": "yixf", "quota_percent": 2, "total_disk_usage": 6610938869.0, "nice_total_disk_usage": "6.2 GB", "email": "xfyin@sibs.ac.cn", "is_admin": false, "tags_used": ["10", "SNP", "Seq", "YH", "dbSNP", "exon", "miRNA", "novel", "pool", "snp"], "model_class": "User", "id": "354d674eb534dccc"}}};

                Galaxy.masthead = new mod_masthead.GalaxyMasthead(masthead_config);
                Galaxy.modal = new mod_modal.View();
                Galaxy.frame = new mod_frame.GalaxyFrame();

                Galaxy.menu = new mod_menu.GalaxyMenu({
                    masthead: Galaxy.masthead,
                    config: masthead_config
                });
                
                Galaxy.upload = new GalaxyUpload(masthead_config);

                if( !Galaxy.currUser ){
                    Galaxy.currUser = new user.User(
{
  "username": "yixf", 
  "quota_percent": 2, 
  "total_disk_usage": 6610938869.0, 
  "nice_total_disk_usage": "6.2 GB", 
  "email": "xfyin@sibs.ac.cn", 
  "is_admin": false, 
  "tags_used": [
    "10", 
    "SNP", 
    "Seq", 
    "YH", 
    "dbSNP", 
    "exon", 
    "miRNA", 
    "novel", 
    "pool", 
    "snp"
  ], 
  "model_class": "User", 
  "id": "354d674eb534dccc"
});
                }
                Galaxy.quotaMeter = new quotameter.UserQuotaMeter({
                    model   : Galaxy.currUser,
                    el      : Galaxy.masthead.$('.quota-meter-container')
                }).render();
            });
        });
    </script>


            </div>
            <div id="messagebox" class="panel-info-message">
                None
            </div>
            
    
    

    

    <div id="top-modal" class="modal fade " style='display: none;'>
        <div id="top-modal-backdrop" class="modal-backdrop fade " style="z-index: -1"></div>
        <div id="top-modal-dialog" class="modal-dialog">
            <div class="modal-content">
                <div class="modal-header">
                    <button type='button' class='close' style="display: none;">&times;</button>
                    <h4 class='title'></h4>
                </div>
                <div class="modal-body"></div>
                <div class="modal-footer">
                    <div class="buttons" style="float: right;"></div>
                    <div class="extra_buttons" style=""></div>
                    <div style="clear: both;"></div>
                </div>
            </div>
        </div>
    </div>

            <div id="center" class="inbound">
                
    
    
    
    
    

    
    <div class="unified-panel-header" unselectable="on">
        <div class="unified-panel-header-inner">
                    <a href="/page/list_published">Published Pages</a> | 
                    <a href="/page/list_published?f-username=aun1">aun1</a>
            | 
    Galaxy 101: The first thing you need to try
            
            <div style="float: right">
                

            </div>
        </div>
    </div>
    
    <div class="unified-panel-body">
        <div style="overflow: auto; height: 100%;">        
            <div class="page-body">
                <div>
                    

                </div>
                
                
    <h1>Galaxy 101: The first thing you should try</h1><p>In this very simple example we will introduce you to bare basics of Galaxy:</p><ul><li>Getting data from UCSC</li><li>Performing simple data manipulation</li><li>Understanding Galaxy's History system</li><li>Creating and editing workflows</li><li>Applying workflows to your data</li></ul><p>You can watch a step-by-step explanation of this entire tutorial&#160;<a href="http://screencast.g2.bx.psu.edu/galaxy101/">here</a>.</p><h2><strong>What are we trying to do?</strong></h2><p>Suppose you get the following question: "Mom (or Dad) ... Which coding exon has the highest number of single nucleotide polymorphisms on chromosome 22?". &#160;You think to yourself "Wow! This is a simple question ... I know exactly where the data is (at UCSC) but how do I actually compute this?" The truth is, there is really no straightforward way of answering this question in a time frame comparable to the attention span of a 7-year-old. Well ... actually there is and it is called Galaxy. So let's try it...</p><h2><strong>0. Organizing your windows and setting up Galaxy account</strong></h2><h3>0.0. Getting your display sorted out</h3><p>To get the most of this tutorial open two browser windows. One you already have (it is this page). To open the other, right click&#160;<a href="http://usegalaxy.org">this link</a>&#160;and&#160;choose "Open in a New Window" (or something similar depending on your operating system and browser):</p><p><img src="http://galaxy.psu.edu/galaxy101/newWindow.png" /></p><p>Then organize your windows as something like this (depending on the size of your monitor you may or may not be able to organize things this way, but you get the idea):</p><p><img src="http://galaxy.psu.edu/galaxy101/windows.png" /></p><h3>0.1. Setting up Galaxy account</h3><p>Go to the <strong>User </strong>link at the top of Galaxy interface and choose <strong>Register </strong>(unless of course you already have an account)<strong>:</strong></p><p><img src="http://galaxy.psu.edu/galaxy101/register.png" /></p><p>Then enter your information and you're in!</p><h2><strong>1. Getting data from UCSC</strong></h2><h3>1.0. Getting coding exons</h3><p>First thing we will do is to obtain data from UCSC by clicking "<em>Get Data -&gt; UCSC Main</em>":</p><p><img src="http://galaxy.psu.edu/galaxy101/getDataUCSC.png" /></p><p>You will see Galaxy's middle pane change to looks like this:</p><p><img src="http://galaxy.psu.edu/galaxy101/ucscGenes.png" /></p><p>Make sure that your settings are exactly the same as shown on the screen (in particular,&#160;<strong>position</strong> should be set to "chr22", <strong>output format</strong> should be set to "BED - browser extensible data", and "Galaxy" should be checked by&#160;<strong>Send output to</strong>&#160;option). Click <strong>get output </strong>and you will see the next screen:</p><p><img src="http://galaxy.psu.edu/galaxy101/ucscGenes2.png" /></p><p>here make sure <strong>Create one BED record per</strong>&#160;is set to "Coding Exons" and click <strong>Send Query to Galaxy</strong>. After this you will see your first History Item in Galaxy's right pane. It will go through gray (preparing) and yellow (running) states to become green:</p><p><img src="http://galaxy.psu.edu/galaxy101/firstHistoryItem.png" />&#160;</p><h3>1.1. Getting SNPs</h3><p>Now is the time to obtain SNP data. This is done almost exactly the same way.&#160;First thing we will do is to again click on "<em>Get Data -&gt; UCSC Main</em>":</p><p><img src="http://galaxy.psu.edu/galaxy101/getDataUCSC.png" /></p><p>but now change <strong>group</strong>&#160;to "Variation and Repeats":</p><p><img src="http://galaxy.psu.edu/galaxy101/varRep.png" /></p><p>so that the whole page looks like this:</p><p><img src="http://galaxy.psu.edu/galaxy101/ucscSNPs.png" /></p><p>click <strong>get output</strong>&#160;and you should see this:</p><p><img src="http://galaxy.psu.edu/galaxy101/ucscSNPs2.png" /></p><p>where you need to make sure that <strong>Whole Gene</strong>&#160;is selected ("Whole Gene" here really means "Whole Feature") and click <strong>Send Query to Galaxy</strong>. You will get your second item in the history:</p><p><img src="http://galaxy.psu.edu/galaxy101/secondHistoryItem.png" /></p><p>Now we will rename the two history items to "Exons" and "SNPs" by clicking on the Pencil icon adjacent to each item. Also we will rename history to "Galaxy 101" (or whatever you want) by clicking on "Unnamed history" so everything looks like this:</p><p><img src="http://galaxy.psu.edu/galaxy101/rename.png" /></p><h2>2. Finding Exons with the highest number of SNPs</h2><h3>2.0. Joining exons with SNPs</h3><p>Let's remind ourselves that our objective was to find which exon contains the most SNPs. This first step in answering this question will be joining exons with SNPs (a fancy word for printing exons and SNPs that overlap side by side). This is done using "<em>Operate on Genomics Intervals -&gt; Join</em><strong>"</strong>&#160;tool:</p><p><img src="http://galaxy.psu.edu/galaxy101/join.png" /></p><p>make sure your exons are first and SNPs are second and click <strong>Execute</strong>. You will get the third history item:</p><p><img src="http://galaxy.psu.edu/galaxy101/thirdHistoryItem.png" /></p><p>which will contain the following data:</p><pre>chr22 16258185 16258303 uc002zlh.1_cds_1_0_chr22_16258186_r 0 - chr22 16258278 16258279 rs2845178  0 +
chr22 16266928 16267095 uc002zlh.1_cds_2_0_chr22_16266929_r 0 - chr22 16267011 16267012 rs7290262  0 +
chr22 16266928 16267095 uc002zlh.1_cds_2_0_chr22_16266929_r 0 - chr22 16266963 16266964 rs10154680 0 +
chr22 16266928 16267095 uc002zlh.1_cds_2_0_chr22_16266929_r 0 - chr22 16267037 16267038 rs2818572  0 +
chr22 16266928 16267095 uc002zlh.1_cds_2_0_chr22_16266929_r 0 - chr22 16267031 16267032 rs7292200  0 +</pre><p>Let's take a look at this dataset. The first six columns correspond to exons. The last six correspond to SNPs. You can see that exon with ID uc002zlh.1_cds_2_0_chr22_16266929_r contains four SNPs with IDs&#160;rs7290262, rs10154680, rs2818572, and rs7292200.&#160;</p><h3>2.1. Counting the number of SNPs per exon</h3><p>Above we've seen that exon&#160;uc002zlh.1_cds_2_0_chr22_16266929_r&#160;is repeated four times in the above dataset. Thus we can easily compute the number of SNPs per exon by simply counting the number of repetitions of name for each exon. This can be easily done with the "<em>Join, Subtract, and Group -&gt; Group</em>" tool:</p><p><img src="http://galaxy.psu.edu/galaxy101/group1.png" /></p><p>choose column 4 by selecting "c4" in <strong>Group by column</strong>. Then click on <strong>Add new Operation</strong>&#160;and make sure the interface looks exactly as shown below:</p><p><img src="http://galaxy.psu.edu/galaxy101/group2.png" /></p><p>click <strong>Execute</strong>. Your history will look like this:</p><p><img src="http://galaxy.psu.edu/galaxy101/fourthHistoryItem.png" /></p><p>if you look at the above image you will see that the result of grouping (dataset #4) contains two columns. This first contains the exon name while the second shows the number of times this name has been repeated in dataset #3.&#160;</p><h3>2.3. Sorting exons by SNP count</h3><p>To see which exon has the highest number of SNPs we can simply sort the dataset #4 on the second column in descending order. This is done with "<em>Filter and Sort -&gt; Sort</em>":</p><p><img src="http://galaxy.psu.edu/galaxy101/sort.png" /></p><p>This will generate the fifth history item:</p><p><img src="http://galaxy.psu.edu/galaxy101/fifthHistoryItem.png" /></p><p>and you can now see that the highest number of SNPs per exon is 67.&#160;</p><h3>2.4. Selecting top five</h3><p>Now let's select top five exons with the highest number of SNPs. For this we will use "<em>Text Manipulation -&gt; Select First</em>" tool:</p><p><img src="http://galaxy.psu.edu/galaxy101/selectFirst.png" /></p><p>Clicking <strong>Execute</strong>&#160;will produce the sixth history item that will contain just five lines:</p><p><img src="http://galaxy.psu.edu/galaxy101/sixthHistoryItem.png" /></p><h3>2.5. Recovering exon info and displaying data in genome browsers</h3><p>Now we know that in this dataset the five top exons contain between 41 and 67 SNPs. But what else can we learn about these? To know more we need to get back the positional information (coordinates) of these exons. This information was lost at the grouping step and now all we have is just two columns. To get coordinates back we will match the names of exons in dataset #6 (column 1) against names of the exons in the original dataset #1 (column 4). This can be done with "<em>Join, Subtract and Group -&gt; Compare two Queries</em>" tool (note the settings of the tool in the middle pane):</p><p><img src="http://galaxy.psu.edu/galaxy101/compare.png" /></p><p>this adds the seventh dataset to the history:</p><p><img src="http://galaxy.psu.edu/galaxy101/seventhHistoryItem.png" /></p><p>The best way to learn about these exons is to look at their genomic surrounding. There is really no better way to do this than using genome browsers. Because this analysis was performed on "standard" human genome, you have two choices - UCSC Genome Browser and Ensembl:</p><p><img src="http://galaxy.psu.edu/galaxy101/browsers.png" /></p><p>For example, clicking on "display at UCSC main" will show this (to see your regions look at "User Track" on top of browser image):</p><p><img src="http://galaxy.psu.edu/galaxy101/ucsc.png" /></p><h2>3. Understanding histories</h2><p>In Galaxy your analyses live in histories such as this one:</p><p><img src="http://galaxy.psu.edu/galaxy101/seventhHistoryItem.png" /></p><p>Histories can be very large, you can have as many histories as you want, and all history behavior is controlled by the&#160;<strong>Options</strong>&#160;button on the top of the History pane:</p><p><img src="http://galaxy.psu.edu/galaxy101/historyOptions.png" /></p><p>Many of the options here are self explanatory. If you create a new history, your current history does not disappear. If you would like to list all of your histories just choose&#160;<strong>Saved Histories</strong>&#160;and you will see a list of all your histories in the center pane:</p><p><img src="http://galaxy.psu.edu/galaxy101/historyList.png" />&#160;</p><h2>4. Converting histories into workflows</h2><p>One of the history options listed above is very special. It allows you to easily convert existing histories into analysis workflows. Why would you want to create a workflows out of a history? To redo the analysis again with minimal clicking.&#160;</p><h3>4.0. Extracting workfklow</h3><p>Lets take a look at the history again:</p><p><img src="http://galaxy.psu.edu/galaxy101/seventhHistoryItem.png" /></p><p>You can see that this history contains all steps of our analysis. So by building this history we have actually built a complete record of our analysis with Galaxy preserving all parameter settings applied at every step. Wouldn't it be nice to just convert this history into a workflow that we'll be able to execute again and again? This can be done by clicking on <strong>Options </strong>button:</p><p><img src="http://galaxy.psu.edu/galaxy101/options.png" /></p><p>and selecting <strong>Extract Workflow</strong>&#160;option:</p><p><img src="http://galaxy.psu.edu/galaxy101/extractWorkflow.png" /></p><p>The center pane will change as shown below and you will be able to choose which steps to include/exclude and how to name the newly created workflow. In this case I named it "galaxy101":</p><p><img src="http://galaxy.psu.edu/galaxy101/createWorkflow.png" /></p><p>once you click <strong>Create Workflow</strong>&#160;you will get the following message: "Workflow 'galaxy101' created from current history". But where did it go? Click on <strong>Workflow</strong>&#160;link at the top of Galaxy interface and&#160;you will a list of all workflows with "galaxy101" listed at the top:</p><p><img src="http://galaxy.psu.edu/galaxy101/workflowList.png" /></p><h3>4.1. Opening workflow editor</h3><p>If you click on a triangle adjacent to the workflow's name you will see the following dialogue:</p><p><img src="http://galaxy.psu.edu/galaxy101/workflowDropdown.png" /></p><p>Click <strong>Edit</strong>&#160;and the workflow editor will launch. It will allow you to examine and change settings of this workflow as shown below. Note that the box corresponding to the "<em>Select First</em>" tool is selected (highlighted with the blue border) and you can see parameters of this tool on the right pane. This is how you can view and change parameters of all tools involved in the workflow.</p><p><img src="http://galaxy.psu.edu/galaxy101/wfEditor.png" /></p><h3>4.2. Hiding intermediate steps</h3><p>Among multiple things you can do with workflows I will just mention one. When workflow is executed one is usually interested in the final product and not in the intermediate steps. These steps can be hidden by mousing over a small asterisk in the lower right corner of every tool box:</p><p><img src="http://galaxy.psu.edu/galaxy101/hideStep.png" /></p><p>Yet there is a catch. In a newly created workflow all steps are hidden by default and default behavior of Galaxy is that if all steps of a given workflow are hidden, then nothing gets hidden in the history. This may be counterintuitive, but this is done to decrease the amount of clicking if you do want to hide some steps. So in our case if we want to hide all intermediate steps with the exception of the last one we will click that asterisk in last step of the workflow:</p><p><img src="http://galaxy.psu.edu/galaxy101/lastStep.png" /></p><p>Once you do this the representation of the workflow in the bottom right corner of the editor will change with the last step becoming orange. This means that this is the only step, which will generate a dataset visible in the history:</p><p><img src="http://galaxy.psu.edu/galaxy101/workflowOverview.png" /></p><h3>4.3. Renaming inputs</h3><p>Right now both inputs to the workflow look exactly the same. This is a problem as will be very confusing which input should be exons and which should be SNPs:</p><p><img src="http://galaxy.psu.edu/galaxy101/namingInputs1.png" /></p><p>One the image above you will see that the top input dataset (the one with the blue border) connects to the <em>Join</em> tool first, so it must correspond to the exon data. If you click on this box (in the image above it is already clicked on because it is outlined with the blue border) you will be able to rename the dataset in the right pane:</p><p><img src="http://galaxy.psu.edu/galaxy101/renameExons.png" /></p><p>Then click on the second input dataset and rename it "Features" (this would make this workflow a bit more generic, which will be useful later in this tutorial):</p><p><img src="http://galaxy.psu.edu/galaxy101/renameFeatures.png" /></p><h3>4.4. Renaming outputs</h3><p>Finally let's rename the workflow's output. For this click on the last dataset ("<em>Compare two Queries</em>") and in the <strong>Edit Step Actions</strong>&#160;dialogue box select "Rename Dataset"</p><p><img src="http://galaxy.psu.edu/galaxy101/renameOutput1.png" /></p><p>Click <strong>Create:</strong></p><p><strong><img src="http://galaxy.psu.edu/galaxy101/renameOutput2.png" /></strong></p><p>and call it something like "top 5 exons":</p><p><img src="http://galaxy.psu.edu/galaxy101/top5.png" /></p><h3>4.5. Save! It is important...</h3><p>Now let's save the changes we've made by clicking <strong>Options </strong>(top of the center pane) and selecting <strong>Save:</strong></p><p><strong><img src="http://galaxy.psu.edu/galaxy101/wfSave.png" /></strong></p><h2><br />5. Run workflow on whole genome data</h2><p>Now that we have a workflow, let's do something grand like, for example, finding exons with the highest number of repetitive elements.</p><p><strong>5.0. Create a new history</strong></p><p>Before we start let's create a new history by clicking <strong>Options</strong>&#160;and selecting <strong>Create New</strong>:</p><p><img src="http://galaxy.psu.edu/galaxy101/createNewHistory.png" /></p><h3>5.1. Get Exons</h3><p>Now let's get coding exons for the entire genome by going to "<em>Get Data -&gt; UCSC Main</em>" and setting up parameters as shown below. Note that this time <strong>region</strong>&#160;radio button is set to "genome":</p><p><img src="http://galaxy.psu.edu/galaxy101/getAllGenes.png" /></p><p>Click <strong>get output</strong>&#160;and you will get the next page (if it looks different from the image below, go back and make sure <strong>output format</strong>&#160;is set to "BED - browser extensible format"):</p><p><img src="http://galaxy.psu.edu/galaxy101/ucscGenes2.png" /></p><p>Choose "Coding exons" and click <strong>Send query to Galaxy</strong>.</p><h3>5.2. Get Repeats</h3><p>Go again to "<em>Get Data -&gt; UCSC Main</em>" and make sure the following settings are selected (in particular <strong>group </strong>= "Variation and Repeats" and <strong>track </strong>= "RepeatMasker"):</p><p><img src="http://galaxy.psu.edu/galaxy101/allRepeats.png" /></p><p>Click&#160;<strong>get output</strong>&#160;and you will get the next page (if it looks different from the image below, go back and make sure&#160;<strong>output format</strong>&#160;is set to "BED - browser extensible format"):</p><p><img src="http://galaxy.psu.edu/galaxy101/allRepeats2.png" /></p><p>Select "Whole gene" and click <strong>Send Query to Galaxy.</strong></p><h3>5.3. Start the Workflow</h3><p>At this point you will have two items in your history - one with exons and one with repeats. These datasets are very large (especially repeats) and it will take some time for them to become green. Luckily you do not have to wait as Galaxy will automatically start jobs once uploads have ended. So nothing stops us from starting the workflow we have created. First, click on the <strong>Workflow</strong>&#160;link at the top of Galaxy interface,&#160;mouse over "galaxy101", and click on the arrow:</p><p><img src="http://galaxy.psu.edu/galaxy101/workflowList.png" /></p><p>choose <strong>Run:</strong></p><p><img src="http://galaxy.psu.edu/galaxy101/workflowDropdown.png" /></p><p>Center pane will change to allow you launching the workflow. Select appropriate datasets for Repeats and Exon inputs as shown below, scroll down, and click <strong>Run workflow.</strong></p><p><img src="http://galaxy.psu.edu/galaxy101/launchWorkflow.png" /></p><p>Once workflow has started you will initially be able to see all its steps. Note that you are joining 77,614 exons with&#160;5,298,130&#160;repeats, so naturally this will take some time:</p><p><img src="http://galaxy.psu.edu/galaxy101/launchedWorkflow.png" /></p><h3>5.4. Get coffee</h3><p>As we mentioned above this will take some time, so go get coffee and then you will see this. Note that because all intermediate steps of the workflow were hidden, once it is finished you will only see the final dataset #7:</p><p><img src="http://galaxy.psu.edu/galaxy101/final.png" />htt</p><h2>6. We did not fake this:</h2><p>The two histories and the workflow described in this page are accessible directly from this page below. The histories are embedded below (click on the green plus icon to import):</p>


<div id="history-6a2e1073bdfcd697" class='embedded-item display history'>
    <div class='title'>
        <div style="float: left">
            <a class="expand-content-btn icon-button toggle-expand" href="/u/aun1/h/galaxy-101"
               title="Show or hide history contents"></a>
        </div>
        <div style="float: right;">
            <a title="Import history" class="icon-button import" href="/history/imp?id=6a2e1073bdfcd697"></a>
            <a title="View history" class="icon-button go-to-full-screen" href="/u/aun1/h/galaxy-101"></a>
        </div>
        <h4>
            <a class="toggle-embed" href="/u/aun1/h/galaxy-101" title="Show or hide history contents">
                Galaxy History | 
    Galaxy 101
            </a>
        </h4>
    </div>
    <div class='summary-content'>
    </div>
    <div class='expanded-content'>
        <div class='item-content'>
            <div class='history-panel'></div>
        </div>
    </div>
</div>

<script type="text/javascript">
// Embedding the same history more than once will confuse DOM ids.
//  In order to handle this, find this script and cache the previous node (the div above).
//  (Since we need thisScript to be locally scoped or it will get overwritten, enclose in self-calling function)
(function(){
    var scripts = document.getElementsByTagName( 'script' ),
        // this is executed immediately, so the last script will be this script
        thisScript = scripts[ scripts.length - 1 ],
        $embeddedHistory = $( thisScript ).prev();

    require.config({
        baseUrl : "/static/scripts"
    });
    require([ 'mvc/history/annotated-history-panel' ], function( panelMod ){

        function toggleExpanded( ev ){
            var $embeddedHistory = $( thisScript ).prev();
            $embeddedHistory.find( '.expand-content-btn' ).toggleClass( 'toggle-expand' ).toggleClass( 'toggle' );
            $embeddedHistory.find( ".summary-content" ).slideToggle( "fast" );
            $embeddedHistory.find( ".annotation" ).slideToggle( "fast" );
            $embeddedHistory.find( ".expanded-content" ).slideToggle( "fast" );
            ev.preventDefault();
        }

        $(function(){
            var debugging = JSON.parse( sessionStorage.getItem( 'debugging' ) ) || false,
                historyModel = require( 'mvc/history/history-model' ),
                panel = new panelMod.AnnotatedHistoryPanel({
                    el      : $embeddedHistory.find( ".history-panel" ),
                    model   : new historyModel.History(
                        {"slug": "galaxy-101", "user_id": "859c9f6eabd3410d", "name": "Galaxy 101", "tags": [], "deleted": false, "state_ids": {"paused": [], "ok": ["7bda7eb42f2a0978", "703a643066c25e9f", "df07c4e0fafca3a6", "bcbf89cb85fbd8e1", "65e9ac069d329e6d", "88e82af448958b49", "d17cd682834bb35a", "c6d1c99d4b9a6f67"], "failed_metadata": [], "upload": [], "discarded": [], "running": [], "setting_metadata": [], "error": [], "new": [], "queued": [], "empty": []}, "username_and_slug": "u/aun1/h/galaxy-101", "purged": false, "state_details": {"paused": 0, "ok": 8, "failed_metadata": 0, "upload": 0, "discarded": 0, "running": 0, "setting_metadata": 0, "error": 0, "new": 0, "queued": 0, "empty": 0}, "genome_build": "?", "state": "ok", "published": true, "nice_size": "16.8 MB", "model_class": "History", "annotation": null, "id": "6a2e1073bdfcd697", "importable": true},
                        [{"misc_blurb": "13,218 regions", "peek": "<table cellspacing=\"0\" cellpadding=\"3\"><tr><th>1.Chrom</th><th>2.Start</th><th>3.End</th><th>4.Name</th><th>5</th><th>6.Strand</th></tr><tr><td>chr22</td><td>16258185</td><td>16258303</td><td>uc002zlh.1_cds_1_0_chr22_16258186_r</td><td>0</td><td>-</td></tr><tr><td>chr22</td><td>16266928</td><td>16267095</td><td>uc002zlh.1_cds_2_0_chr22_16266929_r</td><td>0</td><td>-</td></tr><tr><td>chr22</td><td>16268136</td><td>16268181</td><td>uc002zlh.1_cds_3_0_chr22_16268137_r</td><td>0</td><td>-</td></tr><tr><td>chr22</td><td>16269872</td><td>16269943</td><td>uc002zlh.1_cds_4_0_chr22_16269873_r</td><td>0</td><td>-</td></tr><tr><td>chr22</td><td>16275206</td><td>16275277</td><td>uc002zlh.1_cds_5_0_chr22_16275207_r</td><td>0</td><td>-</td></tr><tr><td>chr22</td><td>16277747</td><td>16277885</td><td>uc002zlh.1_cds_6_0_chr22_16277748_r</td><td>0</td><td>-</td></tr></table>", "update_time": "2010-11-02T00:33:39.889218", "metadata_endCol": 3, "data_type": "bed", "tags": [], "deleted": false, "history_id": "6a2e1073bdfcd697", "metadata_column_names": null, "visible": true, "genome_build": "hg19", "metadata_nameCol": 4, "hid": 1, "metadata_startCol": 2, "metadata_chromCol": 1, "file_size": 849532, "metadata_viz_filter_cols": [4], "metadata_data_lines": 13218, "file_ext": "bed", "api_type": "file", "id": "7bda7eb42f2a0978", "misc_info": "UCSC Main on Human: knownGene (chr22:1-51304566)", "hda_ldda": "hda", "visualizations": ["trackster", "circster"], "name": "Exons", "metadata_columns": 6, "uuid": null, "download_url": "/api/histories/6a2e1073bdfcd697/contents/7bda7eb42f2a0978/display", "annotation": "Coding exon from chromosome 22 fetched from UCSC", "state": "ok", "metadata_comment_lines": null, "metadata_strandCol": 6, "model_class": "HistoryDatasetAssociation", "accessible": true, "metadata_dbkey": "hg19", "purged": false, "metadata_column_types": ["str", "int", "int", "str", "int", "str"]}, {"misc_blurb": "379,047 regions", "peek": "<table cellspacing=\"0\" cellpadding=\"3\"><tr><th>1.Chrom</th><th>2.Start</th><th>3.End</th><th>4.Name</th><th>5</th><th>6.Strand</th></tr><tr><td>chr22</td><td>16050115</td><td>16050116</td><td>rs77005907</td><td>0</td><td>-</td></tr><tr><td>chr22</td><td>16050251</td><td>16050252</td><td>rs3016036</td><td>0</td><td>+</td></tr><tr><td>chr22</td><td>16050352</td><td>16050353</td><td>rs56342815</td><td>0</td><td>+</td></tr><tr><td>chr22</td><td>16050352</td><td>16050353</td><td>rs2334386</td><td>0</td><td>+</td></tr><tr><td>chr22</td><td>16050374</td><td>16050375</td><td>rs2844882</td><td>0</td><td>+</td></tr><tr><td>chr22</td><td>16050407</td><td>16050408</td><td>rs2844883</td><td>0</td><td>+</td></tr></table>", "update_time": "2011-06-17T05:45:14.631182", "metadata_endCol": 3, "data_type": "bed", "tags": [], "deleted": false, "history_id": "6a2e1073bdfcd697", "metadata_column_names": null, "visible": true, "genome_build": "hg19", "metadata_nameCol": 4, "hid": 2, "metadata_startCol": 2, "metadata_chromCol": 1, "file_size": 14605197, "metadata_viz_filter_cols": [4], "metadata_data_lines": 379047, "file_ext": "bed", "api_type": "file", "id": "703a643066c25e9f", "misc_info": "UCSC Main on Human: snp131 (chr22:1-51304566)", "hda_ldda": "hda", "visualizations": ["trackster", "circster"], "name": "SNPs", "metadata_columns": 6, "uuid": null, "download_url": "/api/histories/6a2e1073bdfcd697/contents/703a643066c25e9f/display", "annotation": "Chromosome 22 SNPs fetched from UCSC", "state": "ok", "metadata_comment_lines": null, "metadata_strandCol": 6, "model_class": "HistoryDatasetAssociation", "accessible": true, "metadata_dbkey": "hg19", "purged": false, "metadata_column_types": ["str", "int", "int", "str", "int", "str"]}, {"misc_blurb": "16,190 regions", "peek": "<table cellspacing=\"0\" cellpadding=\"3\"><tr><th>1.Chrom</th><th>2.Start</th><th>3.End</th><th>4.Name</th><th>5</th><th>6.Strand</th><th>7</th><th>8</th><th>9</th><th>10</th><th>11</th><th>12</th></tr><tr><td>chr22</td><td>16258185</td><td>16258303</td><td>uc002zlh.1_cds_1_0_chr22_16258186_r</td><td>0</td><td>-</td><td>chr22</td><td>16258278</td><td>16258279</td><td>rs2845178</td><td>0</td><td>+</td></tr><tr><td>chr22</td><td>16266928</td><td>16267095</td><td>uc002zlh.1_cds_2_0_chr22_16266929_r</td><td>0</td><td>-</td><td>chr22</td><td>16267011</td><td>16267012</td><td>rs7290262</td><td>0</td><td>+</td></tr><tr><td>chr22</td><td>16266928</td><td>16267095</td><td>uc002zlh.1_cds_2_0_chr22_16266929_r</td><td>0</td><td>-</td><td>chr22</td><td>16266963</td><td>16266964</td><td>rs10154680</td><td>0</td><td>+</td></tr><tr><td>chr22</td><td>16266928</td><td>16267095</td><td>uc002zlh.1_cds_2_0_chr22_16266929_r</td><td>0</td><td>-</td><td>chr22</td><td>16267037</td><td>16267038</td><td>rs2818572</td><td>0</td><td>+</td></tr><tr><td>chr22</td><td>16266928</td><td>16267095</td><td>uc002zlh.1_cds_2_0_chr22_16266929_r</td><td>0</td><td>-</td><td>chr22</td><td>16267031</td><td>16267032</td><td>rs7292200</td><td>0</td><td>+</td></tr><tr><td>chr22</td><td>16269872</td><td>16269943</td><td>uc002zlh.1_cds_4_0_chr22_16269873_r</td><td>0</td><td>-</td><td>chr22</td><td>16269933</td><td>16269934</td><td>rs2845206</td><td>0</td><td>+</td></tr></table>", "update_time": "2011-06-17T16:53:04.723438", "metadata_endCol": 3, "data_type": "interval", "tags": [], "deleted": false, "history_id": "6a2e1073bdfcd697", "metadata_column_names": null, "visible": true, "genome_build": "hg19", "metadata_nameCol": 4, "hid": 3, "metadata_startCol": 2, "metadata_chromCol": 1, "file_size": 1663598, "metadata_data_lines": 16190, "file_ext": "interval", "api_type": "file", "id": "df07c4e0fafca3a6", "misc_info": "", "hda_ldda": "hda", "visualizations": ["trackster", "circster"], "name": "Join on data 2 and data 1", "metadata_columns": 12, "uuid": null, "download_url": "/api/histories/6a2e1073bdfcd697/contents/df07c4e0fafca3a6/display", "annotation": "Here we join coordinates of exons with coordinates of SNPs to find exons that overlap with SNPs", "state": "ok", "metadata_comment_lines": null, "metadata_strandCol": 6, "model_class": "HistoryDatasetAssociation", "accessible": true, "metadata_dbkey": "hg19", "purged": false, "metadata_column_types": ["str", "int", "int", "str", "int", "str", "str", "int", "int", "str", "int", "str"]}, {"misc_blurb": "7,030 lines", "peek": "<table cellspacing=\"0\" cellpadding=\"3\"><tr><th>1</th><th>2</th></tr><tr><td>uc002zlh.1_cds_1_0_chr22_16258186_r</td><td>1</td></tr><tr><td>uc002zlh.1_cds_2_0_chr22_16266929_r</td><td>4</td></tr><tr><td>uc002zlh.1_cds_4_0_chr22_16269873_r</td><td>1</td></tr><tr><td>uc002zlh.1_cds_5_0_chr22_16275207_r</td><td>2</td></tr><tr><td>uc002zlh.1_cds_6_0_chr22_16277748_r</td><td>5</td></tr><tr><td>uc002zlh.1_cds_7_0_chr22_16279195_r</td><td>2</td></tr></table>", "update_time": "2011-06-16T22:26:18.809672", "data_type": "tabular", "tags": [], "deleted": false, "history_id": "6a2e1073bdfcd697", "metadata_column_names": null, "visible": true, "genome_build": "hg19", "hid": 4, "file_size": 269056, "metadata_data_lines": 7030, "file_ext": "tabular", "api_type": "file", "id": "bcbf89cb85fbd8e1", "misc_info": "--Group by c4: count[c4]", "hda_ldda": "hda", "visualizations": [], "name": "Group on data 3", "metadata_columns": 2, "uuid": null, "download_url": "/api/histories/6a2e1073bdfcd697/contents/bcbf89cb85fbd8e1/display", "annotation": "Counting the occurrencies of each exons tells\n us the number of SNPs per exons", "state": "ok", "metadata_comment_lines": null, "model_class": "HistoryDatasetAssociation", "accessible": true, "metadata_dbkey": "hg19", "purged": false, "metadata_column_types": ["str", "int"]}, {"misc_blurb": "7,030 lines", "peek": "<table cellspacing=\"0\" cellpadding=\"3\"><tr><th>1</th><th>2</th></tr><tr><td>uc010gsw.1_cds_1_0_chr22_21480537_r</td><td>67</td></tr><tr><td>uc002zoc.2_cds_0_0_chr22_18834445_f</td><td>58</td></tr><tr><td>uc003bhh.2_cds_0_0_chr22_46652458_r</td><td>46</td></tr><tr><td>uc002zsd.3_cds_0_0_chr22_20456382_r</td><td>41</td></tr><tr><td>uc002zuq.3_cds_0_0_chr22_21738148_f</td><td>41</td></tr><tr><td>uc002zuy.3_cds_0_0_chr22_21900346_r</td><td>41</td></tr></table>", "update_time": "2011-06-17T02:13:23.953231", "data_type": "tabular", "tags": [], "deleted": false, "history_id": "6a2e1073bdfcd697", "metadata_column_names": null, "visible": true, "genome_build": "hg19", "hid": 5, "file_size": 269056, "metadata_data_lines": 7030, "file_ext": "tabular", "api_type": "file", "id": "65e9ac069d329e6d", "misc_info": "", "hda_ldda": "hda", "visualizations": [], "name": "Sort on data 4", "metadata_columns": 2, "uuid": null, "download_url": "/api/histories/6a2e1073bdfcd697/contents/65e9ac069d329e6d/display", "annotation": "Sorting to find exons with the highest number of SNPs", "state": "ok", "metadata_comment_lines": null, "model_class": "HistoryDatasetAssociation", "accessible": true, "metadata_dbkey": "hg19", "purged": false, "metadata_column_types": ["str", "int"]}, {"misc_blurb": "5 lines", "peek": "<table cellspacing=\"0\" cellpadding=\"3\"><tr><th>1</th><th>2</th></tr><tr><td>uc010gsw.1_cds_1_0_chr22_21480537_r</td><td>67</td></tr><tr><td>uc002zoc.2_cds_0_0_chr22_18834445_f</td><td>58</td></tr><tr><td>uc003bhh.2_cds_0_0_chr22_46652458_r</td><td>46</td></tr><tr><td>uc002zsd.3_cds_0_0_chr22_20456382_r</td><td>41</td></tr><tr><td>uc002zuq.3_cds_0_0_chr22_21738148_f</td><td>41</td></tr></table>", "update_time": "2011-06-17T17:18:21.852463", "data_type": "tabular", "tags": [], "deleted": false, "history_id": "6a2e1073bdfcd697", "metadata_column_names": null, "visible": true, "genome_build": "hg19", "hid": 6, "file_size": 195, "metadata_data_lines": 5, "file_ext": "tabular", "api_type": "file", "id": "88e82af448958b49", "misc_info": "", "hda_ldda": "hda", "visualizations": [], "name": "Select first on data 5", "metadata_columns": 2, "uuid": null, "download_url": "/api/histories/6a2e1073bdfcd697/contents/88e82af448958b49/display", "annotation": null, "state": "ok", "metadata_comment_lines": null, "model_class": "HistoryDatasetAssociation", "accessible": true, "metadata_dbkey": "hg19", "purged": false, "metadata_column_types": ["str", "int"]}, {"misc_blurb": "5 regions", "peek": "<table cellspacing=\"0\" cellpadding=\"3\"><tr><th>1</th><th>2</th><th>3</th><th>4</th><th>5</th><th>6</th></tr><tr><td>chr22</td><td>18834444</td><td>18835833</td><td>uc002zoc.2_cds_0_0_chr22_18834445_f</td><td>0</td><td>+</td></tr><tr><td>chr22</td><td>20456381</td><td>20461301</td><td>uc002zsd.3_cds_0_0_chr22_20456382_r</td><td>0</td><td>-</td></tr><tr><td>chr22</td><td>21738147</td><td>21743067</td><td>uc002zuq.3_cds_0_0_chr22_21738148_f</td><td>0</td><td>+</td></tr><tr><td>chr22</td><td>46652457</td><td>46659219</td><td>uc003bhh.2_cds_0_0_chr22_46652458_r</td><td>0</td><td>-</td></tr><tr><td>chr22</td><td>21480536</td><td>21481925</td><td>uc010gsw.1_cds_1_0_chr22_21480537_r</td><td>0</td><td>-</td></tr></table>", "update_time": "2011-06-17T11:00:40.695811", "metadata_endCol": 3, "data_type": "bedstrict", "tags": [], "deleted": false, "history_id": "6a2e1073bdfcd697", "metadata_column_names": null, "visible": false, "genome_build": "hg19", "metadata_nameCol": 4, "hid": 7, "metadata_startCol": 2, "metadata_chromCol": 1, "file_size": 320, "metadata_viz_filter_cols": [4], "metadata_data_lines": 5, "file_ext": "bedstrict", "api_type": "file", "id": "d17cd682834bb35a", "misc_info": "5 regions converted to BED.", "hda_ldda": "hda", "visualizations": ["trackster", "circster"], "name": "Compare two Queries on data 6 and data 1", "metadata_columns": 6, "uuid": null, "download_url": "/api/histories/6a2e1073bdfcd697/contents/d17cd682834bb35a/display", "annotation": null, "state": "ok", "metadata_comment_lines": null, "metadata_strandCol": 6, "model_class": "HistoryDatasetAssociation", "accessible": true, "metadata_dbkey": "hg19", "purged": false, "metadata_column_types": ["str", "int", "int", "str", "int", "str"]}, {"misc_blurb": "5 regions", "peek": "<table cellspacing=\"0\" cellpadding=\"3\"><tr><th>1.Chrom</th><th>2.Start</th><th>3.End</th><th>4.Name</th><th>5</th><th>6.Strand</th></tr><tr><td>chr22</td><td>18834444</td><td>18835833</td><td>uc002zoc.2_cds_0_0_chr22_18834445_f</td><td>0</td><td>+</td></tr><tr><td>chr22</td><td>20456381</td><td>20461301</td><td>uc002zsd.3_cds_0_0_chr22_20456382_r</td><td>0</td><td>-</td></tr><tr><td>chr22</td><td>21738147</td><td>21743067</td><td>uc002zuq.3_cds_0_0_chr22_21738148_f</td><td>0</td><td>+</td></tr><tr><td>chr22</td><td>46652457</td><td>46659219</td><td>uc003bhh.2_cds_0_0_chr22_46652458_r</td><td>0</td><td>-</td></tr><tr><td>chr22</td><td>21480536</td><td>21481925</td><td>uc010gsw.1_cds_1_0_chr22_21480537_r</td><td>0</td><td>-</td></tr></table>", "update_time": "2011-06-17T11:00:40.704702", "metadata_endCol": 3, "data_type": "bed", "tags": [], "deleted": false, "history_id": "6a2e1073bdfcd697", "metadata_column_names": null, "visible": true, "genome_build": "hg19", "metadata_nameCol": 4, "hid": 7, "metadata_startCol": 2, "metadata_chromCol": 1, "file_size": 320, "metadata_viz_filter_cols": [4], "metadata_data_lines": 5, "file_ext": "bed", "api_type": "file", "id": "c6d1c99d4b9a6f67", "misc_info": "join (GNU coreutils) 8.5\nCopyright (C) 2010 Free Software Foundation, Inc.\nLicense GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>.\nThis is free software: you are free to change and redistribute it.\nThere is NO WARRANTY, to the exten", "hda_ldda": "hda", "visualizations": ["trackster", "circster"], "name": "Compare two Queries on data 6 and data 1", "metadata_columns": 6, "uuid": null, "download_url": "/api/histories/6a2e1073bdfcd697/contents/c6d1c99d4b9a6f67/display", "annotation": "The top 5!", "state": "ok", "metadata_comment_lines": null, "metadata_strandCol": 6, "model_class": "HistoryDatasetAssociation", "accessible": true, "metadata_dbkey": "hg19", "purged": false, "metadata_column_types": ["str", "int", "int", "str", "int", "str"]}],
                        { logger: ( debugging )?( console ):( null ) }
                    )
                }).render();

            $embeddedHistory.find( '.expand-content-btn' ).click( toggleExpanded );
            $embeddedHistory.find( '.toggle-embed' ).click( toggleExpanded );
        });
    });
})();
</script>



<div id="history-c1287a8c8ec5e9cc" class='embedded-item display history'>
    <div class='title'>
        <div style="float: left">
            <a class="expand-content-btn icon-button toggle-expand" href="/u/aun1/h/exons-vs-repeats"
               title="Show or hide history contents"></a>
        </div>
        <div style="float: right;">
            <a title="Import history" class="icon-button import" href="/history/imp?id=c1287a8c8ec5e9cc"></a>
            <a title="View history" class="icon-button go-to-full-screen" href="/u/aun1/h/exons-vs-repeats"></a>
        </div>
        <h4>
            <a class="toggle-embed" href="/u/aun1/h/exons-vs-repeats" title="Show or hide history contents">
                Galaxy History | 
    Exons vs Repeats
            </a>
        </h4>
    </div>
    <div class='summary-content'>
    </div>
    <div class='expanded-content'>
        <div class='item-content'>
            <div class='history-panel'></div>
        </div>
    </div>
</div>

<script type="text/javascript">
// Embedding the same history more than once will confuse DOM ids.
//  In order to handle this, find this script and cache the previous node (the div above).
//  (Since we need thisScript to be locally scoped or it will get overwritten, enclose in self-calling function)
(function(){
    var scripts = document.getElementsByTagName( 'script' ),
        // this is executed immediately, so the last script will be this script
        thisScript = scripts[ scripts.length - 1 ],
        $embeddedHistory = $( thisScript ).prev();

    require.config({
        baseUrl : "/static/scripts"
    });
    require([ 'mvc/history/annotated-history-panel' ], function( panelMod ){

        function toggleExpanded( ev ){
            var $embeddedHistory = $( thisScript ).prev();
            $embeddedHistory.find( '.expand-content-btn' ).toggleClass( 'toggle-expand' ).toggleClass( 'toggle' );
            $embeddedHistory.find( ".summary-content" ).slideToggle( "fast" );
            $embeddedHistory.find( ".annotation" ).slideToggle( "fast" );
            $embeddedHistory.find( ".expanded-content" ).slideToggle( "fast" );
            ev.preventDefault();
        }

        $(function(){
            var debugging = JSON.parse( sessionStorage.getItem( 'debugging' ) ) || false,
                historyModel = require( 'mvc/history/history-model' ),
                panel = new panelMod.AnnotatedHistoryPanel({
                    el      : $embeddedHistory.find( ".history-panel" ),
                    model   : new historyModel.History(
                        {"slug": "exons-vs-repeats", "user_id": "859c9f6eabd3410d", "name": "Exons vs Repeats", "tags": [], "deleted": false, "state_ids": {"paused": [], "ok": ["359495778e21d3ac", "eb30a62b22e7c84e", "b5e5669592a8ce68", "5b247bfca77b5c45", "7262a9323cfaea47", "3571431dd9cb56bf", "cd1e95b7aa37e5ff"], "failed_metadata": [], "upload": [], "discarded": [], "running": [], "setting_metadata": [], "error": [], "new": [], "queued": [], "empty": []}, "username_and_slug": "u/aun1/h/exons-vs-repeats", "purged": false, "state_details": {"paused": 0, "ok": 7, "failed_metadata": 0, "upload": 0, "discarded": 0, "running": 0, "setting_metadata": 0, "error": 0, "new": 0, "queued": 0, "empty": 0}, "genome_build": "?", "state": "ok", "published": true, "nice_size": "223.4 MB", "model_class": "History", "annotation": null, "id": "c1287a8c8ec5e9cc", "importable": true},
                        [{"misc_blurb": "586,513 regions", "peek": "<table cellspacing=\"0\" cellpadding=\"3\"><tr><th>1.Chrom</th><th>2.Start</th><th>3.End</th><th>4.Name</th><th>5</th><th>6.Strand</th></tr><tr><td>chr1</td><td>12189</td><td>12227</td><td>uc010nxq.1_cds_0_0_chr1_12190_f</td><td>0</td><td>+</td></tr><tr><td>chr1</td><td>12594</td><td>12721</td><td>uc010nxq.1_cds_1_0_chr1_12595_f</td><td>0</td><td>+</td></tr><tr><td>chr1</td><td>13402</td><td>13639</td><td>uc010nxq.1_cds_2_0_chr1_13403_f</td><td>0</td><td>+</td></tr><tr><td>chr1</td><td>69090</td><td>70008</td><td>uc001aal.1_cds_0_0_chr1_69091_f</td><td>0</td><td>+</td></tr><tr><td>chr1</td><td>324342</td><td>324345</td><td>uc009vjk.2_cds_1_0_chr1_324343_f</td><td>0</td><td>+</td></tr><tr><td>chr1</td><td>324438</td><td>325605</td><td>uc009vjk.2_cds_2_0_chr1_324439_f</td><td>0</td><td>+</td></tr></table>", "update_time": "2010-11-03T02:28:20.095115", "metadata_endCol": 3, "data_type": "bed", "tags": [], "deleted": false, "history_id": "c1287a8c8ec5e9cc", "metadata_column_names": null, "visible": true, "genome_build": "hg19", "metadata_nameCol": 4, "hid": 1, "metadata_startCol": 2, "metadata_chromCol": 1, "file_size": 37811429, "metadata_viz_filter_cols": [4], "metadata_data_lines": 586513, "file_ext": "bed", "api_type": "file", "id": "359495778e21d3ac", "misc_info": "UCSC Main on Human: knownGene (genome)", "hda_ldda": "hda", "visualizations": ["trackster", "circster"], "name": "UCSC Main on Human: knownGene (genome)", "metadata_columns": 6, "uuid": null, "download_url": "/api/histories/c1287a8c8ec5e9cc/contents/359495778e21d3ac/display", "annotation": null, "state": "ok", "metadata_comment_lines": null, "metadata_strandCol": 6, "model_class": "HistoryDatasetAssociation", "accessible": true, "metadata_dbkey": "hg19", "purged": false, "metadata_column_types": ["str", "int", "int", "str", "int", "str"]}, {"misc_blurb": "5,298,130 regions", "peek": "<table cellspacing=\"0\" cellpadding=\"3\"><tr><th>1.Chrom</th><th>2.Start</th><th>3.End</th><th>4.Name</th><th>5</th><th>6.Strand</th></tr><tr><td>chr1</td><td>16777160</td><td>16777470</td><td>AluSp</td><td>2147</td><td>+</td></tr><tr><td>chr1</td><td>25165800</td><td>25166089</td><td>AluY</td><td>2626</td><td>-</td></tr><tr><td>chr1</td><td>33553606</td><td>33554646</td><td>L2b</td><td>626</td><td>+</td></tr><tr><td>chr1</td><td>50330063</td><td>50332153</td><td>L1PA10</td><td>12545</td><td>+</td></tr><tr><td>chr1</td><td>58720067</td><td>58720973</td><td>L1PA2</td><td>8050</td><td>-</td></tr><tr><td>chr1</td><td>75496180</td><td>75498100</td><td>L1MB7</td><td>10586</td><td>+</td></tr></table>", "update_time": "2010-11-03T02:33:34.890862", "metadata_endCol": 3, "data_type": "bed", "tags": [], "deleted": false, "history_id": "c1287a8c8ec5e9cc", "metadata_column_names": null, "visible": true, "genome_build": "hg19", "metadata_nameCol": 4, "hid": 2, "metadata_startCol": 2, "metadata_chromCol": 1, "file_size": 193492882, "metadata_viz_filter_cols": [4], "metadata_data_lines": 5298130, "file_ext": "bed", "api_type": "file", "id": "eb30a62b22e7c84e", "misc_info": "UCSC Main on Human: rmsk (genome)", "hda_ldda": "hda", "visualizations": ["trackster", "circster"], "name": "UCSC Main on Human: rmsk (genome)", "metadata_columns": 6, "uuid": null, "download_url": "/api/histories/c1287a8c8ec5e9cc/contents/eb30a62b22e7c84e/display", "annotation": null, "state": "ok", "metadata_comment_lines": null, "metadata_strandCol": 6, "model_class": "HistoryDatasetAssociation", "accessible": true, "metadata_dbkey": "hg19", "purged": false, "metadata_column_types": ["str", "int", "int", "str", "int", "str"]}, {"misc_blurb": "17,891 regions", "peek": "<table cellspacing=\"0\" cellpadding=\"3\"><tr><th>1.Chrom</th><th>2.Start</th><th>3.End</th><th>4.Name</th><th>5</th><th>6.Strand</th><th>7</th><th>8</th><th>9</th><th>10</th><th>11</th><th>12</th></tr><tr><td>chr1</td><td>324342</td><td>324345</td><td>uc009vjk.2_cds_1_0_chr1_324343_f</td><td>0</td><td>+</td><td>chr1</td><td>324007</td><td>324427</td><td>MER1A</td><td>2216</td><td>+</td></tr><tr><td>chr1</td><td>324342</td><td>324345</td><td>uc001aau.2_cds_1_0_chr1_324343_f</td><td>0</td><td>+</td><td>chr1</td><td>324007</td><td>324427</td><td>MER1A</td><td>2216</td><td>+</td></tr><tr><td>chr1</td><td>880421</td><td>880526</td><td>uc001aby.3_cds_1_0_chr1_880422_r</td><td>0</td><td>-</td><td>chr1</td><td>880453</td><td>880497</td><td>CT-rich</td><td>211</td><td>+</td></tr><tr><td>chr1</td><td>880436</td><td>880526</td><td>uc001abz.3_cds_1_0_chr1_880437_r</td><td>0</td><td>-</td><td>chr1</td><td>880453</td><td>880497</td><td>CT-rich</td><td>211</td><td>+</td></tr><tr><td>chr1</td><td>880421</td><td>880526</td><td>uc009vjq.2_cds_1_0_chr1_880422_r</td><td>0</td><td>-</td><td>chr1</td><td>880453</td><td>880497</td><td>CT-rich</td><td>211</td><td>+</td></tr><tr><td>chr1</td><td>894010</td><td>894461</td><td>uc009vjr.1_cds_0_0_chr1_894011_r</td><td>0</td><td>-</td><td>chr1</td><td>893930</td><td>894175</td><td>L2a</td><td>278</td><td>+</td></tr></table>", "update_time": "2010-11-03T03:25:26.033060", "metadata_endCol": 3, "data_type": "interval", "tags": [], "deleted": false, "history_id": "c1287a8c8ec5e9cc", "metadata_column_names": null, "visible": false, "genome_build": "hg19", "metadata_nameCol": 4, "hid": 3, "metadata_startCol": 2, "metadata_chromCol": 1, "file_size": 1814913, "metadata_data_lines": 17891, "file_ext": "interval", "api_type": "file", "id": "b5e5669592a8ce68", "misc_info": "", "hda_ldda": "hda", "visualizations": ["trackster", "circster"], "name": "Join on data 2 and data 1", "metadata_columns": 12, "uuid": null, "download_url": "/api/histories/c1287a8c8ec5e9cc/contents/b5e5669592a8ce68/display", "annotation": null, "state": "ok", "metadata_comment_lines": null, "metadata_strandCol": 6, "model_class": "HistoryDatasetAssociation", "accessible": true, "metadata_dbkey": "hg19", "purged": false, "metadata_column_types": ["str", "int", "int", "str", "int", "str", "str", "int", "int", "str", "int", "str"]}, {"misc_blurb": "15,310 lines", "peek": "<table cellspacing=\"0\" cellpadding=\"3\"><tr><th>1</th><th>2</th></tr><tr><td>uc001aau.2_cds_1_0_chr1_324343_f</td><td>1</td></tr><tr><td>uc001aby.3_cds_1_0_chr1_880422_r</td><td>1</td></tr><tr><td>uc001abz.3_cds_1_0_chr1_880437_r</td><td>1</td></tr><tr><td>uc001aca.1_cds_0_0_chr1_896074_f</td><td>1</td></tr><tr><td>uc001acb.1_cds_1_0_chr1_897206_f</td><td>1</td></tr><tr><td>uc001aci.2_cds_0_0_chr1_934439_r</td><td>1</td></tr></table>", "update_time": "2010-11-03T03:25:33.430733", "data_type": "tabular", "tags": [], "deleted": false, "history_id": "c1287a8c8ec5e9cc", "metadata_column_names": null, "visible": false, "genome_build": "hg19", "hid": 4, "file_size": 586035, "metadata_data_lines": 15310, "file_ext": "tabular", "api_type": "file", "id": "5b247bfca77b5c45", "misc_info": "--Group by c4: count[c4]", "hda_ldda": "hda", "visualizations": [], "name": "Group on data 3", "metadata_columns": 2, "uuid": null, "download_url": "/api/histories/c1287a8c8ec5e9cc/contents/5b247bfca77b5c45/display", "annotation": null, "state": "ok", "metadata_comment_lines": null, "model_class": "HistoryDatasetAssociation", "accessible": true, "metadata_dbkey": "hg19", "purged": false, "metadata_column_types": ["str", "int"]}, {"misc_blurb": "15,310 lines", "peek": "<table cellspacing=\"0\" cellpadding=\"3\"><tr><th>1</th><th>2</th></tr><tr><td>uc003qqn.2_cds_0_0_chr6_157099238_f</td><td>12</td></tr><tr><td>uc003qqo.2_cds_0_0_chr6_157099238_f</td><td>12</td></tr><tr><td>uc003qqp.2_cds_0_0_chr6_157099238_f</td><td>12</td></tr><tr><td>uc003hqu.2_cds_4_0_chr4_88534937_f</td><td>10</td></tr><tr><td>uc001vqv.2_cds_1_0_chr13_110434389_r</td><td>8</td></tr><tr><td>uc001vsb.1_cds_0_0_chr13_112721973_f</td><td>8</td></tr></table>", "update_time": "2010-11-03T03:25:40.647358", "data_type": "tabular", "tags": [], "deleted": false, "history_id": "c1287a8c8ec5e9cc", "metadata_column_names": null, "visible": false, "genome_build": "hg19", "hid": 5, "file_size": 586035, "metadata_data_lines": 15310, "file_ext": "tabular", "api_type": "file", "id": "7262a9323cfaea47", "misc_info": "", "hda_ldda": "hda", "visualizations": [], "name": "Sort on data 4", "metadata_columns": 2, "uuid": null, "download_url": "/api/histories/c1287a8c8ec5e9cc/contents/7262a9323cfaea47/display", "annotation": null, "state": "ok", "metadata_comment_lines": null, "model_class": "HistoryDatasetAssociation", "accessible": true, "metadata_dbkey": "hg19", "purged": false, "metadata_column_types": ["str", "int"]}, {"misc_blurb": "5 lines", "peek": "<table cellspacing=\"0\" cellpadding=\"3\"><tr><th>1</th><th>2</th></tr><tr><td>uc003qqn.2_cds_0_0_chr6_157099238_f</td><td>12</td></tr><tr><td>uc003qqo.2_cds_0_0_chr6_157099238_f</td><td>12</td></tr><tr><td>uc003qqp.2_cds_0_0_chr6_157099238_f</td><td>12</td></tr><tr><td>uc003hqu.2_cds_4_0_chr4_88534937_f</td><td>10</td></tr><tr><td>uc001vqv.2_cds_1_0_chr13_110434389_r</td><td>8</td></tr></table>", "update_time": "2010-11-03T03:25:47.226745", "data_type": "tabular", "tags": [], "deleted": false, "history_id": "c1287a8c8ec5e9cc", "metadata_column_names": null, "visible": false, "genome_build": "hg19", "hid": 6, "file_size": 194, "metadata_data_lines": 5, "file_ext": "tabular", "api_type": "file", "id": "3571431dd9cb56bf", "misc_info": "", "hda_ldda": "hda", "visualizations": [], "name": "Select first on data 5", "metadata_columns": 2, "uuid": null, "download_url": "/api/histories/c1287a8c8ec5e9cc/contents/3571431dd9cb56bf/display", "annotation": null, "state": "ok", "metadata_comment_lines": null, "model_class": "HistoryDatasetAssociation", "accessible": true, "metadata_dbkey": "hg19", "purged": false, "metadata_column_types": ["str", "int"]}, {"misc_blurb": "5 regions", "peek": "<table cellspacing=\"0\" cellpadding=\"3\"><tr><th>1.Chrom</th><th>2.Start</th><th>3.End</th><th>4.Name</th><th>5</th><th>6.Strand</th></tr><tr><td>chr13</td><td>110434388</td><td>110438400</td><td>uc001vqv.2_cds_1_0_chr13_110434389_r</td><td>0</td><td>-</td></tr><tr><td>chr4</td><td>88534936</td><td>88537720</td><td>uc003hqu.2_cds_4_0_chr4_88534937_f</td><td>0</td><td>+</td></tr><tr><td>chr6</td><td>157099237</td><td>157100605</td><td>uc003qqn.2_cds_0_0_chr6_157099238_f</td><td>0</td><td>+</td></tr><tr><td>chr6</td><td>157099237</td><td>157100605</td><td>uc003qqo.2_cds_0_0_chr6_157099238_f</td><td>0</td><td>+</td></tr><tr><td>chr6</td><td>157099237</td><td>157100605</td><td>uc003qqp.2_cds_0_0_chr6_157099238_f</td><td>0</td><td>+</td></tr></table>", "update_time": "2010-11-03T03:26:18.790903", "metadata_endCol": 3, "data_type": "bed", "tags": [], "deleted": false, "history_id": "c1287a8c8ec5e9cc", "metadata_column_names": null, "visible": true, "genome_build": "hg19", "metadata_nameCol": 4, "hid": 7, "metadata_startCol": 2, "metadata_chromCol": 1, "file_size": 324, "metadata_viz_filter_cols": [4], "metadata_data_lines": 5, "file_ext": "bed", "api_type": "file", "id": "cd1e95b7aa37e5ff", "misc_info": "join (GNU coreutils) 8.5\nCopyright (C) 2010 Free Software Foundation, Inc.\nLicense GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>.\nThis is free software: you are free to change and redistribute it.\nThere is NO WARRANTY, to the exten", "hda_ldda": "hda", "visualizations": ["trackster", "circster"], "name": "top 5 exons", "metadata_columns": 6, "uuid": null, "download_url": "/api/histories/c1287a8c8ec5e9cc/contents/cd1e95b7aa37e5ff/display", "annotation": null, "state": "ok", "metadata_comment_lines": null, "metadata_strandCol": 6, "model_class": "HistoryDatasetAssociation", "accessible": true, "metadata_dbkey": "hg19", "purged": false, "metadata_column_types": ["str", "int", "int", "str", "int", "str"]}],
                        { logger: ( debugging )?( console ):( null ) }
                    )
                }).render();

            $embeddedHistory.find( '.expand-content-btn' ).click( toggleExpanded );
            $embeddedHistory.find( '.toggle-embed' ).click( toggleExpanded );
        });
    });
})();
</script>
<p>The workflow is embedded here&#160;(click on the green plus icon to import):</p>

<div class='embedded-item display 
workflow'>
    <div class='title'>
        
    

    
    
    
    <div style="float: left">
        <a class="display_in_embed icon-button toggle-expand" item_id="86beaa9c1d0aab70" item_class="$item.__class__.__name__" href="/workflow/display_by_username_and_slug?username=aun1&slug=galaxy101"
            title="Show workflow content"></a>
        <a class="toggle icon-button" href="/workflow/display_by_username_and_slug?username=aun1&slug=galaxy101" title="Hide workflow content"></a>
    </div>
    <div style="float: right;">
        
	<a href="/u/aun1/w/galaxy101/json-download" title="Save Worflow" class="icon-button disk">
	<!--
	<a href="/u/aun1/w/galaxy101/json" title="Worflow Link for Import" class="icon-button TODO ">
	-->
	
    

    
    
    
    
    <a href="/workflow/imp?id=86beaa9c1d0aab70" title="Import workflow" class="icon-button import"></a>
    <a class="icon-button go-to-full-screen" href="/workflow/display_by_username_and_slug?username=aun1&slug=galaxy101" title="Go to workflow"></a>


    </div>
    <h4><a class="toggle-embed" href="/workflow/display_by_username_and_slug?username=aun1&slug=galaxy101" title="Show or hide workflow content">Galaxy 
Workflow | 
    galaxy101</a></h4>
    
    <input type="hidden" name="ajax-item-content-url" value="/workflow/get_item_content_async?id=86beaa9c1d0aab70"/>

    </div>
    <div class='summary-content'>
        


    </div>
    <div class='expanded-content'>
        <div class='item-content'></div>
    </div>
</div>






<h1>THE END!</h1>

            </div>
        

        </div>
    </div>


            </div><!--end center-->
                <div id="right">
                    

    
    
    


    <div class="unified-panel-header" unselectable="on">
        <div class="unified-panel-header-inner">
            About this 
Page
        </div>
    </div>
    
    <div class="unified-panel-body">
        <div style="overflow: auto; height: 100%;">
            <div style="padding: 10px;">
            
                <div style="float: right;"><img src="https://secure.gravatar.com/avatar/b9399bac36a64f0b790621e527615664?d=identicon"></div>
            
                <h4>Author</h4>
                
                <p>aun1</p>
                
                <h4>Related Pages</h4>
                <p>
                    <a href="/page/list_published">All published pages</a><br>
                    <a href="/page/list_published?f-username=aun1">Published pages by aun1</a>
                
                <h4>Rating</h4>

                
                <div style="padding-bottom: 0.75em; float: left">
                    Community<br>
                    <span style="font-size:80%">
                        (<span id="num_ratings">49</span> ratings, 
                         <span id="ave_rating">4.9</span> average)
                    <span>
                </div>
                <div style="float: right">
                    <input name="star1" type="radio" class="community_rating_star star" disabled="disabled" value="1"
                    
                    />
                    <input name="star1" type="radio" class="community_rating_star star" disabled="disabled" value="2"
                    />
                    <input name="star1" type="radio" class="community_rating_star star" disabled="disabled" value="3"
                    />
                    <input name="star1" type="radio" class="community_rating_star star" disabled="disabled" value="4"
                    />
                    <input name="star1" type="radio" class="community_rating_star star" disabled="disabled" value="5"
                        checked="checked"
                    />
                </div>
                <div style="clear: both;"></div>
                    <div style="float: left">
                        Yours<br><span id="rating_feedback" style="font-size:80%; display: none">(thanks!)</span>
                    </div>
                    <div style="float: right">
                        <input name="star2" type="radio" class="user_rating_star" value="1"
                        />
                        <input name="star2" type="radio" class="user_rating_star" value="2"
                        />
                        <input name="star2" type="radio" class="user_rating_star" value="3"
                        />
                        <input name="star2" type="radio" class="user_rating_star" value="4"
                        />
                        <input name="star2" type="radio" class="user_rating_star" value="5"
                        />
                    </div>
                <div style="clear: both;"></div>
                        
                <h4>Tags</h4>
                <p>
                <div>
                    Community:
                    
    
    
    
    <div class="tag-element"
            id="6325198276987286528"
    >
        <div class="tag-area 
        ">

                
                <span class="tag-button">
                    <span class="tag-name">tutorial</span>
                </span>
                
                <span class="tag-button">
                    <span class="tag-name">snps</span>
                </span>
                
                <span class="tag-button">
                    <span class="tag-name">exons</span>
                </span>
            
        </div>
    </div>

    
    <script type="text/javascript">
        init_tag_click_function($('#6325198276987286528'), community_tag_click);
    </script>

                </div>
                    <p>
                    <div>
                        Yours:
                        
    
    
    
    
    <div class="tag-element"
            id="8232988295509883904"
    >
        <div class="tag-area 
                individual-tag-area
        ">

            
                    <input class="tag-input" type='text' size='15'/>
                    <img src='/static/images/fugue/tag--plus.png' class="add-tag-button" title="Add tags"/>
        </div>
    </div>

    
    <script type="text/javascript">
        //
        // Set up autocomplete tagger.
        //

        //
        // Default function get text to display on the toggle link.
        //
        var default_get_toggle_link_text_fn = function(tags)
        {
            var text = "";
            var num_tags = obj_length(tags);
            if (num_tags != 0)
              {
                text = num_tags + (num_tags != 1 ? " Tags" : " Tag");
                /*
                // Show first N tags; hide the rest.
                var max_to_show = 1;
    
                // Build tag string.
                var tag_strs = new Array();
                var count = 0;
                for (tag_name in tags)
                  {
                    tag_value = tags[tag_name];
                    tag_strs[tag_strs.length] = build_tag_str(tag_name, tag_value);
                    if (++count == max_to_show)
                      break;
                  }
                tag_str = tag_strs.join(", ");
            
                // Finalize text.
                var num_tags_hiding = num_tags - max_to_show;
                text = "Tags: " + tag_str + 
                  (num_tags_hiding != 0 ? " and " + num_tags_hiding + " more" : "");
                */
              }
            else
              {
                // No tags.
                text = "Add tags";
              }
            return text;
        };
        
        // Default function to handle a tag click.
        var default_tag_click_fn = function(tag_name, tag_value) { };
        
        
        var options =
        {
            tags : {},
            editable : true,
            get_toggle_link_text_fn: default_get_toggle_link_text_fn,
            tag_click_fn: community_tag_click,
            ajax_autocomplete_tag_url: "/tag/tag_autocomplete_data?item_id=924b1a46d03eaf1e&item_class=Page",
            ajax_add_tag_url: "/tag/add_tag_async?item_id=924b1a46d03eaf1e&item_class=Page&context=view.mako",
            ajax_delete_tag_url: "/tag/remove_tag_async?item_id=924b1a46d03eaf1e&item_class=Page&context=view.mako",
            delete_tag_img: "/static/images/delete_tag_icon_gray.png",
            delete_tag_img_rollover: "/static/images/delete_tag_icon_white.png",
            use_toggle_link: false
         };
         
        $('#8232988295509883904').find( 'input' ).autocomplete_tagging(options);
    </script>
    
    <style>
    .tag-area {
        display: block;
    }
    </style>

    <noscript>
    <style>
    .tag-area {
        display: block;
    }
    </style>
    </noscript>

                    </div>
            </div>    
        </div>
    </div>


                    <div class="unified-panel-footer">
                        <div class="panel-collapse right"></div>
                        <div class="drag"></div>
                    </div>
                </div><!--end right-->
        </div><!--end everything-->
    </body>
    

    <script src="/static/scripts/libs/jquery/jquery.event.drag.js?v=1396981224" type="text/javascript"></script>
<script src="/static/scripts/libs/jquery/jquery.event.hover.js?v=1396981224" type="text/javascript"></script>
<script src="/static/scripts/libs/jquery/jquery.form.js?v=1396981224" type="text/javascript"></script>
<script src="/static/scripts/libs/jquery/jquery.rating.js?v=1396981224" type="text/javascript"></script>
<script src="/static/scripts/galaxy.panels.js?v=1396981224" type="text/javascript"></script>
    <script type="text/javascript">
        
    ensure_dd_helper();
        
        
            var rp = new Panel( { panel: $("#right"), center: $("#center"), drag: $("#right > .unified-panel-footer > .drag" ), toggle: $("#right > .unified-panel-footer > .panel-collapse" ), right: true } );
            window.handle_minwidth_hint = function( x ) { rp.handle_minwidth_hint( x ) };
            force_right_panel = function( x ) { rp.force_panel( x ) };
    
    </script>
    <script type="text/javascript">
        var upload_form_error = function( msg ) {
            var $galaxy_mainBody = $("iframe#galaxy_main").contents().find("body"),
                $errMsg = $galaxy_mainBody.find( 'div.errormessage' );
            if ( !$errMsg.size() ){
                $errMsg = $( '<div/>' ).addClass( 'errormessage' ).prependTo( $galaxy_mainBody );
            }
            $errMsg.text( msg );
        }

        var uploads_in_progress = 0;
        function decrementUploadsInProgress(){
            uploads_in_progress -= 1;
            if( uploads_in_progress === 0 ){
                window.onbeforeunload = null;
            }
        }
        jQuery( function() {
            $("iframe#galaxy_main").load( function() {
                $(this).contents().find("form").each( function() {
                    if ( $(this).find("input[galaxy-ajax-upload]").length > 0 ){
                        var $form = $( this );

                        $(this).submit( function( event ) {
                            // Only bother using a hidden iframe if there's a file (e.g. big data) upload
                            var file_upload = false;
                            $(this).find("input[galaxy-ajax-upload]").each( function() {
                                if ( $(this).val() != '' ) {
                                    file_upload = true;
                                }
                            });
                            if ( ! file_upload ) {
                                return true;
                            }
                            // Make a synchronous request to create the datasets first
                            var async_datasets;
                            var upload_error = false;

                            //NOTE: in order for upload.py to match the datasets created below, we'll move the js File
                            //  object's name into the file_data field (not in the form only for what we send to
                            //  upload_async_create)
                            var formData = $( this ).serializeArray();
                            var name = function(){
                                var $fileInput = $form.find( 'input[name="files_0|file_data"]' );
                                if( /msie/.test( navigator.userAgent.toLowerCase() ) ){
                                    return $fileInput.val().replace( 'C:\\fakepath\\', '' );
                                } else {
                                    return $fileInput.get( 0 ).files[0].name;
                                }
                            }
                            formData.push({ name: "files_0|file_data", value: name });

                            $.ajax( {
                                async:      false,
                                type:       "POST",
                                url:        "/tool_runner/upload_async_create",
                                data:       formData,
                                dataType:   "json",
                                success:    function(array_obj, status) {
                                                if (array_obj.length > 0) {
                                                    if (array_obj[0] == 'error') {
                                                        upload_error = true;
                                                        upload_form_error(array_obj[1]);
                                                    } else {
                                                        async_datasets = array_obj.join();
                                                    }
                                                } else {
                                                    // ( gvk 1/22/10 ) FIXME: this block is never entered, so there may be a bug somewhere
                                                    // I've done some debugging like checking to see if array_obj is undefined, but have not
                                                    // tracked down the behavior that will result in this block being entered.  I believe the
                                                    // intent was to have this block entered if the upload button is clicked on the upload
                                                    // form but no file was selected.
                                                    upload_error = true;
                                                    upload_form_error( 'No data was entered in the upload form.  You may choose to upload a file, paste some data directly in the data box, or enter URL(s) to fetch data.' );
                                                }
                                            }
                            } );

                            // show the dataset we created above in the history panel
                            Galaxy && Galaxy.currHistoryPanel && Galaxy.currHistoryPanel.refreshHdas();

                            if (upload_error == true) {
                                return false;
                            } else {
                                $(this).find("input[name=async_datasets]").val( async_datasets );
                                $(this).append("<input type='hidden' name='ajax_upload' value='true'>");
                            }
                            // iframe submit is required for nginx (otherwise the encoding is wrong)
                            $(this).ajaxSubmit({
                                //iframe: true,
                                error: function( xhr, msg, status ){
                                    decrementUploadsInProgress();
                                },
                                success: function ( response, x, y, z ) {
                                    decrementUploadsInProgress();
                                }
                            });
                            uploads_in_progress++;
                            window.onbeforeunload = function() {
                                return "Navigating away from the Galaxy analysis interface will interrupt the "
                                        + "file upload(s) currently in progress.  Do you really want to do this?";
                            }
                            if ( $(this).find("input[name='folder_id']").val() != undefined ) {
                                var library_id = $(this).find("input[name='library_id']").val();
                                var show_deleted = $(this).find("input[name='show_deleted']").val();
                                if ( location.pathname.indexOf( 'admin' ) != -1 ) {
                                    $("iframe#galaxy_main").attr("src","/library_common/browse_library?cntrller=library_admin&id=" + library_id + "&created_ldda_ids=" + async_datasets + "&show_deleted=" + show_deleted);
                                } else {
                                    $("iframe#galaxy_main").attr("src","/library_common/browse_library?cntrller=library&id=" + library_id + "&created_ldda_ids=" + async_datasets + "&show_deleted=" + show_deleted);
                                }
                            } else {
                                $("iframe#galaxy_main").attr("src","/tool_runner/upload_async_message");
                            }
                            event.preventDefault();
                            return false;
                        });
                    }
                });
            });
        });
    </script>


</html>
